<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.450">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>161250 Data Analysis - Chapter 3: Probability Concepts and Distributions</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="../site_libs/quarto-nav/quarto-nav.js"></script>
<script src="../site_libs/clipboard/clipboard.min.js"></script>
<script src="../site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="../site_libs/quarto-search/fuse.min.js"></script>
<script src="../site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="../">
<link href="../studyguide/4-inference.html" rel="next">
<link href="../studyguide/2-eda.html" rel="prev">
<script src="../site_libs/quarto-html/quarto.js"></script>
<script src="../site_libs/quarto-html/popper.min.js"></script>
<script src="../site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="../site_libs/quarto-html/anchor.min.js"></script>
<link href="../site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="../site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="../site_libs/bootstrap/bootstrap.min.js"></script>
<link href="../site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="../site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit",
    "search-label": "Search"
  }
}</script>

<script src="../site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="../site_libs/lightable-0.0.1/lightable.css" rel="stylesheet">

  <script src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<link rel="stylesheet" href="../styles.css">
</head>

<body class="nav-sidebar floating nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="../index.html">
    <span class="navbar-title">161250 Data Analysis</span>
    </a>
  </div>
            <div id="quarto-search" class="" title="Search"></div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="../about.html" rel="" target="">
 <span class="menu-text">About</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../slides.html" rel="" target="">
 <span class="menu-text">Slides</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="../studyguide/index.html" rel="" target="" aria-current="page">
 <span class="menu-text">Study Guide</span></a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="../workshops/ws01.html" rel="" target="">
 <span class="menu-text">Workshops</span></a>
  </li>  
  <li class="nav-item dropdown ">
    <a class="nav-link dropdown-toggle" href="#" id="nav-menu-extra-code" role="button" data-bs-toggle="dropdown" aria-expanded="false" rel="" target="">
 <span class="menu-text">Extra Code</span>
    </a>
    <ul class="dropdown-menu" aria-labelledby="nav-menu-extra-code">    
        <li>
    <a class="dropdown-item" href="../exercises/Chap2more.R" rel="" target="">
 <span class="dropdown-text">Ex2</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/Chap3more.R" rel="" target="">
 <span class="dropdown-text">Ex3</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/Chap4more.R" rel="" target="">
 <span class="dropdown-text">Ex4</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/Chap5more.R" rel="" target="">
 <span class="dropdown-text">Ex5</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/Chap6more.R" rel="" target="">
 <span class="dropdown-text">Ex6</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/Chap7more.R" rel="" target="">
 <span class="dropdown-text">Ex7</span></a>
  </li>  
        <li>
    <a class="dropdown-item" href="../exercises/Chap8more.R" rel="" target="">
 <span class="dropdown-text">Ex8</span></a>
  </li>  
    </ul>
  </li>
</ul>
            <div class="quarto-navbar-tools ms-auto">
</div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
  <nav class="quarto-secondary-nav">
    <div class="container-fluid d-flex">
      <button type="button" class="quarto-btn-toggle btn" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
        <i class="bi bi-layout-text-sidebar-reverse"></i>
      </button>
      <nav class="quarto-page-breadcrumbs" aria-label="breadcrumb"><ol class="breadcrumb"><li class="breadcrumb-item"><a href="../studyguide/index.html">Study Guide</a></li><li class="breadcrumb-item"><a href="../studyguide/3-probability.html">Chapter 3: Probability Concepts and Distributions</a></li></ol></nav>
      <a class="flex-grow-1" role="button" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass" aria-controls="quarto-sidebar" aria-expanded="false" aria-label="Toggle sidebar navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">      
      </a>
    </div>
  </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
  <nav id="quarto-sidebar" class="sidebar collapse collapse-horizontal sidebar-navigation floating overflow-auto">
    <div class="sidebar-menu-container"> 
    <ul class="list-unstyled mt-1">
        <li class="sidebar-item sidebar-item-section">
      <div class="sidebar-item-container"> 
            <a href="../studyguide/index.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Study Guide</span></a>
          <a class="sidebar-item-toggle text-start" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar-section-1" aria-expanded="true" aria-label="Toggle section">
            <i class="bi bi-chevron-right ms-2"></i>
          </a> 
      </div>
      <ul id="quarto-sidebar-section-1" class="collapse list-unstyled sidebar-section depth1 show">  
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/1-data-collection.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 1: Data Collection and Quality Issues</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/2-eda.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 2: Exploratory Data Analysis (EDA)</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/3-probability.html" class="sidebar-item-text sidebar-link active">
 <span class="menu-text">Chapter 3: Probability Concepts and Distributions</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/4-inference.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 4: Statistical Inference</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/5-tabulated.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 5: Tabulated Counts</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/6-single.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 6: Models with a Single Continuous Predictor</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/7-multiple.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 7: Models with Multiple Continuous Predictors</span></a>
  </div>
</li>
          <li class="sidebar-item">
  <div class="sidebar-item-container"> 
  <a href="../studyguide/8-anova.html" class="sidebar-item-text sidebar-link">
 <span class="menu-text">Chapter 8: Analysis of Variance (ANOVA) and Covariance (ANCOVA)</span></a>
  </div>
</li>
      </ul>
  </li>
    </ul>
    </div>
</nav>
<div id="quarto-sidebar-glass" data-bs-toggle="collapse" data-bs-target="#quarto-sidebar,#quarto-sidebar-glass"></div>
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction">Introduction</a></li>
  <li><a href="#probability-a-placeholder-for-incomplete-knowledge" id="toc-probability-a-placeholder-for-incomplete-knowledge" class="nav-link" data-scroll-target="#probability-a-placeholder-for-incomplete-knowledge">Probability: a placeholder for incomplete knowledge</a>
  <ul class="collapse">
  <li><a href="#probability-as-a-relative-frequency" id="toc-probability-as-a-relative-frequency" class="nav-link" data-scroll-target="#probability-as-a-relative-frequency">Probability as a relative frequency</a></li>
  <li><a href="#mutually-exclusive-events" id="toc-mutually-exclusive-events" class="nav-link" data-scroll-target="#mutually-exclusive-events">Mutually exclusive events</a></li>
  <li><a href="#statistical-independence" id="toc-statistical-independence" class="nav-link" data-scroll-target="#statistical-independence">Statistical independence</a></li>
  <li><a href="#examples" id="toc-examples" class="nav-link" data-scroll-target="#examples">Examples</a>
  <ul class="collapse">
  <li><a href="#blood-group-example" id="toc-blood-group-example" class="nav-link" data-scroll-target="#blood-group-example">Blood group example</a></li>
  <li><a href="#beached-whales-example" id="toc-beached-whales-example" class="nav-link" data-scroll-target="#beached-whales-example">Beached whales example</a></li>
  </ul></li>
  <li><a href="#probability-tree-diagram" id="toc-probability-tree-diagram" class="nav-link" data-scroll-target="#probability-tree-diagram">Probability tree diagram</a>
  <ul class="collapse">
  <li><a href="#rules-of-the-probability-tree" id="toc-rules-of-the-probability-tree" class="nav-link" data-scroll-target="#rules-of-the-probability-tree">Rules of the Probability Tree</a></li>
  </ul></li>
  <li><a href="#bayes-rule" id="toc-bayes-rule" class="nav-link" data-scroll-target="#bayes-rule">Bayes rule</a></li>
  </ul></li>
  <li><a href="#probability-distributions" id="toc-probability-distributions" class="nav-link" data-scroll-target="#probability-distributions">Probability Distributions</a>
  <ul class="collapse">
  <li><a href="#probability-massdensity-functions" id="toc-probability-massdensity-functions" class="nav-link" data-scroll-target="#probability-massdensity-functions">Probability Mass/Density Functions</a></li>
  <li><a href="#discrete-probability-distributions" id="toc-discrete-probability-distributions" class="nav-link" data-scroll-target="#discrete-probability-distributions">Discrete probability distributions</a>
  <ul class="collapse">
  <li><a href="#binomial-distribution" id="toc-binomial-distribution" class="nav-link" data-scroll-target="#binomial-distribution">Binomial distribution</a></li>
  <li><a href="#poisson-distribution" id="toc-poisson-distribution" class="nav-link" data-scroll-target="#poisson-distribution">Poisson distribution</a></li>
  </ul></li>
  <li><a href="#continuous-probability-distributions" id="toc-continuous-probability-distributions" class="nav-link" data-scroll-target="#continuous-probability-distributions">Continuous probability distributions</a>
  <ul class="collapse">
  <li><a href="#normal-distribution" id="toc-normal-distribution" class="nav-link" data-scroll-target="#normal-distribution">Normal distribution</a></li>
  <li><a href="#skewed-distributions" id="toc-skewed-distributions" class="nav-link" data-scroll-target="#skewed-distributions">Skewed distributions</a></li>
  <li><a href="#empirical-distributions" id="toc-empirical-distributions" class="nav-link" data-scroll-target="#empirical-distributions">Empirical distributions</a></li>
  <li><a href="#importance-of-the-normal-distribution" id="toc-importance-of-the-normal-distribution" class="nav-link" data-scroll-target="#importance-of-the-normal-distribution">Importance of the normal distribution</a></li>
  <li><a href="#normal-data-in-practice" id="toc-normal-data-in-practice" class="nav-link" data-scroll-target="#normal-data-in-practice">Normal data in practice</a></li>
  <li><a href="#assessment-of-normality" id="toc-assessment-of-normality" class="nav-link" data-scroll-target="#assessment-of-normality">Assessment of Normality</a></li>
  <li><a href="#normal-quantile-quantile-plots" id="toc-normal-quantile-quantile-plots" class="nav-link" data-scroll-target="#normal-quantile-quantile-plots">Normal quantile-quantile plots</a></li>
  <li><a href="#some-non-normal-distributions" id="toc-some-non-normal-distributions" class="nav-link" data-scroll-target="#some-non-normal-distributions">Some non-normal distributions</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#summary" id="toc-summary" class="nav-link" data-scroll-target="#summary">Summary</a></li>
  </ul>
<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="3-probability.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<div class="quarto-title-block"><div><h1 class="title">Chapter 3: Probability Concepts and Distributions</h1><button type="button" class="btn code-tools-button dropdown-toggle" id="quarto-code-tools-menu" data-bs-toggle="dropdown" aria-expanded="false"><i class="bi"></i> Code</button><ul class="dropdown-menu dropdown-menu-end" aria-labelelledby="quarto-code-tools-menu"><li><a id="quarto-show-all-code" class="dropdown-item" href="javascript:void(0)" role="button">Show All Code</a></li><li><a id="quarto-hide-all-code" class="dropdown-item" href="javascript:void(0)" role="button">Hide All Code</a></li><li><hr class="dropdown-divider"></li><li><a id="quarto-view-source" class="dropdown-item" href="javascript:void(0)" role="button">View Source</a></li></ul></div></div>
</div>



<div class="quarto-title-meta">

    
  
    
  </div>
  

</header>

<blockquote class="blockquote">
<p>“<em>Misunderstanding of probability may be the greatest of all impediments to scientific literacy</em>”</p>
<p>— Stephen Jay Gould</p>
</blockquote>
<blockquote class="blockquote">
<p>“<em>Statisticians, like artists, have the bad habit of falling in love with their models.</em>”</p>
<p>— George Box</p>
</blockquote>
<section id="introduction" class="level1">
<h1>Introduction</h1>
<p>The concept of probability is the foundation of statistical inference. In other words, making inferences about a populations based on a sample relies on probability notions, and any uncertainty in our conclusions can be expressed in probabilistic terms. We also informally used the probability concepts in data collection, particularly with the <em>EPSEM</em> sampling methods.</p>
<p>The concepts we cover in this Chapter will be largely familiar to you from a first year course. Note that we are covering probability concepts and distributions at a very basic level without much maths.</p>
</section>
<section id="probability-a-placeholder-for-incomplete-knowledge" class="level1">
<h1>Probability: a placeholder for incomplete knowledge</h1>
<p>Probability is a mathematical framework used to quantify and describe expectations of random events. Okay, so what is randomness? Where does it come from? This is a philosophical question, really.</p>
<p>From a practical viewpoint, I find it useful to think of randomness and probability as <em>placeholders for incomplete knowledge</em>. If we have complete knowledge of something, we don’t need the concepts of randomness and probability.</p>
<p>To illustrate, when you roll a die, you don’t know what number you’ll get. On inspection of the die, you can see that it has six sides, numbered 1-6. If we assume that it will land on one side, and that there’s equal probability of landing on each of the six sides, then we can form a basic probability model.</p>
<table class="table">
<thead>
<tr class="header">
<th><span class="math inline">\(x\)</span></th>
<th><span class="math inline">\(P(x)\)</span></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>1</td>
<td>1/6</td>
</tr>
<tr class="even">
<td>2</td>
<td>1/6</td>
</tr>
<tr class="odd">
<td>3</td>
<td>1/6</td>
</tr>
<tr class="even">
<td>4</td>
<td>1/6</td>
</tr>
<tr class="odd">
<td>5</td>
<td>1/6</td>
</tr>
<tr class="even">
<td>6</td>
<td>1/6</td>
</tr>
</tbody>
</table>
<p>There are six possible outcomes with equal probability, so the probability of each is 1/6. The probabilities sum to 1.</p>
<p>Where does the randomness come into this scenario? Is there randomness inherent in the process of rolling a die? I would say no. If we had complete knowledge of the system, including the shape of the die, the starting position, the angle and velocity with which it is thrown, and the shape and texture of the surface it lands on, and a good physical model of how these things interact, then we could theoretically be able to predict perfectly the number on which it will land. The trouble is, of course, that all this information is extremely difficult to collect in enough detail to allow us to make such a prediction.</p>
<p>A simpler case is a shuffled deck of cards. When you shuffle a deck of cards for long enough, we’d consider the question of which card is on the top to be random. However, if we knew the starting conditions and exactly how the cards were shuffled, then there’s no randomness, and no need for a probability model.</p>
<p>But most of the world isn’t as simple as shuffling cards or even rolling a die. Our knowledge is so very limited that concepts of uncertainty, randomness, and probability are indispensable to anyone who wants to make any real sense of it.</p>
<section id="probability-as-a-relative-frequency" class="level2">
<h2 class="anchored" data-anchor-id="probability-as-a-relative-frequency">Probability as a relative frequency</h2>
<p>The naive or classical definition of probability is just a relative frequency. If <span class="math inline">\(n\)</span> is the total number of equally likely possible outcomes and if <span class="math inline">\(m\)</span> of those <span class="math inline">\(n\)</span> outcomes result in the occurrence of some given event, then the probability of that event is <span class="math inline">\(m/n\)</span>.</p>
<p>For example, if a fair die is rolled, there are <span class="math inline">\(n = 6\)</span> possible outcomes. Let the event of interest be <em>getting a number 4 or more</em>. So, the probability of this event is 3 out of 6 or <span class="math inline">\(p=1/2\)</span>.</p>
<p>The <em>sample space</em> or the set of all possible outcomes need not be finite. For example, tossing a coin until the first head appears will result in an infinite sample space.So, the probability can be viewed as a limiting or long run fraction of <span class="math inline">\(m/n\)</span> (i.e.&nbsp;when <span class="math inline">\(n \to \infty\)</span>).</p>
<p>When the sample space is finite and outcomes are equally likely, we can assume that classical probability will be the same as empirical probability. Note that to find the probability of a fair coin landing heads it is not necessary to toss the coin repeatedly and observe the proportion of heads. So when the number of trails is reasonably large, empirical probability may be treated as the true value.</p>
<p>Subjective probability (such as expert opinion) is useful in areas such as decision theory, and its definition does not fit as a relative frequency.</p>
<p>Probabilities can only be between <span class="math inline">\(0\)</span> and <span class="math inline">\(1\)</span> (impossible to certain).</p>
</section>
<section id="mutually-exclusive-events" class="level2">
<h2 class="anchored" data-anchor-id="mutually-exclusive-events">Mutually exclusive events</h2>
<p>A German researcher <span class="citation" data-cites="witt">(<a href="#ref-witt" role="doc-biblioref">Witt 1956</a>)</span> studied the effects of various drugs on the web-building abilities of orb-web spiders (<a href="#fig-webs">Figure&nbsp;1</a>).</p>
<div id="fig-webs" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/webs.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;1: Some examples of webs made by spiders dosed with various intoxicants</figcaption>
</figure>
</div>
<p>Say he classified the webs into three types:</p>
<ul>
<li><code>complete</code>, <code>partial</code>, or <code>fail</code>.</li>
</ul>
<p>He found the relative frequencies for the three types as follows:</p>
<ul>
<li><p><span class="math inline">\(P(W=\text{complete}) = 0.2\)</span></p></li>
<li><p><span class="math inline">\(P(W=\text{partial}) = 0.3\)</span></p></li>
<li><p><span class="math inline">\(P(W=\text{fail}) = 0.5\)</span></p></li>
</ul>
<p>The random variable <span class="math inline">\(W\)</span> has three possible states but any given observation can only belong to one state. You cannot observe a web that is both “partial” and “complete”, or “complete” and “fail”. These values or states are said to be mutually exclusive.</p>
<p>For mutually exclusive events,</p>
<ol type="1">
<li><p>The probability of any two events co-occurring is zero.</p></li>
<li><p>The probability of one event or another event occurring is the sum of the two respective probabilities.</p></li>
</ol>
<p>For example, either a complete web or a partial web being observed is equal to the sum of <span class="math inline">\(P(W = \text{complete})\)</span> and <span class="math inline">\(P(W = \text {partial})\)</span>. That is,</p>
<ol start="3" type="1">
<li>The probability of any one event not occurring is the sum of those remaining is <span class="math display">\[P(\text{not complete})    = P(W = \text{partial or}~W \text{ = fail})= 0.3 + 0.5 = 0.8 = 1 – P(W = \text{complete}) \]</span></li>
</ol>
</section>
<section id="statistical-independence" class="level2">
<h2 class="anchored" data-anchor-id="statistical-independence">Statistical independence</h2>
<p>If events <span class="math inline">\(A\)</span> and <span class="math inline">\(B\)</span> are statistically independent, then <span class="math display">\[P(A \textbf{ and } B) = P(A) \times P(B)\]</span></p>
<p>If the <span class="math inline">\(n\)</span> events <span class="math inline">\(A_1, A_2, \dots, A_n\)</span> are mutually independent, then: <span class="math display">\[P(A_1 \textbf{ and } A_2 \textbf{ and } \dots \textbf{ and } A_n) = P(A_1) \times  P(A_2) \times \dots \times P(A_n)\]</span></p>
<p>The probability of <em>event A occurring given that event B has already occurred</em> is written in shorthand as <span class="math inline">\(P(A|B)\)</span>.</p>
<p>For example, the probability of a card you’ve drawn being a 5, given that it is a spade. This is known as a <strong>conditional probability</strong>.</p>
<p>Note that the sample space is reduced to that where B has occurred.</p>
<p>We say that two events (A and B) are <strong>independent</strong> if</p>
<p><span class="math display">\[P(A | B) = P(A) \textbf{ and } P(B | A) = P(B)\]</span></p>
<p>This means that observing event A doesn’t make event B any more or less likely, and vice versa.</p>
<p>For any two events <span class="math inline">\(A\)</span> and <span class="math inline">\(B\)</span>,</p>
<p><span class="math display">\[P(A \textbf{ and } B ) = P(A|B) \times P(B)\]</span> and also</p>
<p><span class="math display">\[P(A \textbf{ and } B ) = P(B|A) \times P(A)\]</span></p>
</section>
<section id="examples" class="level2">
<h2 class="anchored" data-anchor-id="examples">Examples</h2>
<section id="blood-group-example" class="level3">
<h3 class="anchored" data-anchor-id="blood-group-example">Blood group example</h3>
<p>There are a large number of genetic-based blood-type systems that have been used for categorising blood. Two of these are the Rh system (Rh+ and Rh–) and the Kell system (K+ and K–). It is found that for any person, their blood type in any one system is independent of their blood type in any other.</p>
<p>For Europeans in New Zealand, about 81% are Rh+ and about 8% are K+. So, we can form a Table of counts as shown in <a href="#fig-Rh">Figure&nbsp;2</a>:</p>
<div id="fig-Rh" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/RhK.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;2: Categorising blood types</figcaption>
</figure>
</div>
<p>If a European New Zealander is chosen at random, what is the probability that they are (Rh+ and K+) or (Rh– and K–)?</p>
<p>The answer is 0.0648 + 0.1748 = 0.2396 (verify).</p>
<p>Suppose that a murder victim has a bloodstain on him with type (Rh– and K+), presumably from the assailant. What is the probability that a randomly selected person matches this type?</p>
<p>The answer is 0.0152 (verify).</p>
</section>
<section id="beached-whales-example" class="level3">
<h3 class="anchored" data-anchor-id="beached-whales-example">Beached whales example</h3>
<p>Assume that 0.63 is the probability of a randomly selected beached whale is a male. So the probability of a beached whale being female is <span class="math inline">\(1-0.63=0.37\)</span>. Suppose that we also observe whether beached whales are juvenile or adult, and the probability of a whale being juvenile, given that it is female, is 0.8, and the probability it is juvenile, given that it is male, is 0.4.</p>
<p>Let us denote the random variables and their outcomes as follows:</p>
<p><em>Sex</em> (S) = either female (<span class="math inline">\(f\)</span>) or male (<span class="math inline">\(m\)</span>)</p>
<p><em>Age</em> (A) = either juvenile (<span class="math inline">\(j\)</span>) or adult (<span class="math inline">\(a\)</span>)</p>
<p>We are given</p>
<p><span class="math inline">\(P(S=m) = 0.63\)</span><br>
<span class="math inline">\(P(A=j | S=f) = 0.8\)</span><br>
<span class="math inline">\(P(A=j | S=m) = 0.4\)</span></p>
</section>
</section>
<section id="probability-tree-diagram" class="level2">
<h2 class="anchored" data-anchor-id="probability-tree-diagram">Probability tree diagram</h2>
<p>Easy to visualise the probabilities with this representation as shown in <a href="#fig-PTD">Figure&nbsp;3</a>.</p>
<div id="fig-PTD" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/tree.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;3: Probability tree diagram</figcaption>
</figure>
</div>
<section id="rules-of-the-probability-tree" class="level3">
<h3 class="anchored" data-anchor-id="rules-of-the-probability-tree">Rules of the Probability Tree</h3>
<ol type="1">
<li><p>Within each level, all branches are mutually exclusive events.</p></li>
<li><p>The tree covers all possibilities (i.e., the entire sample space).</p></li>
<li><p>We multiply as we move along branches.</p></li>
<li><p>We add when we move across branches.</p></li>
</ol>
<p>What is the overall probability that a whale is juvenile? That is, what is <span class="math inline">\(P(A = j)\)</span> (or <span class="math inline">\(P(j)\)</span> for short)?</p>
<p><span class="math display">\[P(j) = P(m \textbf{ and } j) + P(f \textbf{ and } j)= 0.63\times 0.4+0.37 \times 0.8 = 0.548 \]</span></p>
</section>
</section>
<section id="bayes-rule" class="level2">
<h2 class="anchored" data-anchor-id="bayes-rule">Bayes rule</h2>
<p>What is the probability of the whale being female given that it is juvenile? i.e., what is <span class="math inline">\(P(f | j)\)</span> ?</p>
<p><span class="math inline">\(P(f | j)\)</span> doesn’t appear in the tree anywhere, so we will have to go back to first principles.</p>
<p>we know that</p>
<p><span class="math display">\[P(f \textbf{ and }  j)  = P(f | j) × P(j)\]</span> and<br>
<span class="math display">\[P(f \textbf{ and }  j)  = P(j | f) × P(f)\]</span></p>
<p>Thus, <span class="math display">\[P(f | j) × P(j) = P(j | f) × P(f)\]</span></p>
<p><span class="math display">\[P(f | j) = \frac {P(j | f) × P(f)} {P(j)}\]</span> This is known as the <em>Bayes rule</em> or theorem.</p>
<p>For our question <span class="math display">\[P(f | j) = \frac {P(j | f) × P(f)} {P(j)}= \frac{0.8 \times 0.37}{0.548}= 0.54\]</span></p>
</section>
</section>
<section id="probability-distributions" class="level1">
<h1>Probability Distributions</h1>
<p>Consider the number of eggs <span class="math inline">\((X)\)</span> in an Adelie penguin’s nest. The values range from <span class="math inline">\(1\)</span> to <span class="math inline">\(5\)</span>, each with a certain probability (or relative frequency) of occurrence.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>dfm <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">X=</span><span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>, </span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Probability=</span><span class="fu">c</span>(<span class="fl">0.1</span>, <span class="fl">0.2</span>, <span class="fl">0.3</span>,<span class="fl">0.25</span>,<span class="fl">0.15</span>)</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(kableExtra)</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a><span class="fu">kable</span>(dfm, <span class="at">booktabs=</span><span class="cn">TRUE</span>) <span class="sc">|&gt;</span> </span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<table class="table table-striped table-sm small" data-quarto-postprocess="true">
<thead>
<tr class="header">
<th style="text-align: right;" data-quarto-table-cell-role="th">X</th>
<th style="text-align: right;" data-quarto-table-cell-role="th">Probability</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: right;">1</td>
<td style="text-align: right;">0.10</td>
</tr>
<tr class="even">
<td style="text-align: right;">2</td>
<td style="text-align: right;">0.20</td>
</tr>
<tr class="odd">
<td style="text-align: right;">3</td>
<td style="text-align: right;">0.30</td>
</tr>
<tr class="even">
<td style="text-align: right;">4</td>
<td style="text-align: right;">0.25</td>
</tr>
<tr class="odd">
<td style="text-align: right;">5</td>
<td style="text-align: right;">0.15</td>
</tr>
</tbody>
</table>


</div>
</div>
<p>The barplot of these probabilities is shown as <a href="#fig-discr">Figure&nbsp;4</a>.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpmisc)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="fu">theme_set</span>(<span class="fu">theme_minimal</span>())</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>dfm <span class="sc">|&gt;</span> </span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(<span class="at">x=</span>X, <span class="at">y=</span><span class="st">`</span><span class="at">Probability</span><span class="st">`</span>) <span class="sc">+</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span> </span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span><span class="sc">:</span><span class="dv">9</span>))<span class="sc">+</span></span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="at">geom =</span> <span class="st">"table"</span>, <span class="at">label =</span> <span class="fu">list</span>(dfm), <span class="at">x=</span><span class="dv">8</span>, <span class="at">y=</span>.<span class="dv">15</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-discr" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-discr-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;4: Barplot of probabilities</figcaption>
</figure>
</div>
</div>
</div>
<p>This is an example of a theoretical probability distribution, which describes the behaviour of a data-producing system. It’s a model for producing random values with specific probabilities.</p>
<p>Note the probabilities add to <span class="math inline">\(1\)</span> because <span class="math inline">\({1,2,3,4,5}\)</span> is a complete distribution. There are no other possibilities.</p>
<p>Two important parameters for describing the probability distribution of a random variable <span class="math inline">\(X\)</span> are:</p>
<ul>
<li>The mean <span class="math inline">\(\mu\)</span> (or “expected value” <span class="math inline">\(\text{E}(X)\)</span>). This is the “point of balance”, that locates the distribution and puts equal mass either side.</li>
<li>The variance <span class="math inline">\(\sigma_X^2\)</span> (or <span class="math inline">\(\text{Var}(X)\)</span>) is a measure of spread. It is the average squared difference from a value to the mean.</li>
</ul>
<p>For a discrete probability distribution, the population mean <span class="math inline">\(\mu_X\)</span> is simply the sum of each outcome multiplied by its probability.</p>
<p><span class="math display">\[\mu_X = \text{E}(X) = \sum xP(X=x) = \sum xP(x)\]</span></p>
<p>In R,</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>X <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">5</span></span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>P <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fl">0.1</span>, <span class="fl">0.2</span>, <span class="fl">0.3</span>,<span class="fl">0.25</span>,<span class="fl">0.15</span>)</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>(Mean <span class="ot">&lt;-</span> <span class="fu">sum</span>(X<span class="sc">*</span>P))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 3.15</code></pre>
</div>
</div>
<p>The population variance is given by</p>
<p><span class="math display">\[\sigma_X^2 = \text{Var}(X) = \sum (x-\mu_X)^2 P(x)\]</span></p>
<p>The population SD is simply the square-root of the variance.</p>
<p>In R,</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>X <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">5</span></span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>P <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fl">0.1</span>, <span class="fl">0.2</span>, <span class="fl">0.3</span>,<span class="fl">0.25</span>,<span class="fl">0.15</span>)</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>Mean <span class="ot">&lt;-</span> <span class="fu">sum</span>(X <span class="sc">*</span> P)</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>(Variance <span class="ot">&lt;-</span> <span class="fu">sum</span>((X<span class="sc">-</span>Mean)<span class="sc">^</span><span class="dv">2</span><span class="sc">*</span>P))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 1.4275</code></pre>
</div>
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a>(SD <span class="ot">&lt;-</span> <span class="fu">sqrt</span>(Variance))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 1.19478</code></pre>
</div>
</div>
<section id="probability-massdensity-functions" class="level2">
<h2 class="anchored" data-anchor-id="probability-massdensity-functions">Probability Mass/Density Functions</h2>
<p>The above example of the distribution of numbers of eggs of penguins is an odd case, because we’ve simply listed the outcomes and probabilities. More often, we use a known probability distribution where the probabilities of any value of <span class="math inline">\(X\)</span> may be calculated using a Probability Mass Function (PMF, for discrete distributions) or Probability Density Function (PDF, for continuous distributions). A PMF or PDF is a formula that can be used to calculate the probability associated with any value; that is, <span class="math inline">\(\text{P}(X=x)\)</span>, the probability that the random variable <span class="math inline">\(X\)</span> will take a particular value <span class="math inline">\(x\)</span>. These probabilities arise from making particular assumptions about how the values arise.</p>
<p>The remainder of this chapter will discuss some of the more common probability distributions.</p>
</section>
<section id="discrete-probability-distributions" class="level2">
<h2 class="anchored" data-anchor-id="discrete-probability-distributions">Discrete probability distributions</h2>
<p>Discrete probability distributions are those that can produce a limited set of values, usually non-negative integers {0, 1, 2, 3, …}. The two most useful ones are the Binomial and the Poisson.</p>
<section id="binomial-distribution" class="level3">
<h3 class="anchored" data-anchor-id="binomial-distribution">Binomial distribution</h3>
<p>Let <span class="math inline">\(X\)</span> be the number of heads when two coins are tossed. The possible outcomes are shown in <a href="#fig-twocoins">Figure&nbsp;5</a>.</p>
<div id="fig-twocoins" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="images/binom.png" class="img-fluid figure-img"></p>
<figcaption class="figure-caption">Figure&nbsp;5: Possible outcomes for two coin tosses</figcaption>
</figure>
</div>
<p>In general, when there only two possible outcomes say <em>success</em> or <em>failure</em> (<span class="math inline">\(1\)</span> or <span class="math inline">\(0\)</span>; Yes or No; On or Off; Heads or Tails; etc.), we deal with a <em>binary</em> or <strong>Bernoulli</strong> random variable.</p>
<p>The count of the number of <em>successes</em> <span class="math inline">\(X\)</span> out of a fixed total of <span class="math inline">\(n\)</span> <em>independent</em> trials (in the above coin tossing example, there were <span class="math inline">\(n = 2\)</span> trials) follows the binomial distribution.</p>
<p>In other words, <span class="math inline">\(X \sim Bin(n, p)\)</span>, where <span class="math inline">\(p\)</span> the probability of a success. The binomial probability mass function <span class="math inline">\(P(X=x)\)</span> or <span class="math inline">\(P(x)\)</span> is given by</p>
<p><span class="math display">\[P(x)={n \choose x}p^{x}(1-p)^{n-x}\]</span></p>
<p>It is easy to plot these probabilities in R. For <span class="math inline">\(n=10\)</span>, <span class="math inline">\(p=0.3\)</span>, the binomial probabilities <span class="math inline">\(P(x)~~~ x=0,1,2, \dots, 10\)</span> are plotted in <a href="#fig-dbin">Figure&nbsp;6</a>.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-dbin" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-dbin-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;6: Binomial probabilities</figcaption>
</figure>
</div>
</div>
</div>
<p>Example: A microbiologist conducts an experiment to create a recombinant strain of bacteria that is resistant to penicillin. She plates out the bacteria on a plate, and picks out 10 colonies. She knows that the probability of successfully creating a recombinant is 0.15.</p>
<p>What is the probability that if she mixes all 10 colonies in a growth medium with penicillin, something (anything) will grow?</p>
<p>Given:</p>
<p><span class="math inline">\(X \sim Bin(n = 10, p = 0.15)\)</span>. What is <span class="math inline">\(P(x &gt; 0)\)</span>?</p>
<p>Note <span class="math inline">\(P(x &gt; 0)=1-P(x = 0)\)</span>. So in R, compute this as follows:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span> <span class="sc">-</span> <span class="fu">dbinom</span>(<span class="at">x=</span><span class="dv">0</span>, <span class="at">size=</span><span class="dv">10</span>, <span class="at">prob=</span>.<span class="dv">15</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.8031256</code></pre>
</div>
</div>
<p>or</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span><span class="sc">-</span><span class="fu">pbinom</span>(<span class="at">q=</span><span class="dv">0</span>, <span class="at">size=</span><span class="dv">10</span>, <span class="at">prob=</span>.<span class="dv">15</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.8031256</code></pre>
</div>
</div>
<p>Note that function <code>pbinom(k,size=n,prob=p)</code> gives the cumulative probabilities up to and including the quantile <span class="math inline">\(k\)</span> (which is equal to zero in our example).</p>
<p><span class="math display">\[P(X\leq k)=\sum _{i=0}^{k}{n \choose x}p^{x}(1-p)^{n-x}\]</span></p>
<p>The mean and variance of the binomial random variable is given by</p>
<p><span class="math display">\[\mu_X=np~~~~ \sigma^2_X=np(1-p)\]</span></p>
<p>So the expected number of recombinant strain of bacteria would be</p>
<p><span class="math display">\[\mu_X=np=10*0.15=1.5\]</span></p>
<p>with SD</p>
<p><span class="math display">\[\sigma_X=\sqrt {np(1-p)}=1.129159\]</span></p>
</section>
<section id="poisson-distribution" class="level3">
<h3 class="anchored" data-anchor-id="poisson-distribution">Poisson distribution</h3>
<p>The Poisson distribution is used to obtain the probabilities of counts of relatively rare events or outcomes that occur independently in space or time.</p>
<p>Some Examples:</p>
<ol type="1">
<li>The number of snails in a quadrat (<span class="math inline">\(1~m^2\)</span>)</li>
<li>Fish counts in a visual transect (25 x 5 m)</li>
<li>Bacterial colonies in 2 litres of milk</li>
</ol>
<p>The random variable <span class="math inline">\(X\)</span>, the number of occurrences (count), often follows the Poisson distribution whose probability function is given by</p>
<p><span class="math display">\[\Pr(x)= \frac{\lambda^x e^{-\lambda}}{x!}~~~ x=0,1,2,\dots, \infty\]</span></p>
<p>The parameter <span class="math inline">\(\lambda\)</span> is the mean which is also equal to the variance.</p>
<p><span class="math display">\[\mu_X=\lambda~~~~ \sigma^2_X=\lambda\]</span></p>
<p>Main assumptions:</p>
<ol type="1">
<li><p>The events occur at a constant average rate of <span class="math inline">\(\lambda\)</span> per unit time or space.</p></li>
<li><p>Occurrences are independent of one another as well as they do not happen at exactly the same unit time or space.</p></li>
</ol>
<p>Example: Consider the number of changes that accumulate along a stretch of a neutrally evolving gene over a given period of time.</p>
<p>This is a Poisson random variable with a population mean of <span class="math inline">\(\lambda=kt\)</span>, where <span class="math inline">\(k\)</span> is the number of mutations per generation, and <span class="math inline">\(t\)</span> is the time in generations that has elapsed.</p>
<p>Assume that <span class="math inline">\(k = 1\times10^{-4}\)</span> and <span class="math inline">\(t = 500\)</span>.</p>
<p>For <span class="math inline">\(\lambda=kt=0.05\)</span>, the Poisson probabilities are shown in <a href="#fig-dpoi">Figure&nbsp;7</a>.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-dpoi" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-dpoi-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;7: Poisson probabilities</figcaption>
</figure>
</div>
</div>
</div>
<p>What is the probability that at least one mutation has occurred over this period?</p>
<p><span class="math inline">\(P(x &gt; 0)=1-P(x = 0)\)</span> is found in R as follows:</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span> <span class="sc">-</span> <span class="fu">dpois</span>(<span class="at">x=</span><span class="dv">0</span>, <span class="at">lambda=</span><span class="fl">0.05</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.04877058</code></pre>
</div>
</div>
<p>Another example: The number of parasites in a host is a Poisson random variable with mean 1.5. What is the probability that there will be exactly one parasite in the host? Examine <a href="#fig-parasite">Figure&nbsp;8</a> for the answer.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-parasite" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-parasite-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;8: Parasites distribution</figcaption>
</figure>
</div>
</div>
</div>
</section>
</section>
<section id="continuous-probability-distributions" class="level2">
<h2 class="anchored" data-anchor-id="continuous-probability-distributions">Continuous probability distributions</h2>
<p>A discrete random variable takes values which are simply points on a real line. In other words, there is an inherent discontinuity in the values a discrete random variable can take.</p>
<p>If a random variable, <span class="math inline">\(X\)</span>, can take any value (i.e., not just integers) in some interval of the real line, it is called a <em>continuous</em> random variable. Examples include variables such as height, weight, length, percentage protein etc.</p>
<p>For a discrete random variable <span class="math inline">\(X\)</span>, the associated probabilities <span class="math inline">\(P(X=x)\)</span> are also just points or masses, and hence the probability function <span class="math inline">\(P(x)\)</span> is also called as the probability mass function (<em>pmf</em>).</p>
<p>For continuous random variables, probabilities can be computed when the variable falls in an interval such as <span class="math inline">\(5\)</span> to <span class="math inline">\(15\)</span>, but not when it takes a fixed value such as <span class="math inline">\(10\)</span> (which is equal to zero).</p>
<p>The probability density function (pdf) gives the probabilities as an area under the curve.</p>
<p>For example, consider a random proportion <span class="math inline">\((X)\)</span> between <span class="math inline">\(0\)</span> and <span class="math inline">\(1\)</span>. Here <span class="math inline">\(X\)</span> follows a (standard) continuous uniform distribution whose (probability) density function <span class="math inline">\(f(x)\)</span> is defined as follows:</p>
<p><span class="math display">\[f(x)=\begin{cases}{1}~~~\mathrm {for} \ 0\leq x\leq 1,\\[9pt]0~~~\mathrm {for} \ x&lt;0\ \mathrm {or} \ x&gt;1\end{cases}\]</span></p>
<p>This constant density function is very simple; see <a href="#fig-constdens">Figure&nbsp;9</a>.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="fu">data.frame</span>(<span class="at">x =</span> <span class="fu">c</span>(<span class="sc">-</span>.<span class="dv">5</span>, <span class="fl">1.5</span>)) <span class="sc">|&gt;</span> </span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>() <span class="sc">+</span> </span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(x) <span class="sc">+</span></span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(</span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">fun =</span> dunif, </span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">n =</span> <span class="dv">10001</span>, </span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">args =</span> <span class="fu">list</span>(<span class="at">min =</span> <span class="dv">0</span>, <span class="at">max =</span> <span class="dv">1</span>)</span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">+</span></span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">"f(x)"</span>) <span class="sc">+</span></span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-constdens" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-constdens-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;9: Constant density between 0 and 1</figcaption>
</figure>
</div>
</div>
</div>
<p>The cumulative distribution function, CDF, <span class="math inline">\(F(x)\)</span> gives the left tail area or probability up to <span class="math inline">\(x\)</span>. This is probability is found as</p>
<p><span class="math display">\[F_{X}(x)=\int _{-\infty }^{x}f_{X}(t)\,dt\]</span> The relationship between the density function <span class="math inline">\(f(x)\)</span> and the distribution function <span class="math inline">\(F(x)\)</span> is given by the <em>Fundamental Theorem of Calculus</em>.</p>
<p><span class="math display">\[f(x)={dF(x) \over dx}\]</span></p>
<p>The total area under the pdf curve is <span class="math inline">\(1\)</span>. The probability of obtaining a value between two points (<span class="math inline">\(a\)</span> and <span class="math inline">\(b\)</span>) is the area under the pdf curve between those two points. This probability is given by <span class="math inline">\(F(b)-F(a)\)</span>.</p>
<p>For the uniform distribution <span class="math inline">\(U(0,1)\)</span>, <span class="math inline">\(f(x)=1\)</span>. So,</p>
<p><span class="math display">\[F_{X}(x)=\int _{-\infty }^{x}\,dt=x\]</span></p>
<p>For example, the probability of a randomly drawn fraction from the interval <span class="math inline">\([0,1]\)</span> to fall below <span class="math inline">\(x=0.5\)</span> is 50%.</p>
<p>The probability of a random fraction falling between <span class="math inline">\(a=0.2\)</span> and <span class="math inline">\(b=0.8\)</span> is</p>
<p><span class="math display">\[F(b)-F(a)=0.8-0.2=0.6\]</span></p>
<section id="normal-distribution" class="level3">
<h3 class="anchored" data-anchor-id="normal-distribution">Normal distribution</h3>
<p>The Gaussian or normal distribution is parameterised in terms of the mean <span class="math inline">\(\mu\)</span> and the variance <span class="math inline">\(\sigma ^{2}\)</span> and its density function is given by</p>
<p><span class="math display">\[f(x)={\frac {1}{\sigma {\sqrt {2\pi }}}}e^{-{\frac {1}{2}}\left({\frac {x-\mu }{\sigma }}\right)^{2}}\]</span> If <span class="math inline">\(X \sim N(\mu, \sigma)\)</span>, its Z-score is calculated from <span class="math inline">\(X\)</span> by subtracting <span class="math inline">\(\mu\)</span> and dividing by the standard deviation <span class="math inline">\(\sigma\)</span>.</p>
<p><span class="math display">\[Z={\frac {X-\mu }{\sigma }}\]</span> The random variable <span class="math inline">\(Z \sim N(0,1)\)</span>. Its pdf is given by</p>
<p><span class="math display">\[f(z)={\frac {1}{ {\sqrt {2\pi }}}}e^{-{\frac {1}{2}}z^{2}}\]</span></p>
<p>We often deal with the Standard normal because the <em>symmetric</em> bell shape of the normal distribution remains the same for all <span class="math inline">\(\mu\)</span> and <span class="math inline">\(\sigma\)</span>; see <a href="#fig-nrm">Figure&nbsp;10</a>.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-nrm" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-nrm-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;10: Standard Normal Density</figcaption>
</figure>
</div>
</div>
</div>
<p>Example: The weight of an individual of <em>Amphibola crenata</em> (snail) is normally distributed with a mean of <span class="math inline">\(40g\)</span> and variance of <span class="math inline">\(20g^2\)</span> as shown in <a href="#fig-nrmeg">Figure&nbsp;11</a>.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a>dfs <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">x=</span><span class="fu">seq</span>(<span class="dv">20</span>, <span class="dv">60</span>, <span class="at">length=</span><span class="dv">1000</span>), </span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">f(x)</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">dnorm</span>(x, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>))</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>ps <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfs) <span class="sc">+</span> <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> <span class="st">`</span><span class="at">f(x)</span><span class="st">`</span>) <span class="sc">+</span> </span>
<span id="cb17-7"><a href="#cb17-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">fill=</span><span class="st">"gray"</span>) <span class="sc">+</span></span>
<span id="cb17-8"><a href="#cb17-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept=</span><span class="dv">40</span>) </span>
<span id="cb17-9"><a href="#cb17-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb17-10"><a href="#cb17-10" aria-hidden="true" tabindex="-1"></a>ps</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-nrmeg" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-nrmeg-1.png" class="img-fluid figure-img" width="384"></p>
<figcaption class="figure-caption">Figure&nbsp;11: Distribution of snail weight</figcaption>
</figure>
</div>
</div>
</div>
<p>What is the probability of getting a snail that weighs between <span class="math inline">\(35g\)</span> and <span class="math inline">\(50g\)</span>? In R, the function <code>pnorm()</code> gives the CDF.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">pnorm</span>(<span class="dv">50</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>)) <span class="sc">-</span> </span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pnorm</span>(<span class="dv">35</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>)) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.8555501</code></pre>
</div>
</div>
<p><a href="#fig-snail">Figure&nbsp;12</a> illustrates the computation.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>ps <span class="sc">+</span> </span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> dfs <span class="sc">|&gt;</span> <span class="fu">filter</span>(x <span class="sc">&lt;</span> <span class="dv">50</span> <span class="sc">&amp;</span> x <span class="sc">&gt;</span> <span class="dv">35</span>),</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>            <span class="at">fill=</span><span class="st">"coral1"</span>, <span class="at">alpha=</span>.<span class="dv">5</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-snail" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-snail-1.png" class="img-fluid figure-img" width="384"></p>
<figcaption class="figure-caption">Figure&nbsp;12: Probability of snail weight being between 35 g and 50 g</figcaption>
</figure>
</div>
</div>
</div>
<p>What is the probability of getting a snail that weighs below <span class="math inline">\(35g\)</span> or over <span class="math inline">\(50g\)</span>?</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="fu">pnorm</span>(<span class="dv">35</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>)) <span class="sc">+</span> </span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pnorm</span>(<span class="dv">50</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>), <span class="at">lower.tail=</span><span class="cn">FALSE</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>[1] 0.1444499</code></pre>
</div>
</div>
<p>Under standard normal, the areas under the pdf curve are shown in <a href="#fig-ncurves">Figure&nbsp;13</a> for various situations.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-ncurves" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-ncurves-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;13: Standard normal coverage</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="skewed-distributions" class="level3">
<h3 class="anchored" data-anchor-id="skewed-distributions">Skewed distributions</h3>
<p>There are many probability distributions which are asymmetrical. A few of them are briefed here.</p>
<p><strong>Log-normal</strong></p>
<p>A random variable <span class="math inline">\(X\)</span> is log-normally distributed, when <span class="math inline">\(log_e(X)\)</span> follows normal. Alternatively, if <span class="math inline">\(X\)</span> follows normal, then <span class="math inline">\(e^X\)</span> follows log-normal.</p>
<p>The log-normal density function is shown below:</p>
<p><span class="math display">\[f(x)={\frac {1}{x\sigma {\sqrt {2\pi }}}}\ \exp \left(-{\frac {\left(\ln x-\mu \right)^{2}}{2\sigma ^{2}}}\right),~~ x&gt;0\]</span> While the R function <code>dnorm()</code> gives the normal density, the function <code>dlnorm()</code> gives the log-normal density.</p>
<p>The standard log-normal pdf and CDF are plotted in <a href="#fig-lnrm">Figure&nbsp;14</a></p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-lnrm" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-lnrm-1.png" class="img-fluid figure-img" width="480"></p>
<figcaption class="figure-caption">Figure&nbsp;14: Log-normal pdf and CDF</figcaption>
</figure>
</div>
</div>
</div>
<p>The mean &amp; variance of the lognormal random variables are:</p>
<p><span class="math display">\[\mu_X=e^{\left(\mu +{\frac {\sigma ^{2}}{2}}\right)}~~~~\sigma_X^2=(e^{\sigma ^{2}}-1) e^{(2\mu +\sigma ^{2})}\]</span></p>
<p><strong>Weibull</strong></p>
<p>The probability function of the Weibull distribution is given below:</p>
<p><span class="math display">\[f(t;\eta,\beta) =
\begin{cases}
\frac{\beta}{\eta}\left(\frac{x}{\eta}\right)^{\beta-1}e^{-(x/\eta)^{\beta}}  ~~x\geq0 ,\\
0~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  x&lt;0,
\end{cases}\]</span></p>
<p><span class="math inline">\(\beta~(&gt; 0)\)</span> is the called the <strong>shape</strong> parameter and <span class="math inline">\(\eta~(&gt; 0)\)</span> is the called <strong>scale</strong> parameter, and different values of <span class="math inline">\(\beta\)</span> will result in different distributional shapes; see <a href="#fig-weib">Figure&nbsp;15</a>.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-weib" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-weib-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;15: Weibull distributional shapes</figcaption>
</figure>
</div>
</div>
</div>
<p>The Weibull distribution becomes the exponential distribution for <span class="math inline">\(\beta=1\)</span>.</p>
<p>The scale parameter <span class="math inline">\(\eta\)</span> is called the <em>characteristic life</em> because <span class="math inline">\(\eta\)</span> becomes the quantile with slightly less than two-thirds of the population (63.21%) below it irrespective of the shape <span class="math inline">\(\beta\)</span> when this distribution is used to model lifetimes.</p>
<p><strong>Gamma</strong></p>
<p>The probability function of the gamma distribution with shape parameter <span class="math inline">\(\alpha\)</span> and scale parameter <span class="math inline">\(\beta\)</span> is given below:</p>
<p><span class="math display">\[\displaystyle {\begin{aligned}f(x)={\frac {\beta ^{\alpha }x^{\alpha -1}e^{-\beta x}}{\Gamma (\alpha )}}\quad {\text{ for }}x&gt;0\quad \alpha ,\beta &gt;0,\\[6pt]\end{aligned}}\]</span> where <span class="math inline">\(\displaystyle \Gamma (\alpha)=\int _{0}^{\infty }x^{\alpha-1}e^{-x}\,dx.\)</span></p>
<p><a href="#fig-gam">Figure&nbsp;16</a> shows the plot of gamma density for various shape and scale parameters.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-gam" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-gam-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;16: Gamma distributional shapes</figcaption>
</figure>
</div>
</div>
</div>
<p>Lognormal, Weibull and gamma distributions are useful to model right skewed data, particularly lifetimes data, but we need large amount of data, say over 300, to discriminate the three fitted distributions.</p>
<p><strong>Beta distribution</strong></p>
<p>The beta distribution is bounded on the interval <span class="math inline">\([0, 1]\)</span> and parameterised by two positive shape parameters, say <span class="math inline">\(\alpha\)</span> and <span class="math inline">\(\beta\)</span>.</p>
<p>The probability function of the Weibull distribution is given below:</p>
<p><span class="math display">\[\begin{aligned}f(x;\alpha ,\beta ) ={\frac {x^{\alpha -1}(1-x)^{\beta -1}}{\displaystyle \int _{0}^{1}u^{\alpha -1}(1-u)^{\beta -1}\,du}}={\frac {1}{\mathrm {B} (\alpha ,\beta )}}x^{\alpha -1}(1-x)^{\beta -1}\end{aligned} \]</span> where <span class="math inline">\(\mathrm {B} (\alpha ,\beta )=\frac {\Gamma (\alpha )\Gamma (\beta )}{\Gamma (\alpha +\beta )}\)</span>. <a href="#fig-betas">Figure&nbsp;17</a> shows the plot of beta density for various shape parameter combinations.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-betas" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-betas-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;17: Beta distributional shapes</figcaption>
</figure>
</div>
</div>
</div>
<p>When <span class="math inline">\(\alpha=\beta=1\)</span>, the beta distribution becomes the continuous uniform distribution. This distribution is useful to model component fractions such as percentage of protein in milk powder, and is also widely used in Bayesian analysis.</p>
</section>
<section id="empirical-distributions" class="level3">
<h3 class="anchored" data-anchor-id="empirical-distributions">Empirical distributions</h3>
<p>Data from a normal (or any other distribution) may not look very much like the expected theoretical shape (‘bell-curve’ for normal) unless there is very large amount of it. <a href="#fig-rn200">Figure&nbsp;18</a> is for a random sample of 200 from the standard normal distribution. Compare the departures from the expected theoretical shape. Recall that we covered the ECDF curves in the last Chapter. The CDF curve is just theoretical one but fitted with data. More on this later.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-rn200" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-rn200-1.png" class="img-fluid figure-img" width="384"></p>
<figcaption class="figure-caption">Figure&nbsp;18: Normal(0,1) data (n=200)</figcaption>
</figure>
</div>
</div>
</div>
<p><strong>Small sample effect</strong> For small samples, we do not expect many outliers but the ‘ideal’ shape may not be seen; see <a href="#fig-rn50">Figure&nbsp;19</a>.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-rn50" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-rn50-1.png" class="img-fluid figure-img" width="384"></p>
<figcaption class="figure-caption">Figure&nbsp;19: Normal(0,1) data (n=50)</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="importance-of-the-normal-distribution" class="level3">
<h3 class="anchored" data-anchor-id="importance-of-the-normal-distribution">Importance of the normal distribution</h3>
<p><strong><em>Question</em></strong>: Why do we use the Normal distribution so much? Answer: Because:</p>
<ol type="1">
<li><p>Many physical, biological, engineering and economic quantities do seem to follow a Normal distribution (sometimes after a transformation of the data to an appropriate scale). An example is that the logarithm of the ‘waiting time’ between earthquakes in Wellington is normally distributed. (The use of the logarithm may worry you, but it is natural for the subject area. Note that the magnitude of earthquakes is also measured on a logarithmic scale, namely the familiar Richter scale.)</p></li>
<li><p>Data that are <em>approximately</em> Normal seem to turn up very frequently in practice. In particular this may occur because</p></li>
<li><p>The Normal distribution is often what you get <em>(for mathematical reasons)</em> when you add together (or average) lots of different things. This result is known as the central limit theorem (CLT). For example, test scores are calculated by adding together marks from different questions, or share market movements are the result of many individuals and firms making private decisions. There is another theory which establishes that the sampling distributions of quantiles of continuous distributions of are approximately normal for <em>large</em> sample sizes.</p></li>
<li><p>We shall see in this course that residuals play a large part. These are, for example, the ‘errors’ left over after fitting a regression line to data. Quite often, we expect residuals to follow a Normal distribution.</p></li>
</ol>
</section>
<section id="normal-data-in-practice" class="level3">
<h3 class="anchored" data-anchor-id="normal-data-in-practice">Normal data in practice</h3>
<p>In practice, data from a ‘Normal distribution’ <em>may</em> not look very much like the ‘bell-curve’ unless there is very large amount of it. This statement has practical implications but first let us check whether the statement is true. We can do this by simulating batches of data from a Normal distribution (using software) and noting the variations across the data sets. Moreover, because the Normal distribution and its bell shaped curve are so important and ubiquitous, it is instructive to apply EDA methods to Normal data, for we would certainly like EDA methods to work well for that type of curve.</p>
<p>For the first batch, we simulated 1000 numbers from a Normal distribution with mean 80 and standard deviation of 12. The numbers could represent, for example, the diastolic blood pressure of 1000 adult New Zealanders. <a href="#fig-simnor">Figure&nbsp;20</a> shows the histogram and the boxplot of the random normal data. Even though the typical bell-shaped pattern (theoretical normal curve shown in dotted lines) is evident, the histogram is not perfectly symmetric as the numbers were chosen at random so that by chance more fall into some classifications than would be expected. Also there are a few unusual values at the both extremes (see the boxplot). In practice if we didn’t know that the data came from a Normal distribution we might investigate the three highest points to see if they were valid points or outliers. The extreme edges of a data often comprise very atypical values we would not tend to place too much weight on the conflicting evidence they give about symmetry. The mean and median of this simulated batch are somewhat closer but the median is slightly off-centred from the quartiles.<em>Well, so far so good. Now what if we have smaller batches?</em></p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-simnor" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-simnor-1.png" class="img-fluid figure-img" width="576"></p>
<figcaption class="figure-caption">Figure&nbsp;20: Simulated normal data, n=100 and n=1000</figcaption>
</figure>
</div>
</div>
</div>
<p>For a normal distribution,</p>
<p>68% of the values fall within 1 standard deviation of the mean<br>
95% of the values fall within 2 standard deviation of the mean<br>
99% of the values fall within 3 standard deviation of the mean.<br>
</p>
<p>As a special case, 50% of the values will fall within 0.6745 standard deviations of the mean. For symmetric distributions, mean equals the median. Hence the fourths (quartiles) of random data obtained drawn from theoretical normal population will be</p>
<p><strong>Lower hinge</strong> = median - <span class="math inline">\(0.6745\times S~(\simeq\)</span>lower quartile),</p>
<p>and</p>
<p><strong>Upper hinge</strong> = median + <span class="math inline">\(0.6745 \times S~(\simeq\)</span> upper quartile),</p>
<p>where <span class="math inline">\(S\)</span> is the standard deviation. Thus F-spread (or <span class="math inline">\(\simeq\)</span> the IQR) <span class="math inline">\(\simeq\)</span> 1.35<span class="math inline">\(\times S\)</span>. Hence</p>
<p><strong>inner fences</strong>= median <span class="math inline">\(\pm\)</span> 2 F-spread = median <span class="math inline">\(\pm\)</span> 2.7<span class="math inline">\(\times S\)</span></p>
<p>The probability that an observation falls between these fences is 0.993, or the probability of falling outside these fences is 0.007. Thus in our simulated batch of size 1000, we expect about 7 possible outliers but none for the simulated batch size 100. <strong>Outer fences</strong> are defined as being 2 steps (3<span class="math inline">\(\times\)</span>F-spread) away from the fourths so that</p>
<p><strong>outer fences</strong>= median <span class="math inline">\(\pm\)</span> 3.5<span class="math inline">\(\times\)</span>F-spread = median <span class="math inline">\(\pm\)</span> 4.72<span class="math inline">\(\times S\)</span></p>
<p>The probability of a point falling outside of this range is the extremely small amount of 0.000002. It is not surprising, then, that none of the values in our simulated batches fell outside of the outer fences. If they did, we would class them as probable outliers. Of course, with distributions other than the Gaussian one, and particularly for asymmetric distributions, the probability of a value falling outside these fences is higher.</p>
</section>
<section id="assessment-of-normality" class="level3">
<h3 class="anchored" data-anchor-id="assessment-of-normality">Assessment of Normality</h3>
<p>The shapes of the ECDF curves shown in <a href="#fig-simnorecdf">Figure&nbsp;21</a> are typical of the empirical cumulative distributions for data from Normal populations. This curve was drawn using 1000 and 100 simulated random normal data. It could perhaps be described as an S-shape on its side (or lazy or stretched out S). The curve is flatter at either end which reflects the small amounts of relative frequency in the tails of the Normal distribution. The theoretical Cumulative Distribution Function (CDF) curve for the normal distribution is also shown in <a href="#fig-simnorecdf">Figure&nbsp;21</a> as a smooth curve. For smaller samples from a Normal population, the ECDF curve will vary about the theoretical curve, and, of course, the variation is likely to be larger the smaller the sample; see <a href="#fig-simnorecdf">Figure&nbsp;21</a>(B).</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-simnorecdf" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-simnorecdf-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;21: Simulated normal data, n=100 and n=1000</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="normal-quantile-quantile-plots" class="level3">
<h3 class="anchored" data-anchor-id="normal-quantile-quantile-plots">Normal quantile-quantile plots</h3>
<p>To visually check whether a given set of data are normally distributed, one can create a normal quantile-quantile (Q-Q) plot. In this plot the quantiles of the sample are plotted against the theoretical quantiles of a standard normal distribution N(0,1). If the data, <span class="math inline">\(Y_{i}\)</span>, are normally distributed N(<span class="math inline">\(\mu, \sigma\)</span>) then <span class="math inline">\(Z_{i} = \left(Y_{i} -\mu \right)/\sigma\)</span> has a standard normal distribution. If we plot the quantiles of the data against corresponding quantiles of the standard normal distribution, the points should roughly lie on the straight line: <span class="math display">\[Y_{i} =\mu +\sigma Z_{i}\]</span> that is, a line with intercept <span class="math inline">\(\mu\)</span>, and slope <span class="math inline">\(\sigma\)</span>.</p>
<p>We should note that the mean and standard deviation do not appear in the formula <span class="math inline">\(Y_{i} =\mu +\sigma Z_{i}\)</span> by coincidence. Rather, the mean and standard deviation are intimately connected to the normal distribution: if the data had not been normal then the line with these parameters would not pass through the data.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-normqq" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-normqq-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;22: Normal Quantile-Quantile Plot for Checking Normality</figcaption>
</figure>
</div>
</div>
</div>
<p><a href="#fig-normqq">Figure&nbsp;22</a> shows the normal Q-Q plot for 50 random values from N(80,12). Note that the points in the middle largely plot on a line. It is natural that the tail part of the distribution shows more variation, and hence we will ignore such departures.</p>
<p>The normal Q-Q plot is presented as a <strong>normal probability plot</strong> in some software where the theoretical quantiles are replaced by the associated normal probabilities. Such normal probability plots show the sample data in the <span class="math inline">\(X\)</span>-axis and the normal probabilities in the <span class="math inline">\(Y\)</span>-axis. In both plots, we mainly look whether or not the points plot roughly on a straight line.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">download.file</span>(</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">url =</span> <span class="st">"http://www.massey.ac.nz/~anhsmith/data/rangitikei.RData"</span>,</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">destfile =</span> <span class="st">"rangitikei.RData"</span>)</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">"rangitikei.RData"</span>)</span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(rangitikei) <span class="sc">+</span></span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(<span class="at">sample =</span> people) <span class="sc">+</span> </span>
<span id="cb23-9"><a href="#cb23-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_qq</span>() <span class="sc">+</span> <span class="fu">stat_qq_line</span>()</span>
<span id="cb23-10"><a href="#cb23-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-11"><a href="#cb23-11" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(rangitikei) <span class="sc">+</span> <span class="fu">aes</span>(<span class="at">y=</span>people, <span class="at">x=</span><span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb23-12"><a href="#cb23-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb23-13"><a href="#cb23-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">""</span>) <span class="sc">+</span> </span>
<span id="cb23-14"><a href="#cb23-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>()</span>
<span id="cb23-15"><a href="#cb23-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-16"><a href="#cb23-16" aria-hidden="true" tabindex="-1"></a>gridExtra<span class="sc">::</span><span class="fu">grid.arrange</span>(p1, p2, <span class="at">ncol=</span><span class="dv">1</span>) </span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-normqqP" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-normqqP-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;23: Normal Quantile-Quantile Plot for Checking Normality</figcaption>
</figure>
</div>
</div>
</div>
<p><a href="#fig-normqqP">Figure&nbsp;23</a> shows the normal Q-Q plot for the number of people who made use of a recreational facility (<strong>rangitikei</strong>). This plot clearly reveals that the data are not normal. The plotted points show a curved but asymmetric pattern. This pattern matches with the right skewness seen in the data (see the boxplot given in <a href="#fig-normqqP">Figure&nbsp;23</a>.</p>
<p>Note that <strong>normality and symmetry are not the same</strong>. If the true distribution of the data is symmetric such as uniform, rectangular or triangular distributions, then the normal Q-Q plot will show a symmetric pattern for the plotted points but they will not plot on a straight line. As an example, the normal Q-Q plots for batches of random data from Student’s <span class="math inline">\(t\)</span> (to be covered later in the next Chapter) and few other symmetric distributions is shown in <a href="#fig-normqqsym">Figure&nbsp;24</a>. Clearly normality is rejected for other symmetric non-normal distributions.</p>
<div class="cell">
<div class="cell-output-display">
<div id="fig-normqqsym" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-normqqsym-1.png" class="img-fluid figure-img" width="576"></p>
<figcaption class="figure-caption">Figure&nbsp;24: Normal Quantile-Quantile Plot for Checking Normality</figcaption>
</figure>
</div>
</div>
</div>
</section>
<section id="some-non-normal-distributions" class="level3">
<h3 class="anchored" data-anchor-id="some-non-normal-distributions">Some non-normal distributions</h3>
<p>For a normal distribution, which is specified by its mean <span class="math inline">\(\mu\)</span> and variance <span class="math inline">\(\sigma^2\)</span>, parameter estimation is fairly straight forward. The sample mean and variance become the estimates. The Poisson parameter <span class="math inline">\(\lambda\)</span> is also the mean of the distribution and hence the sample mean becomes the estimate. On the other hand, distributions such as beta, and gamma involve shape and/or scale parameters and hence their estimation becomes complicated. The population mean and SD of these distributions are functions of the distributional parameters. Data from skewed distributions are not well represented by the sample mean and SD.</p>
<p>In order to estimate the distributional parameters, we follow a method called the <strong>Maximum Likelihood</strong> (ML) method. The <em>likelihood</em> function (or simply the likelihood) is based on the joint probability of the observed data as a function of the distributional parameters. The ML method maximises the likelihood function to estimate the distributional (model) parameters. There are also other methods of estimation such as the method of moments and we will not delve into the theory behind these methods in this course. We will simply use R packages such as <code>VGAM</code> or <code>fitdistrplus</code> to estimate the parameters.</p>
<p>Example: Your are familiar with the <strong>rangitikei</strong> dataset which gives the number of people who made use of a recreational facility. This variable is right skewed, and let us fit a lognormal distribution to this variable using the <code>fitdistrplus</code> package.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(fitdistrplus, <span class="at">exclude =</span> <span class="fu">c</span>(<span class="st">"select"</span>, <span class="st">"area"</span>))</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in rm(list = exclude, envir = env): object 'select' not found</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Warning in rm(list = exclude, envir = env): object 'area' not found</code></pre>
</div>
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"lnorm"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Fitting of the distribution ' lnorm ' by maximum likelihood 
Parameters:
        estimate Std. Error
meanlog 3.775514  0.1789851
sdlog   1.028191  0.1265611</code></pre>
</div>
</div>
<p>The adequacy of the lognormal distribution fitted, can be examined using graphical displays as in <a href="#fig-lnormfit">Figure&nbsp;25</a>.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>lnormfit <span class="ot">&lt;-</span> <span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"lnorm"</span>)</span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(lnormfit)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-lnormfit" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-lnormfit-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;25: Log-normal fit for people</figcaption>
</figure>
</div>
</div>
</div>
<p>The lognormal fit does not appear too bad to people data. How about fitting gamma distribution instead of lognormal? This fit can again be accomplished using the <code>fitdistrplus</code> package.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a><span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"gamma"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>Fitting of the distribution ' gamma ' by maximum likelihood 
Parameters:
        estimate  Std. Error
shape 1.14299006 0.249453814
rate  0.01593588 0.004314727</code></pre>
</div>
</div>
<p>The EDA plots to assess the gamma distribution fit are shown in <a href="#fig-gammafit">Figure&nbsp;26</a>.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>gammafit <span class="ot">&lt;-</span> <span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"gamma"</span>)</span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(gammafit)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div id="fig-gammafit" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-gammafit-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;26: Gamma fit for people</figcaption>
</figure>
</div>
</div>
</div>
<p>Note that this package gives the estimated gamma parameters in a reparametrised form. For small and medium size datasets <span class="math inline">\((n&lt;300)\)</span>, we may not be able discriminate between gamma &amp; Weibull type of right skewed distributions.</p>
<p>We can of course fit a gallery of probability distributions but we have to select the <em>best</em> fitting distribution, if it exists.</p>
<p>The function <code>descdist()</code> in the <code>fitdistrplus</code> package will obtain a plot of kurtosis, a measure of excessive peakedness, against the square of a measure of skew for a number of probability distributions. This plot is known as the <em>Cullen and Frey plot</em>. We can visibly judge the best fitting distribution using this plot. For the number of people data, this plot is shown in <a href="#fig-cfplot">Figure&nbsp;27</a>.</p>
<div class="cell">
<details>
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a><span class="fu">descdist</span>(rangitikei<span class="sc">$</span>people)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>summary statistics
------
min:  4   max:  470 
median:  46 
mean:  71.72727 
estimated sd:  86.28089 
estimated skewness:  3.3264 
estimated kurtosis:  17.11618 </code></pre>
</div>
<div class="cell-output-display">
<div id="fig-cfplot" class="quarto-figure quarto-figure-center anchored">
<figure class="figure">
<p><img src="3-probability_files/figure-html/fig-cfplot-1.png" class="img-fluid figure-img" width="672"></p>
<figcaption class="figure-caption">Figure&nbsp;27: Cullen and Frey plot for people</figcaption>
</figure>
</div>
</div>
</div>
<p>The presence of a large outlier in peoples data particularly affect the measures of skewness and kurtosis. So Cullen and Frey plot does not particularly strongly support the lognormal fit but suggests the lack of fit for symmetrical distributions such as normal (read corresponding to zero skew).</p>
<p>When the observed data is a mixture from two or more distributions or contain a large number of unusual observations, no single distributional fit will be satisfactory. For example, count data may not fit the Poisson distribution due to excessive zeros.</p>
<p>We will cover the power transformation of data in the next Chapter, which is a remedy for dampening the effect of outliers and mixture of subgroups (distributions).</p>
</section>
</section>
</section>
<section id="summary" class="level1">
<h1>Summary</h1>
<p>This Chapter provided a revision of probability concepts usually covered the first year level covered at the seventh form level or in a first year Statistics course. A few popular probability distributions are also introduced.</p>
<p>Identification of a probability distribution is not an easy task for a set of observed data. This issue is not settled purely on statistical grounds. EDA helps to short-list distributions fitting reasonably well to the data. For regression type of models fitted to continuous responses, the normal distribution works generally well.</p>
<p>The material covered in this Chapter forms the foundation for many of the inferential methods.</p>


<!-- -->


</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="list">
<div id="ref-witt" class="csl-entry" role="listitem">
Witt, P. N. 1956. <em>Die Wirkung von Substanzen Auf Den Netzbau Der Spinne Als Biologischer Test</em>. Berline-Gottinger-Heidelberg. Springer-Verlag.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
<nav class="page-navigation">
  <div class="nav-page nav-page-previous">
      <a href="../studyguide/2-eda.html" class="pagination-link">
        <i class="bi bi-arrow-left-short"></i> <span class="nav-page-text">Chapter 2: Exploratory Data Analysis (EDA)</span>
      </a>          
  </div>
  <div class="nav-page nav-page-next">
      <a href="../studyguide/4-inference.html" class="pagination-link">
        <span class="nav-page-text">Chapter 4: Statistical Inference</span> <i class="bi bi-arrow-right-short"></i>
      </a>
  </div>
</nav><div class="modal fade" id="quarto-embedded-source-code-modal" tabindex="-1" aria-labelledby="quarto-embedded-source-code-modal-label" aria-hidden="true"><div class="modal-dialog modal-dialog-scrollable"><div class="modal-content"><div class="modal-header"><h5 class="modal-title" id="quarto-embedded-source-code-modal-label">Source Code</h5><button class="btn-close" data-bs-dismiss="modal"></button></div><div class="modal-body"><div class="">
<div class="sourceCode" id="cb35" data-shortcodes="false"><pre class="sourceCode markdown code-with-copy"><code class="sourceCode markdown"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> "Chapter 3: Probability Concepts and Distributions"</span></span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; “_Misunderstanding of probability may be the greatest of all impediments to scientific literacy_” </span></span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; </span></span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; — Stephen Jay Gould</span></span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; "_Statisticians, like artists, have the bad habit of falling in love with their models._" </span></span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a><span class="at">&gt;</span></span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; — George Box</span></span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-14"><a href="#cb35-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-15"><a href="#cb35-15" aria-hidden="true" tabindex="-1"></a><span class="fu"># Introduction</span></span>
<span id="cb35-16"><a href="#cb35-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-17"><a href="#cb35-17" aria-hidden="true" tabindex="-1"></a>The concept of probability is the foundation of statistical inference. In other words, making inferences about a populations based on a sample relies on probability notions, and any uncertainty in our conclusions can be expressed in probabilistic terms. We also informally used the probability concepts in data collection, particularly with the *EPSEM* sampling methods.</span>
<span id="cb35-18"><a href="#cb35-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-19"><a href="#cb35-19" aria-hidden="true" tabindex="-1"></a>The concepts we cover in this Chapter will be largely familiar to you from a first year course. Note that we are covering probability concepts and distributions at a very basic level without much maths. </span>
<span id="cb35-20"><a href="#cb35-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-21"><a href="#cb35-21" aria-hidden="true" tabindex="-1"></a><span class="fu"># Probability: a placeholder for incomplete knowledge</span></span>
<span id="cb35-22"><a href="#cb35-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-23"><a href="#cb35-23" aria-hidden="true" tabindex="-1"></a>Probability is a mathematical framework used to quantify and describe expectations of random events. Okay, so what is randomness? Where does it come from? This is a philosophical question, really. </span>
<span id="cb35-24"><a href="#cb35-24" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-25"><a href="#cb35-25" aria-hidden="true" tabindex="-1"></a>From a practical viewpoint, I find it useful to think of randomness and probability as _placeholders for incomplete knowledge_. If we have complete knowledge of something, we don't need the concepts of randomness and probability.</span>
<span id="cb35-26"><a href="#cb35-26" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-27"><a href="#cb35-27" aria-hidden="true" tabindex="-1"></a>To illustrate, when you roll a die, you don't know what number you'll get. On inspection of the die, you can see that it has six sides, numbered 1-6. If we assume that it will land on one side, and that there's equal probability of landing on each of the six sides, then we can form a basic probability model. </span>
<span id="cb35-28"><a href="#cb35-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-29"><a href="#cb35-29" aria-hidden="true" tabindex="-1"></a>| $x$ | $P(x)$ |</span>
<span id="cb35-30"><a href="#cb35-30" aria-hidden="true" tabindex="-1"></a>|---|------|</span>
<span id="cb35-31"><a href="#cb35-31" aria-hidden="true" tabindex="-1"></a>| 1 | 1/6  |</span>
<span id="cb35-32"><a href="#cb35-32" aria-hidden="true" tabindex="-1"></a>| 2 | 1/6  |</span>
<span id="cb35-33"><a href="#cb35-33" aria-hidden="true" tabindex="-1"></a>| 3 | 1/6  |</span>
<span id="cb35-34"><a href="#cb35-34" aria-hidden="true" tabindex="-1"></a>| 4 | 1/6  |</span>
<span id="cb35-35"><a href="#cb35-35" aria-hidden="true" tabindex="-1"></a>| 5 | 1/6  |</span>
<span id="cb35-36"><a href="#cb35-36" aria-hidden="true" tabindex="-1"></a>| 6 | 1/6  |</span>
<span id="cb35-37"><a href="#cb35-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-38"><a href="#cb35-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-39"><a href="#cb35-39" aria-hidden="true" tabindex="-1"></a>There are six possible outcomes with equal probability, so the probability of each is 1/6. The probabilities sum to 1. </span>
<span id="cb35-40"><a href="#cb35-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-41"><a href="#cb35-41" aria-hidden="true" tabindex="-1"></a>Where does the randomness come into this scenario? Is there randomness inherent in the process of rolling a die? I would say no. If we had complete knowledge of the system, including the shape of the die, the starting position, the angle and velocity with which it is thrown, and the shape and texture of the surface it lands on, and a good physical model of how these things interact, then we could theoretically be able to predict perfectly the number on which it will land. The trouble is, of course, that all this information is extremely difficult to collect in enough detail to allow us to make such a prediction. </span>
<span id="cb35-42"><a href="#cb35-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-43"><a href="#cb35-43" aria-hidden="true" tabindex="-1"></a>A simpler case is a shuffled deck of cards. When you shuffle a deck of cards for long enough, we'd consider the question of which card is on the top to be random. However, if we knew the starting conditions and exactly how the cards were shuffled, then there's no randomness, and no need for a  probability model. </span>
<span id="cb35-44"><a href="#cb35-44" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-45"><a href="#cb35-45" aria-hidden="true" tabindex="-1"></a>But most of the world isn't as simple as shuffling cards or even rolling a die. Our knowledge is so very limited that concepts of uncertainty, randomness, and probability are indispensable to anyone who wants to make any real sense of it.</span>
<span id="cb35-46"><a href="#cb35-46" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-47"><a href="#cb35-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-48"><a href="#cb35-48" aria-hidden="true" tabindex="-1"></a><span class="fu">## Probability as a relative frequency</span></span>
<span id="cb35-49"><a href="#cb35-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-50"><a href="#cb35-50" aria-hidden="true" tabindex="-1"></a>The naive or classical definition of probability is just a relative frequency. If $n$ is the total number of equally likely possible outcomes and if $m$ of those $n$ outcomes result in the occurrence of some given event, then the probability of that event is $m/n$. </span>
<span id="cb35-51"><a href="#cb35-51" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-52"><a href="#cb35-52" aria-hidden="true" tabindex="-1"></a>For example, if a fair die is rolled, there are $n = 6$ possible outcomes. Let the event of interest be *getting a number 4 or more*. So, the probability of this event is 3 out of 6 or $p=1/2$.</span>
<span id="cb35-53"><a href="#cb35-53" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-54"><a href="#cb35-54" aria-hidden="true" tabindex="-1"></a>The *sample space* or the set of all possible outcomes need not be finite. For example, tossing a coin until the first head appears will result in an infinite sample space.So, the probability can be viewed as a limiting or long run fraction of $m/n$ (i.e. when $n \to \infty$).</span>
<span id="cb35-55"><a href="#cb35-55" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-56"><a href="#cb35-56" aria-hidden="true" tabindex="-1"></a>When the sample space is finite and outcomes are equally likely, we can assume that classical probability will be the same as empirical probability. Note that to find the probability of a fair coin landing heads it is not necessary to toss the coin repeatedly and observe the proportion of heads. So when the number of trails is reasonably large, empirical probability may be treated as the true value.</span>
<span id="cb35-57"><a href="#cb35-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-58"><a href="#cb35-58" aria-hidden="true" tabindex="-1"></a>Subjective probability (such as expert opinion) is useful in areas such as decision theory, and its definition does not fit as a relative frequency.</span>
<span id="cb35-59"><a href="#cb35-59" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-60"><a href="#cb35-60" aria-hidden="true" tabindex="-1"></a>Probabilities can only be between $0$ and $1$ (impossible to certain).</span>
<span id="cb35-61"><a href="#cb35-61" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-62"><a href="#cb35-62" aria-hidden="true" tabindex="-1"></a><span class="fu">## Mutually exclusive events</span></span>
<span id="cb35-63"><a href="#cb35-63" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-64"><a href="#cb35-64" aria-hidden="true" tabindex="-1"></a>A German researcher <span class="co">[</span><span class="ot">@witt</span><span class="co">]</span> studied the effects of various drugs on the web-building abilities of orb-web spiders (@fig-webs). </span>
<span id="cb35-65"><a href="#cb35-65" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-66"><a href="#cb35-66" aria-hidden="true" tabindex="-1"></a><span class="al">![Some examples of webs made by spiders dosed with various intoxicants](images/webs.png)</span>{#fig-webs}</span>
<span id="cb35-67"><a href="#cb35-67" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-68"><a href="#cb35-68" aria-hidden="true" tabindex="-1"></a>Say he classified the webs into three types:</span>
<span id="cb35-69"><a href="#cb35-69" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-70"><a href="#cb35-70" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span><span class="in">`complete`</span>, <span class="in">`partial`</span>, or <span class="in">`fail`</span>.</span>
<span id="cb35-71"><a href="#cb35-71" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-72"><a href="#cb35-72" aria-hidden="true" tabindex="-1"></a>He found the relative frequencies for the three types as follows:</span>
<span id="cb35-73"><a href="#cb35-73" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-74"><a href="#cb35-74" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>$P(W=\text{complete}) = 0.2$</span>
<span id="cb35-75"><a href="#cb35-75" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-76"><a href="#cb35-76" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>$P(W=\text{partial}) = 0.3$</span>
<span id="cb35-77"><a href="#cb35-77" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-78"><a href="#cb35-78" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>$P(W=\text{fail}) = 0.5$</span>
<span id="cb35-79"><a href="#cb35-79" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-80"><a href="#cb35-80" aria-hidden="true" tabindex="-1"></a>The random variable $W$ has three possible states but any given observation can only belong to one state. You cannot observe a web that is both “partial” and “complete”, or “complete” and “fail”. These values or states are said to be mutually exclusive.</span>
<span id="cb35-81"><a href="#cb35-81" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-82"><a href="#cb35-82" aria-hidden="true" tabindex="-1"></a>For mutually exclusive events, </span>
<span id="cb35-83"><a href="#cb35-83" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-84"><a href="#cb35-84" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>The probability of any two events co-occurring is zero.</span>
<span id="cb35-85"><a href="#cb35-85" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-86"><a href="#cb35-86" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>The probability of one event or another event occurring is the sum of the two respective probabilities. </span>
<span id="cb35-87"><a href="#cb35-87" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-88"><a href="#cb35-88" aria-hidden="true" tabindex="-1"></a>For example, either a complete web or a partial web being observed is equal to the sum of $P(W = \text{complete})$ and $P(W = \text {partial})$.  That is, </span>
<span id="cb35-89"><a href="#cb35-89" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-90"><a href="#cb35-90" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-91"><a href="#cb35-91" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>The probability of any one event not occurring is the sum of those remaining is </span>
<span id="cb35-92"><a href="#cb35-92" aria-hidden="true" tabindex="-1"></a>$$P(\text{not complete})    = P(W = \text{partial or}~W \text{ = fail})= 0.3 + 0.5 = 0.8 = 1 – P(W = \text{complete}) $$</span>
<span id="cb35-93"><a href="#cb35-93" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-94"><a href="#cb35-94" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb35-95"><a href="#cb35-95" aria-hidden="true" tabindex="-1"></a><span class="fu">## Statistical independence</span></span>
<span id="cb35-96"><a href="#cb35-96" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-97"><a href="#cb35-97" aria-hidden="true" tabindex="-1"></a>If events $A$ and $B$ are statistically independent, then </span>
<span id="cb35-98"><a href="#cb35-98" aria-hidden="true" tabindex="-1"></a>$$P(A \textbf{ and } B) = P(A) \times P(B)$$</span>
<span id="cb35-99"><a href="#cb35-99" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-100"><a href="#cb35-100" aria-hidden="true" tabindex="-1"></a>If the $n$ events $A_1, A_2, \dots, A_n$  are mutually independent, then:</span>
<span id="cb35-101"><a href="#cb35-101" aria-hidden="true" tabindex="-1"></a>   $$P(A_1 \textbf{ and } A_2 \textbf{ and } \dots \textbf{ and } A_n) = P(A_1) \times  P(A_2) \times \dots \times P(A_n)$$</span>
<span id="cb35-102"><a href="#cb35-102" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-103"><a href="#cb35-103" aria-hidden="true" tabindex="-1"></a>The probability of *event A occurring given that event B has already occurred* is written in shorthand as $P(A|B)$.</span>
<span id="cb35-104"><a href="#cb35-104" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-105"><a href="#cb35-105" aria-hidden="true" tabindex="-1"></a>For example, the probability of a card you’ve drawn being a 5, given that it is a spade.  This is known as a **conditional probability**.</span>
<span id="cb35-106"><a href="#cb35-106" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-107"><a href="#cb35-107" aria-hidden="true" tabindex="-1"></a>Note that the sample space is reduced to that where B has occurred. </span>
<span id="cb35-108"><a href="#cb35-108" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-109"><a href="#cb35-109" aria-hidden="true" tabindex="-1"></a>We say that two events (A and B) are **independent** if</span>
<span id="cb35-110"><a href="#cb35-110" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-111"><a href="#cb35-111" aria-hidden="true" tabindex="-1"></a>$$P(A | B) = P(A) \textbf{ and } P(B | A) = P(B)$$</span>
<span id="cb35-112"><a href="#cb35-112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-113"><a href="#cb35-113" aria-hidden="true" tabindex="-1"></a>This means that observing event A doesn’t make event B any more or less likely, and vice versa. </span>
<span id="cb35-114"><a href="#cb35-114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-115"><a href="#cb35-115" aria-hidden="true" tabindex="-1"></a>For any two events $A$ and $B$,</span>
<span id="cb35-116"><a href="#cb35-116" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb35-117"><a href="#cb35-117" aria-hidden="true" tabindex="-1"></a>$$P(A \textbf{ and } B ) = P(A|B) \times P(B)$$</span>
<span id="cb35-118"><a href="#cb35-118" aria-hidden="true" tabindex="-1"></a>        and also</span>
<span id="cb35-119"><a href="#cb35-119" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-120"><a href="#cb35-120" aria-hidden="true" tabindex="-1"></a>$$P(A \textbf{ and } B ) = P(B|A) \times P(A)$$</span>
<span id="cb35-121"><a href="#cb35-121" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-122"><a href="#cb35-122" aria-hidden="true" tabindex="-1"></a><span class="fu">## Examples</span></span>
<span id="cb35-123"><a href="#cb35-123" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-124"><a href="#cb35-124" aria-hidden="true" tabindex="-1"></a><span class="fu">### Blood group example</span></span>
<span id="cb35-125"><a href="#cb35-125" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-126"><a href="#cb35-126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-127"><a href="#cb35-127" aria-hidden="true" tabindex="-1"></a>There are a large number of genetic-based blood-type systems that have been used for categorising blood. Two of these are the Rh system (Rh+ and Rh–) and the Kell system (K+ and K–). It is found that for any person, their blood type in any one system is independent of their blood type in any other. </span>
<span id="cb35-128"><a href="#cb35-128" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-129"><a href="#cb35-129" aria-hidden="true" tabindex="-1"></a>For Europeans in New Zealand, about 81% are Rh+ and about 8% are K+. So, we can form a Table of counts as shown in @fig-Rh:</span>
<span id="cb35-130"><a href="#cb35-130" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-131"><a href="#cb35-131" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-132"><a href="#cb35-132" aria-hidden="true" tabindex="-1"></a><span class="al">![Categorising blood types](images/RhK.png)</span>{#fig-Rh}</span>
<span id="cb35-133"><a href="#cb35-133" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-134"><a href="#cb35-134" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-135"><a href="#cb35-135" aria-hidden="true" tabindex="-1"></a>If a European New Zealander is chosen at random, what is the probability that they are (Rh+ and K+) or (Rh– and K–)?</span>
<span id="cb35-136"><a href="#cb35-136" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-137"><a href="#cb35-137" aria-hidden="true" tabindex="-1"></a>The answer is 0.0648 + 0.1748 = 0.2396 (verify).</span>
<span id="cb35-138"><a href="#cb35-138" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-139"><a href="#cb35-139" aria-hidden="true" tabindex="-1"></a>Suppose that a murder victim has a bloodstain on him with type (Rh– and K+), presumably from the assailant. What is the probability that a randomly selected person matches this type?</span>
<span id="cb35-140"><a href="#cb35-140" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-141"><a href="#cb35-141" aria-hidden="true" tabindex="-1"></a>The answer is 0.0152 (verify).</span>
<span id="cb35-142"><a href="#cb35-142" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-143"><a href="#cb35-143" aria-hidden="true" tabindex="-1"></a><span class="fu">### Beached whales example</span></span>
<span id="cb35-144"><a href="#cb35-144" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-145"><a href="#cb35-145" aria-hidden="true" tabindex="-1"></a>Assume that 0.63 is the probability of a randomly selected beached whale is a male. So the probability of a beached whale being female is $1-0.63=0.37$. Suppose that we also observe whether beached whales are juvenile or adult, and the probability of a whale being juvenile, given that it is female, is 0.8, and the probability it is juvenile, given that it is male, is 0.4.</span>
<span id="cb35-146"><a href="#cb35-146" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-147"><a href="#cb35-147" aria-hidden="true" tabindex="-1"></a>Let us denote the random variables and their outcomes as follows:</span>
<span id="cb35-148"><a href="#cb35-148" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-149"><a href="#cb35-149" aria-hidden="true" tabindex="-1"></a>*Sex* (S) = either female ($f$) or male ($m$)</span>
<span id="cb35-150"><a href="#cb35-150" aria-hidden="true" tabindex="-1"></a>   </span>
<span id="cb35-151"><a href="#cb35-151" aria-hidden="true" tabindex="-1"></a>*Age* (A) = either juvenile ($j$) or adult ($a$)</span>
<span id="cb35-152"><a href="#cb35-152" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-153"><a href="#cb35-153" aria-hidden="true" tabindex="-1"></a>We are given</span>
<span id="cb35-154"><a href="#cb35-154" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-155"><a href="#cb35-155" aria-hidden="true" tabindex="-1"></a>   $P(S=m) = 0.63$   </span>
<span id="cb35-156"><a href="#cb35-156" aria-hidden="true" tabindex="-1"></a>   $P(A=j | S=f) = 0.8$   </span>
<span id="cb35-157"><a href="#cb35-157" aria-hidden="true" tabindex="-1"></a>   $P(A=j | S=m) = 0.4$</span>
<span id="cb35-158"><a href="#cb35-158" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-159"><a href="#cb35-159" aria-hidden="true" tabindex="-1"></a><span class="fu">## Probability tree diagram</span></span>
<span id="cb35-160"><a href="#cb35-160" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-161"><a href="#cb35-161" aria-hidden="true" tabindex="-1"></a>Easy to visualise the probabilities with this representation as shown in @fig-PTD.</span>
<span id="cb35-162"><a href="#cb35-162" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-163"><a href="#cb35-163" aria-hidden="true" tabindex="-1"></a><span class="al">![Probability tree diagram](images/tree.png)</span>{#fig-PTD}</span>
<span id="cb35-164"><a href="#cb35-164" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-165"><a href="#cb35-165" aria-hidden="true" tabindex="-1"></a><span class="fu">### Rules of the Probability Tree</span></span>
<span id="cb35-166"><a href="#cb35-166" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-167"><a href="#cb35-167" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Within each level, all branches are mutually exclusive events. </span>
<span id="cb35-168"><a href="#cb35-168" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-169"><a href="#cb35-169" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>The tree covers all possibilities (i.e., the entire sample space).</span>
<span id="cb35-170"><a href="#cb35-170" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-171"><a href="#cb35-171" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>We multiply as we move along branches.</span>
<span id="cb35-172"><a href="#cb35-172" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-173"><a href="#cb35-173" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>We add when we move across branches.</span>
<span id="cb35-174"><a href="#cb35-174" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-175"><a href="#cb35-175" aria-hidden="true" tabindex="-1"></a>What is the overall probability that a whale is juvenile? That is, what is $P(A = j)$ (or $P(j)$ for short)?</span>
<span id="cb35-176"><a href="#cb35-176" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-177"><a href="#cb35-177" aria-hidden="true" tabindex="-1"></a>$$P(j) = P(m \textbf{ and } j) + P(f \textbf{ and } j)= 0.63\times 0.4+0.37 \times 0.8 = 0.548 $$</span>
<span id="cb35-178"><a href="#cb35-178" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-179"><a href="#cb35-179" aria-hidden="true" tabindex="-1"></a><span class="fu">## Bayes rule</span></span>
<span id="cb35-180"><a href="#cb35-180" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-181"><a href="#cb35-181" aria-hidden="true" tabindex="-1"></a>What is the probability of the whale being female given that it is juvenile? i.e., what is $P(f | j)$ ?</span>
<span id="cb35-182"><a href="#cb35-182" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-183"><a href="#cb35-183" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-184"><a href="#cb35-184" aria-hidden="true" tabindex="-1"></a>$P(f | j)$ doesn’t appear in the tree anywhere, so we will have to go back to first principles.</span>
<span id="cb35-185"><a href="#cb35-185" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-186"><a href="#cb35-186" aria-hidden="true" tabindex="-1"></a>we know that</span>
<span id="cb35-187"><a href="#cb35-187" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb35-188"><a href="#cb35-188" aria-hidden="true" tabindex="-1"></a>$$P(f \textbf{ and }  j)  = P(f | j) × P(j)$$ and   </span>
<span id="cb35-189"><a href="#cb35-189" aria-hidden="true" tabindex="-1"></a>$$P(f \textbf{ and }  j)  = P(j | f) × P(f)$$</span>
<span id="cb35-190"><a href="#cb35-190" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-191"><a href="#cb35-191" aria-hidden="true" tabindex="-1"></a>Thus,  $$P(f | j) × P(j) = P(j | f) × P(f)$$</span>
<span id="cb35-192"><a href="#cb35-192" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-193"><a href="#cb35-193" aria-hidden="true" tabindex="-1"></a>$$P(f | j) = \frac {P(j | f) × P(f)} {P(j)}$$</span>
<span id="cb35-194"><a href="#cb35-194" aria-hidden="true" tabindex="-1"></a>This is known as the *Bayes rule* or theorem.</span>
<span id="cb35-195"><a href="#cb35-195" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-196"><a href="#cb35-196" aria-hidden="true" tabindex="-1"></a>For our question</span>
<span id="cb35-197"><a href="#cb35-197" aria-hidden="true" tabindex="-1"></a>$$P(f | j) = \frac {P(j | f) × P(f)} {P(j)}= \frac{0.8 \times 0.37}{0.548}= 0.54$$</span>
<span id="cb35-198"><a href="#cb35-198" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-199"><a href="#cb35-199" aria-hidden="true" tabindex="-1"></a><span class="fu"># Probability Distributions</span></span>
<span id="cb35-200"><a href="#cb35-200" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-201"><a href="#cb35-201" aria-hidden="true" tabindex="-1"></a>Consider the number of eggs $(X)$ in an Adelie penguin’s nest. The values range from $1$ to $5$, each with a certain probability (or relative frequency) of occurrence.</span>
<span id="cb35-202"><a href="#cb35-202" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-203"><a href="#cb35-203" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-206"><a href="#cb35-206" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-207"><a href="#cb35-207" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-208"><a href="#cb35-208" aria-hidden="true" tabindex="-1"></a>dfm <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb35-209"><a href="#cb35-209" aria-hidden="true" tabindex="-1"></a>  <span class="at">X=</span><span class="dv">1</span><span class="sc">:</span><span class="dv">5</span>, </span>
<span id="cb35-210"><a href="#cb35-210" aria-hidden="true" tabindex="-1"></a>  <span class="at">Probability=</span><span class="fu">c</span>(<span class="fl">0.1</span>, <span class="fl">0.2</span>, <span class="fl">0.3</span>,<span class="fl">0.25</span>,<span class="fl">0.15</span>)</span>
<span id="cb35-211"><a href="#cb35-211" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb35-212"><a href="#cb35-212" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-213"><a href="#cb35-213" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(kableExtra)</span>
<span id="cb35-214"><a href="#cb35-214" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-215"><a href="#cb35-215" aria-hidden="true" tabindex="-1"></a><span class="fu">kable</span>(dfm, <span class="at">booktabs=</span><span class="cn">TRUE</span>) <span class="sc">|&gt;</span> </span>
<span id="cb35-216"><a href="#cb35-216" aria-hidden="true" tabindex="-1"></a>  <span class="fu">kable_styling</span>(<span class="at">bootstrap_options =</span> <span class="st">"striped"</span>, <span class="at">full_width =</span> F)</span>
<span id="cb35-217"><a href="#cb35-217" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-218"><a href="#cb35-218" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-219"><a href="#cb35-219" aria-hidden="true" tabindex="-1"></a>The barplot of these probabilities is shown as @fig-discr.</span>
<span id="cb35-220"><a href="#cb35-220" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-221"><a href="#cb35-221" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-224"><a href="#cb35-224" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-225"><a href="#cb35-225" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false </span></span>
<span id="cb35-226"><a href="#cb35-226" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb35-227"><a href="#cb35-227" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpmisc)</span>
<span id="cb35-228"><a href="#cb35-228" aria-hidden="true" tabindex="-1"></a><span class="fu">theme_set</span>(<span class="fu">theme_minimal</span>())</span>
<span id="cb35-229"><a href="#cb35-229" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-230"><a href="#cb35-230" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-233"><a href="#cb35-233" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-234"><a href="#cb35-234" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-235"><a href="#cb35-235" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-discr</span></span>
<span id="cb35-236"><a href="#cb35-236" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Barplot of probabilities'</span></span>
<span id="cb35-237"><a href="#cb35-237" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 3</span></span>
<span id="cb35-238"><a href="#cb35-238" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-239"><a href="#cb35-239" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-240"><a href="#cb35-240" aria-hidden="true" tabindex="-1"></a>dfm <span class="sc">|&gt;</span> </span>
<span id="cb35-241"><a href="#cb35-241" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>() <span class="sc">+</span></span>
<span id="cb35-242"><a href="#cb35-242" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(<span class="at">x=</span>X, <span class="at">y=</span><span class="st">`</span><span class="at">Probability</span><span class="st">`</span>) <span class="sc">+</span></span>
<span id="cb35-243"><a href="#cb35-243" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb35-244"><a href="#cb35-244" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span> </span>
<span id="cb35-245"><a href="#cb35-245" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_x_continuous</span>(<span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span><span class="sc">:</span><span class="dv">9</span>))<span class="sc">+</span></span>
<span id="cb35-246"><a href="#cb35-246" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="at">geom =</span> <span class="st">"table"</span>, <span class="at">label =</span> <span class="fu">list</span>(dfm), <span class="at">x=</span><span class="dv">8</span>, <span class="at">y=</span>.<span class="dv">15</span>)</span>
<span id="cb35-247"><a href="#cb35-247" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-248"><a href="#cb35-248" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-249"><a href="#cb35-249" aria-hidden="true" tabindex="-1"></a>This is an example of a theoretical probability distribution, which describes the behaviour of a data-producing system. It's a model for producing random values with specific probabilities. </span>
<span id="cb35-250"><a href="#cb35-250" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-251"><a href="#cb35-251" aria-hidden="true" tabindex="-1"></a>Note the probabilities add to $1$ because ${1,2,3,4,5}$ is a complete distribution. There are no other possibilities.</span>
<span id="cb35-252"><a href="#cb35-252" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-253"><a href="#cb35-253" aria-hidden="true" tabindex="-1"></a>Two important parameters for describing the probability distribution of a random variable $X$ are:</span>
<span id="cb35-254"><a href="#cb35-254" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-255"><a href="#cb35-255" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>The mean $\mu$ (or "expected value" $\text{E}(X)$). This is the "point of balance", that locates the distribution and puts equal mass either side.</span>
<span id="cb35-256"><a href="#cb35-256" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>The variance $\sigma_X^2$ (or $\text{Var}(X)$) is a measure of spread. It is the average squared difference from a value to the mean.</span>
<span id="cb35-257"><a href="#cb35-257" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-258"><a href="#cb35-258" aria-hidden="true" tabindex="-1"></a>For a discrete probability distribution, the population mean $\mu_X$ is simply the sum of each outcome multiplied by its probability. </span>
<span id="cb35-259"><a href="#cb35-259" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-260"><a href="#cb35-260" aria-hidden="true" tabindex="-1"></a>$$\mu_X = \text{E}(X) = \sum xP(X=x) = \sum xP(x)$$</span>
<span id="cb35-261"><a href="#cb35-261" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-262"><a href="#cb35-262" aria-hidden="true" tabindex="-1"></a>In R,</span>
<span id="cb35-263"><a href="#cb35-263" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-264"><a href="#cb35-264" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-265"><a href="#cb35-265" aria-hidden="true" tabindex="-1"></a>X <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">5</span></span>
<span id="cb35-266"><a href="#cb35-266" aria-hidden="true" tabindex="-1"></a>P <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fl">0.1</span>, <span class="fl">0.2</span>, <span class="fl">0.3</span>,<span class="fl">0.25</span>,<span class="fl">0.15</span>)</span>
<span id="cb35-267"><a href="#cb35-267" aria-hidden="true" tabindex="-1"></a>(Mean <span class="ot">&lt;-</span> <span class="fu">sum</span>(X<span class="sc">*</span>P))</span>
<span id="cb35-268"><a href="#cb35-268" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-269"><a href="#cb35-269" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-270"><a href="#cb35-270" aria-hidden="true" tabindex="-1"></a>The population variance is given by</span>
<span id="cb35-271"><a href="#cb35-271" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-272"><a href="#cb35-272" aria-hidden="true" tabindex="-1"></a>$$\sigma_X^2 = \text{Var}(X) = \sum (x-\mu_X)^2 P(x)$$</span>
<span id="cb35-273"><a href="#cb35-273" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-274"><a href="#cb35-274" aria-hidden="true" tabindex="-1"></a>The population SD is simply the square-root of the variance.</span>
<span id="cb35-275"><a href="#cb35-275" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-276"><a href="#cb35-276" aria-hidden="true" tabindex="-1"></a>In R,</span>
<span id="cb35-277"><a href="#cb35-277" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-278"><a href="#cb35-278" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-279"><a href="#cb35-279" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-280"><a href="#cb35-280" aria-hidden="true" tabindex="-1"></a>X <span class="ot">&lt;-</span> <span class="dv">1</span><span class="sc">:</span><span class="dv">5</span></span>
<span id="cb35-281"><a href="#cb35-281" aria-hidden="true" tabindex="-1"></a>P <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fl">0.1</span>, <span class="fl">0.2</span>, <span class="fl">0.3</span>,<span class="fl">0.25</span>,<span class="fl">0.15</span>)</span>
<span id="cb35-282"><a href="#cb35-282" aria-hidden="true" tabindex="-1"></a>Mean <span class="ot">&lt;-</span> <span class="fu">sum</span>(X <span class="sc">*</span> P)</span>
<span id="cb35-283"><a href="#cb35-283" aria-hidden="true" tabindex="-1"></a>(Variance <span class="ot">&lt;-</span> <span class="fu">sum</span>((X<span class="sc">-</span>Mean)<span class="sc">^</span><span class="dv">2</span><span class="sc">*</span>P))</span>
<span id="cb35-284"><a href="#cb35-284" aria-hidden="true" tabindex="-1"></a>(SD <span class="ot">&lt;-</span> <span class="fu">sqrt</span>(Variance))</span>
<span id="cb35-285"><a href="#cb35-285" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-286"><a href="#cb35-286" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-287"><a href="#cb35-287" aria-hidden="true" tabindex="-1"></a><span class="fu">## Probability Mass/Density Functions</span></span>
<span id="cb35-288"><a href="#cb35-288" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-289"><a href="#cb35-289" aria-hidden="true" tabindex="-1"></a>The above example of the distribution of numbers of eggs of penguins is an odd case, because we've simply listed the outcomes and probabilities. More often, we use a known probability distribution where the probabilities of any value of $X$ may be calculated using a Probability Mass Function (PMF, for discrete distributions) or Probability Density Function (PDF, for continuous distributions). A PMF or PDF is a formula that can be used to calculate the probability associated with any value; that is, $\text{P}(X=x)$, the probability that the random variable $X$ will take a particular value $x$. These probabilities arise from making particular assumptions about how the values arise. </span>
<span id="cb35-290"><a href="#cb35-290" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-291"><a href="#cb35-291" aria-hidden="true" tabindex="-1"></a>The remainder of this chapter will discuss some of the more common probability distributions. </span>
<span id="cb35-292"><a href="#cb35-292" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-293"><a href="#cb35-293" aria-hidden="true" tabindex="-1"></a><span class="fu">## Discrete probability distributions</span></span>
<span id="cb35-294"><a href="#cb35-294" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-295"><a href="#cb35-295" aria-hidden="true" tabindex="-1"></a>Discrete probability distributions are those that can produce a limited set of values, usually non-negative integers <span class="sc">\{</span>0, 1, 2, 3, ...<span class="sc">\}</span>. The two most useful ones are the Binomial and the Poisson. </span>
<span id="cb35-296"><a href="#cb35-296" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-297"><a href="#cb35-297" aria-hidden="true" tabindex="-1"></a><span class="fu">### Binomial distribution</span></span>
<span id="cb35-298"><a href="#cb35-298" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-299"><a href="#cb35-299" aria-hidden="true" tabindex="-1"></a>Let $X$ be the number of heads when two coins are tossed. The possible outcomes are shown in @fig-twocoins.</span>
<span id="cb35-300"><a href="#cb35-300" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-301"><a href="#cb35-301" aria-hidden="true" tabindex="-1"></a><span class="al">![Possible outcomes for two coin tosses](images/binom.png)</span>{#fig-twocoins}</span>
<span id="cb35-302"><a href="#cb35-302" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-303"><a href="#cb35-303" aria-hidden="true" tabindex="-1"></a>In general, when there only two possible outcomes say  *success* or *failure* ($1$ or $0$; Yes or No; On or Off; Heads or Tails; etc.), we deal with a  *binary* or **Bernoulli** random variable.</span>
<span id="cb35-304"><a href="#cb35-304" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-305"><a href="#cb35-305" aria-hidden="true" tabindex="-1"></a>The count of the number of *successes* $X$ out of a fixed total of $n$ *independent* trials (in the above coin tossing example, there were $n = 2$ trials) follows the binomial distribution.</span>
<span id="cb35-306"><a href="#cb35-306" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-307"><a href="#cb35-307" aria-hidden="true" tabindex="-1"></a>In other words, $X \sim Bin(n, p)$, where $p$ the probability of a success. The binomial probability mass function $P(X=x)$ or $P(x)$ is given by</span>
<span id="cb35-308"><a href="#cb35-308" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-309"><a href="#cb35-309" aria-hidden="true" tabindex="-1"></a>$$P(x)={n \choose x}p^{x}(1-p)^{n-x}$$</span>
<span id="cb35-310"><a href="#cb35-310" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-311"><a href="#cb35-311" aria-hidden="true" tabindex="-1"></a>It is easy to plot these probabilities in R. For $n=10$, $p=0.3$, the binomial probabilities $P(x)~~~ x=0,1,2, \dots, 10$ are plotted in @fig-dbin.</span>
<span id="cb35-312"><a href="#cb35-312" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-315"><a href="#cb35-315" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-316"><a href="#cb35-316" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-317"><a href="#cb35-317" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-dbin</span></span>
<span id="cb35-318"><a href="#cb35-318" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Binomial probabilities'</span></span>
<span id="cb35-319"><a href="#cb35-319" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-320"><a href="#cb35-320" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 4</span></span>
<span id="cb35-321"><a href="#cb35-321" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb35-322"><a href="#cb35-322" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-323"><a href="#cb35-323" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb35-324"><a href="#cb35-324" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpmisc)</span>
<span id="cb35-325"><a href="#cb35-325" aria-hidden="true" tabindex="-1"></a><span class="fu">data.frame</span>(<span class="at">x =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">10</span>, </span>
<span id="cb35-326"><a href="#cb35-326" aria-hidden="true" tabindex="-1"></a>           <span class="at">probability =</span> <span class="fu">dbinom</span>(<span class="at">x =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">10</span>, <span class="at">size =</span> <span class="dv">10</span>, <span class="at">prob =</span> <span class="fl">0.3</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb35-327"><a href="#cb35-327" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">x=</span><span class="fu">as.factor</span>(x)) <span class="ot">-&gt;</span>dfm</span>
<span id="cb35-328"><a href="#cb35-328" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> probability)) <span class="sc">+</span> <span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb35-329"><a href="#cb35-329" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="at">geom =</span> <span class="st">"table"</span>, <span class="at">label =</span> <span class="fu">list</span>(dfm), <span class="at">x=</span><span class="dv">11</span>, <span class="at">y=</span>.<span class="dv">05</span>)</span>
<span id="cb35-330"><a href="#cb35-330" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-331"><a href="#cb35-331" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-332"><a href="#cb35-332" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-333"><a href="#cb35-333" aria-hidden="true" tabindex="-1"></a>Example: A microbiologist conducts an experiment to create a recombinant strain of bacteria that is resistant to penicillin. She plates out the bacteria on a plate, and picks out 10 colonies. She knows that the probability of successfully creating a recombinant is 0.15.</span>
<span id="cb35-334"><a href="#cb35-334" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-335"><a href="#cb35-335" aria-hidden="true" tabindex="-1"></a>What is the probability that if she mixes all 10 colonies in a growth medium with penicillin, something  (anything) will grow?</span>
<span id="cb35-336"><a href="#cb35-336" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-337"><a href="#cb35-337" aria-hidden="true" tabindex="-1"></a>Given:</span>
<span id="cb35-338"><a href="#cb35-338" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-339"><a href="#cb35-339" aria-hidden="true" tabindex="-1"></a>$X \sim Bin(n = 10, p = 0.15)$. What is $P(x &gt; 0)$? </span>
<span id="cb35-340"><a href="#cb35-340" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-341"><a href="#cb35-341" aria-hidden="true" tabindex="-1"></a>Note  $P(x &gt; 0)=1-P(x = 0)$.  So in R, compute this as follows:</span>
<span id="cb35-342"><a href="#cb35-342" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-343"><a href="#cb35-343" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-344"><a href="#cb35-344" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span> <span class="sc">-</span> <span class="fu">dbinom</span>(<span class="at">x=</span><span class="dv">0</span>, <span class="at">size=</span><span class="dv">10</span>, <span class="at">prob=</span>.<span class="dv">15</span>)</span>
<span id="cb35-345"><a href="#cb35-345" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-346"><a href="#cb35-346" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-347"><a href="#cb35-347" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-348"><a href="#cb35-348" aria-hidden="true" tabindex="-1"></a>or</span>
<span id="cb35-349"><a href="#cb35-349" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-350"><a href="#cb35-350" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-351"><a href="#cb35-351" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span><span class="sc">-</span><span class="fu">pbinom</span>(<span class="at">q=</span><span class="dv">0</span>, <span class="at">size=</span><span class="dv">10</span>, <span class="at">prob=</span>.<span class="dv">15</span>)</span>
<span id="cb35-352"><a href="#cb35-352" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-353"><a href="#cb35-353" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-354"><a href="#cb35-354" aria-hidden="true" tabindex="-1"></a>Note that function <span class="in">`pbinom(k,size=n,prob=p)`</span> gives the cumulative probabilities up to and including the quantile $k$ (which is equal to zero in our example). </span>
<span id="cb35-355"><a href="#cb35-355" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-356"><a href="#cb35-356" aria-hidden="true" tabindex="-1"></a>$$P(X\leq k)=\sum _{i=0}^{k}{n \choose x}p^{x}(1-p)^{n-x}$$</span>
<span id="cb35-357"><a href="#cb35-357" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-358"><a href="#cb35-358" aria-hidden="true" tabindex="-1"></a>The mean and variance of the binomial random variable is given by</span>
<span id="cb35-359"><a href="#cb35-359" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-360"><a href="#cb35-360" aria-hidden="true" tabindex="-1"></a>$$\mu_X=np~~~~ \sigma^2_X=np(1-p)$$</span>
<span id="cb35-361"><a href="#cb35-361" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-362"><a href="#cb35-362" aria-hidden="true" tabindex="-1"></a>So the expected number of recombinant strain of bacteria would be</span>
<span id="cb35-363"><a href="#cb35-363" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-364"><a href="#cb35-364" aria-hidden="true" tabindex="-1"></a>$$\mu_X=np=10*0.15=1.5$$</span>
<span id="cb35-365"><a href="#cb35-365" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-366"><a href="#cb35-366" aria-hidden="true" tabindex="-1"></a>with SD</span>
<span id="cb35-367"><a href="#cb35-367" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-368"><a href="#cb35-368" aria-hidden="true" tabindex="-1"></a>$$\sigma_X=\sqrt {np(1-p)}=1.129159$$</span>
<span id="cb35-369"><a href="#cb35-369" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-370"><a href="#cb35-370" aria-hidden="true" tabindex="-1"></a><span class="fu">### Poisson distribution</span></span>
<span id="cb35-371"><a href="#cb35-371" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-372"><a href="#cb35-372" aria-hidden="true" tabindex="-1"></a>The Poisson distribution is used to obtain the probabilities of counts of relatively rare events or outcomes that occur independently in space or time. </span>
<span id="cb35-373"><a href="#cb35-373" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-374"><a href="#cb35-374" aria-hidden="true" tabindex="-1"></a>Some Examples:</span>
<span id="cb35-375"><a href="#cb35-375" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-376"><a href="#cb35-376" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>The number of snails in a quadrat ($1~m^2$)</span>
<span id="cb35-377"><a href="#cb35-377" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Fish counts in a visual transect (25 x 5 m)</span>
<span id="cb35-378"><a href="#cb35-378" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Bacterial colonies in 2 litres of milk</span>
<span id="cb35-379"><a href="#cb35-379" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-380"><a href="#cb35-380" aria-hidden="true" tabindex="-1"></a>The random variable $X$,  the number of occurrences (count), often follows the Poisson distribution whose probability function is given by</span>
<span id="cb35-381"><a href="#cb35-381" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-382"><a href="#cb35-382" aria-hidden="true" tabindex="-1"></a>$$\Pr(x)= \frac{\lambda^x e^{-\lambda}}{x!}~~~ x=0,1,2,\dots, \infty$$</span>
<span id="cb35-383"><a href="#cb35-383" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-384"><a href="#cb35-384" aria-hidden="true" tabindex="-1"></a>The parameter $\lambda$ is the mean  which is also equal to the variance.</span>
<span id="cb35-385"><a href="#cb35-385" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-386"><a href="#cb35-386" aria-hidden="true" tabindex="-1"></a>$$\mu_X=\lambda~~~~ \sigma^2_X=\lambda$$</span>
<span id="cb35-387"><a href="#cb35-387" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-388"><a href="#cb35-388" aria-hidden="true" tabindex="-1"></a>Main assumptions:</span>
<span id="cb35-389"><a href="#cb35-389" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-390"><a href="#cb35-390" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>The events occur at a constant average rate of $\lambda$ per unit time or space.</span>
<span id="cb35-391"><a href="#cb35-391" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-392"><a href="#cb35-392" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Occurrences are independent of one another as well as they do not happen at exactly the same unit time or space.</span>
<span id="cb35-393"><a href="#cb35-393" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-394"><a href="#cb35-394" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-395"><a href="#cb35-395" aria-hidden="true" tabindex="-1"></a>Example: Consider the number of changes that accumulate along a stretch of a neutrally evolving gene over a given period of time.</span>
<span id="cb35-396"><a href="#cb35-396" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-397"><a href="#cb35-397" aria-hidden="true" tabindex="-1"></a>This is a Poisson random variable with a population mean of $\lambda=kt$, where $k$ is the number of mutations per generation, and $t$ is the time in generations that has elapsed.</span>
<span id="cb35-398"><a href="#cb35-398" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-399"><a href="#cb35-399" aria-hidden="true" tabindex="-1"></a>Assume that $k = 1\times10^{-4}$ and $t = 500$.</span>
<span id="cb35-400"><a href="#cb35-400" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-401"><a href="#cb35-401" aria-hidden="true" tabindex="-1"></a>For  $\lambda=kt=0.05$, the Poisson probabilities are shown in @fig-dpoi.</span>
<span id="cb35-402"><a href="#cb35-402" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-405"><a href="#cb35-405" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-406"><a href="#cb35-406" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-407"><a href="#cb35-407" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-dpoi</span></span>
<span id="cb35-408"><a href="#cb35-408" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Poisson probabilities'</span></span>
<span id="cb35-409"><a href="#cb35-409" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 4</span></span>
<span id="cb35-410"><a href="#cb35-410" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-411"><a href="#cb35-411" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb35-412"><a href="#cb35-412" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpmisc)</span>
<span id="cb35-413"><a href="#cb35-413" aria-hidden="true" tabindex="-1"></a><span class="fu">data.frame</span>(<span class="at">x =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">5</span>, </span>
<span id="cb35-414"><a href="#cb35-414" aria-hidden="true" tabindex="-1"></a>           <span class="at">probability =</span> <span class="fu">dpois</span>(<span class="at">x =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">5</span>, <span class="at">lambda=</span><span class="fl">0.05</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb35-415"><a href="#cb35-415" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">x=</span><span class="fu">as.factor</span>(x), <span class="at">probability =</span> <span class="fu">round</span>(probability,<span class="dv">4</span>)) <span class="ot">-&gt;</span>dfm</span>
<span id="cb35-416"><a href="#cb35-416" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> probability)) <span class="sc">+</span> <span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb35-417"><a href="#cb35-417" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="at">geom =</span> <span class="st">"table"</span>, <span class="at">label =</span> <span class="fu">list</span>(dfm), <span class="at">x=</span><span class="dv">5</span>, <span class="at">y=</span>.<span class="dv">25</span>)</span>
<span id="cb35-418"><a href="#cb35-418" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-419"><a href="#cb35-419" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-420"><a href="#cb35-420" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-421"><a href="#cb35-421" aria-hidden="true" tabindex="-1"></a>What is the probability that at least one mutation has occurred over this period?</span>
<span id="cb35-422"><a href="#cb35-422" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-423"><a href="#cb35-423" aria-hidden="true" tabindex="-1"></a>$P(x &gt; 0)=1-P(x = 0)$ is found in R as follows:</span>
<span id="cb35-424"><a href="#cb35-424" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-427"><a href="#cb35-427" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-428"><a href="#cb35-428" aria-hidden="true" tabindex="-1"></a><span class="dv">1</span> <span class="sc">-</span> <span class="fu">dpois</span>(<span class="at">x=</span><span class="dv">0</span>, <span class="at">lambda=</span><span class="fl">0.05</span>)</span>
<span id="cb35-429"><a href="#cb35-429" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-430"><a href="#cb35-430" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-431"><a href="#cb35-431" aria-hidden="true" tabindex="-1"></a>Another example: The number of parasites in a host is a Poisson random variable with mean 1.5. What is the probability that there will be exactly one parasite in the host?  Examine @fig-parasite for the answer.</span>
<span id="cb35-432"><a href="#cb35-432" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-435"><a href="#cb35-435" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-436"><a href="#cb35-436" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-437"><a href="#cb35-437" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-parasite</span></span>
<span id="cb35-438"><a href="#cb35-438" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Parasites distribution'</span></span>
<span id="cb35-439"><a href="#cb35-439" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 4</span></span>
<span id="cb35-440"><a href="#cb35-440" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-441"><a href="#cb35-441" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-442"><a href="#cb35-442" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb35-443"><a href="#cb35-443" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggpmisc)</span>
<span id="cb35-444"><a href="#cb35-444" aria-hidden="true" tabindex="-1"></a><span class="fu">data.frame</span>(<span class="at">x =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">10</span>, </span>
<span id="cb35-445"><a href="#cb35-445" aria-hidden="true" tabindex="-1"></a>           <span class="at">probability =</span> <span class="fu">dpois</span>(<span class="at">x =</span> <span class="dv">0</span><span class="sc">:</span><span class="dv">10</span>, <span class="at">lambda=</span><span class="fl">1.5</span>)) <span class="sc">%&gt;%</span></span>
<span id="cb35-446"><a href="#cb35-446" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">x=</span><span class="fu">as.factor</span>(x), </span>
<span id="cb35-447"><a href="#cb35-447" aria-hidden="true" tabindex="-1"></a>         <span class="at">probability =</span> <span class="fu">round</span>(probability,<span class="dv">4</span>),</span>
<span id="cb35-448"><a href="#cb35-448" aria-hidden="true" tabindex="-1"></a>         <span class="at">parasites =</span> <span class="fu">ifelse</span>(x <span class="sc">==</span> <span class="dv">1</span>, <span class="st">"1"</span>, <span class="st">"Not 1"</span>)) <span class="ot">-&gt;</span>dfm</span>
<span id="cb35-449"><a href="#cb35-449" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> probability, <span class="at">fill =</span> parasites)) <span class="sc">+</span> </span>
<span id="cb35-450"><a href="#cb35-450" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_col</span>()<span class="sc">+</span></span>
<span id="cb35-451"><a href="#cb35-451" aria-hidden="true" tabindex="-1"></a>  <span class="fu">annotate</span>(<span class="at">geom =</span> <span class="st">"table"</span>, <span class="at">label =</span> <span class="fu">list</span>(dfm[,<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>]), <span class="at">x=</span><span class="dv">10</span>, <span class="at">y=</span>.<span class="dv">3</span>)</span>
<span id="cb35-452"><a href="#cb35-452" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-453"><a href="#cb35-453" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-454"><a href="#cb35-454" aria-hidden="true" tabindex="-1"></a><span class="fu">## Continuous probability distributions</span></span>
<span id="cb35-455"><a href="#cb35-455" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-456"><a href="#cb35-456" aria-hidden="true" tabindex="-1"></a>A discrete random variable takes values which are simply points on a real line.  In other words, there is an inherent discontinuity in the values a discrete random variable can take.</span>
<span id="cb35-457"><a href="#cb35-457" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-458"><a href="#cb35-458" aria-hidden="true" tabindex="-1"></a>If a random variable, $X$, can take any value (i.e., not just integers) in some interval of the real line, it is called a *continuous* random variable.  Examples include variables such as height, weight, length, percentage protein etc.</span>
<span id="cb35-459"><a href="#cb35-459" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-460"><a href="#cb35-460" aria-hidden="true" tabindex="-1"></a>For a discrete random variable $X$, the associated probabilities $P(X=x)$ are also just points or masses, and hence the probability function $P(x)$ is also called as the probability mass function (*pmf*).</span>
<span id="cb35-461"><a href="#cb35-461" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-462"><a href="#cb35-462" aria-hidden="true" tabindex="-1"></a>For continuous random variables, probabilities can be computed when the variable falls in an interval such as $5$ to $15$, but not when it takes a fixed value such as $10$ (which is equal to zero).</span>
<span id="cb35-463"><a href="#cb35-463" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-464"><a href="#cb35-464" aria-hidden="true" tabindex="-1"></a>The probability density function (pdf) gives the probabilities as an area under the curve.</span>
<span id="cb35-465"><a href="#cb35-465" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-466"><a href="#cb35-466" aria-hidden="true" tabindex="-1"></a>For example, consider a random proportion $(X)$ between $0$ and $1$.  Here $X$ follows a (standard) continuous uniform distribution whose (probability) density function $f(x)$ is defined as follows:</span>
<span id="cb35-467"><a href="#cb35-467" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-468"><a href="#cb35-468" aria-hidden="true" tabindex="-1"></a>$$f(x)=\begin{cases}{1}~~~\mathrm {for} \ 0\leq x\leq 1,\\[9pt]0~~~\mathrm {for} \ x&lt;0\ \mathrm {or} \ x&gt;1\end{cases}$$</span>
<span id="cb35-469"><a href="#cb35-469" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-470"><a href="#cb35-470" aria-hidden="true" tabindex="-1"></a>This constant density function is very simple; see @fig-constdens. </span>
<span id="cb35-471"><a href="#cb35-471" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-474"><a href="#cb35-474" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-475"><a href="#cb35-475" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-constdens</span></span>
<span id="cb35-476"><a href="#cb35-476" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: "Constant density between 0 and 1"</span></span>
<span id="cb35-477"><a href="#cb35-477" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-478"><a href="#cb35-478" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-479"><a href="#cb35-479" aria-hidden="true" tabindex="-1"></a><span class="fu">data.frame</span>(<span class="at">x =</span> <span class="fu">c</span>(<span class="sc">-</span>.<span class="dv">5</span>, <span class="fl">1.5</span>)) <span class="sc">|&gt;</span> </span>
<span id="cb35-480"><a href="#cb35-480" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>() <span class="sc">+</span> </span>
<span id="cb35-481"><a href="#cb35-481" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(x) <span class="sc">+</span></span>
<span id="cb35-482"><a href="#cb35-482" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(</span>
<span id="cb35-483"><a href="#cb35-483" aria-hidden="true" tabindex="-1"></a>    <span class="at">fun =</span> dunif, </span>
<span id="cb35-484"><a href="#cb35-484" aria-hidden="true" tabindex="-1"></a>    <span class="at">n =</span> <span class="dv">10001</span>, </span>
<span id="cb35-485"><a href="#cb35-485" aria-hidden="true" tabindex="-1"></a>    <span class="at">args =</span> <span class="fu">list</span>(<span class="at">min =</span> <span class="dv">0</span>, <span class="at">max =</span> <span class="dv">1</span>)</span>
<span id="cb35-486"><a href="#cb35-486" aria-hidden="true" tabindex="-1"></a>    ) <span class="sc">+</span></span>
<span id="cb35-487"><a href="#cb35-487" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ylab</span>(<span class="st">"f(x)"</span>) <span class="sc">+</span></span>
<span id="cb35-488"><a href="#cb35-488" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()</span>
<span id="cb35-489"><a href="#cb35-489" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-490"><a href="#cb35-490" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-491"><a href="#cb35-491" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-492"><a href="#cb35-492" aria-hidden="true" tabindex="-1"></a>The cumulative distribution function, CDF,  $F(x)$ gives the left tail area or probability up to $x$.  This is probability is found as</span>
<span id="cb35-493"><a href="#cb35-493" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-494"><a href="#cb35-494" aria-hidden="true" tabindex="-1"></a>$$F_{X}(x)=\int _{-\infty }^{x}f_{X}(t)\,dt$$</span>
<span id="cb35-495"><a href="#cb35-495" aria-hidden="true" tabindex="-1"></a>The relationship between the density function $f(x)$ and the distribution function $F(x)$ is given by the *Fundamental Theorem of Calculus*.</span>
<span id="cb35-496"><a href="#cb35-496" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-497"><a href="#cb35-497" aria-hidden="true" tabindex="-1"></a>$$f(x)={dF(x) \over dx}$$</span>
<span id="cb35-498"><a href="#cb35-498" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-499"><a href="#cb35-499" aria-hidden="true" tabindex="-1"></a>The total area under the pdf curve is $1$. The probability of obtaining a value between two points ($a$ and $b$) is the area under the pdf curve between those two points.  This probability is given by $F(b)-F(a)$.</span>
<span id="cb35-500"><a href="#cb35-500" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-501"><a href="#cb35-501" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-502"><a href="#cb35-502" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-503"><a href="#cb35-503" aria-hidden="true" tabindex="-1"></a>For the uniform distribution $U(0,1)$, $f(x)=1$. So,</span>
<span id="cb35-504"><a href="#cb35-504" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-505"><a href="#cb35-505" aria-hidden="true" tabindex="-1"></a>$$F_{X}(x)=\int _{-\infty }^{x}\,dt=x$$</span>
<span id="cb35-506"><a href="#cb35-506" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-507"><a href="#cb35-507" aria-hidden="true" tabindex="-1"></a>For example, the probability of a randomly drawn fraction from the interval $<span class="co">[</span><span class="ot">0,1</span><span class="co">]</span>$ to fall below $x=0.5$ is 50%.</span>
<span id="cb35-508"><a href="#cb35-508" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-509"><a href="#cb35-509" aria-hidden="true" tabindex="-1"></a>The probability of a random fraction falling between $a=0.2$ and $b=0.8$ is</span>
<span id="cb35-510"><a href="#cb35-510" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-511"><a href="#cb35-511" aria-hidden="true" tabindex="-1"></a>$$F(b)-F(a)=0.8-0.2=0.6$$</span>
<span id="cb35-512"><a href="#cb35-512" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-513"><a href="#cb35-513" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-514"><a href="#cb35-514" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-515"><a href="#cb35-515" aria-hidden="true" tabindex="-1"></a><span class="fu">### Normal distribution</span></span>
<span id="cb35-516"><a href="#cb35-516" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-517"><a href="#cb35-517" aria-hidden="true" tabindex="-1"></a>The Gaussian or normal distribution is parameterised in terms of the mean $\mu$ and the variance $\sigma ^{2}$ and its density function is given by</span>
<span id="cb35-518"><a href="#cb35-518" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-519"><a href="#cb35-519" aria-hidden="true" tabindex="-1"></a>$$f(x)={\frac {1}{\sigma {\sqrt {2\pi }}}}e^{-{\frac {1}{2}}\left({\frac {x-\mu }{\sigma }}\right)^{2}}$$</span>
<span id="cb35-520"><a href="#cb35-520" aria-hidden="true" tabindex="-1"></a>If $X \sim N(\mu, \sigma)$, its Z-score is calculated from $X$ by subtracting $\mu$ and dividing by the standard deviation $\sigma$.</span>
<span id="cb35-521"><a href="#cb35-521" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-522"><a href="#cb35-522" aria-hidden="true" tabindex="-1"></a>$$Z={\frac {X-\mu }{\sigma }}$$ </span>
<span id="cb35-523"><a href="#cb35-523" aria-hidden="true" tabindex="-1"></a>The random variable $Z \sim N(0,1)$.  Its pdf is given by</span>
<span id="cb35-524"><a href="#cb35-524" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-525"><a href="#cb35-525" aria-hidden="true" tabindex="-1"></a>$$f(z)={\frac {1}{ {\sqrt {2\pi }}}}e^{-{\frac {1}{2}}z^{2}}$$</span>
<span id="cb35-526"><a href="#cb35-526" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-527"><a href="#cb35-527" aria-hidden="true" tabindex="-1"></a>We often deal with the Standard normal because the *symmetric* bell shape of the normal distribution remains the same for all $\mu$ and $\sigma$; see @fig-nrm.</span>
<span id="cb35-528"><a href="#cb35-528" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-531"><a href="#cb35-531" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-532"><a href="#cb35-532" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-533"><a href="#cb35-533" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-nrm</span></span>
<span id="cb35-534"><a href="#cb35-534" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Standard Normal Density'</span></span>
<span id="cb35-535"><a href="#cb35-535" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 5</span></span>
<span id="cb35-536"><a href="#cb35-536" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-537"><a href="#cb35-537" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-538"><a href="#cb35-538" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb35-539"><a href="#cb35-539" aria-hidden="true" tabindex="-1"></a>p <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="dv">4</span>, <span class="dv">4</span>)), <span class="fu">aes</span>(<span class="at">x =</span> x)) <span class="sc">+</span> </span>
<span id="cb35-540"><a href="#cb35-540" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">"standard normal deviate, z"</span>)</span>
<span id="cb35-541"><a href="#cb35-541" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-542"><a href="#cb35-542" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> p <span class="sc">+</span> </span>
<span id="cb35-543"><a href="#cb35-543" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dnorm, </span>
<span id="cb35-544"><a href="#cb35-544" aria-hidden="true" tabindex="-1"></a>                <span class="at">args=</span><span class="fu">list</span>(<span class="at">mean=</span><span class="dv">0</span>, <span class="at">sd=</span><span class="dv">1</span>), </span>
<span id="cb35-545"><a href="#cb35-545" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom =</span> <span class="st">"area"</span>) <span class="sc">+</span> </span>
<span id="cb35-546"><a href="#cb35-546" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">0</span>) <span class="sc">+</span> </span>
<span id="cb35-547"><a href="#cb35-547" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Standard Normal Density"</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"f(x)"</span>)</span>
<span id="cb35-548"><a href="#cb35-548" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-549"><a href="#cb35-549" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> p <span class="sc">+</span> </span>
<span id="cb35-550"><a href="#cb35-550" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> pnorm, <span class="at">args=</span><span class="fu">list</span>(<span class="at">mean=</span><span class="dv">0</span>, <span class="at">sd=</span><span class="dv">1</span>), <span class="at">geom =</span> <span class="st">"line"</span>, <span class="at">lwd=</span><span class="fl">1.02</span>) <span class="sc">+</span> </span>
<span id="cb35-551"><a href="#cb35-551" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept =</span> <span class="dv">0</span>, <span class="at">alpha=</span>.<span class="dv">5</span>) <span class="sc">+</span> </span>
<span id="cb35-552"><a href="#cb35-552" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Cumulative Standard Normal Density"</span>) <span class="sc">+</span></span>
<span id="cb35-553"><a href="#cb35-553" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_hline</span>(<span class="at">yintercept =</span> <span class="fl">0.5</span>, <span class="at">alpha=</span>.<span class="dv">5</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">"F(x)"</span>)</span>
<span id="cb35-554"><a href="#cb35-554" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-555"><a href="#cb35-555" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(patchwork)</span>
<span id="cb35-556"><a href="#cb35-556" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">/</span>p2  <span class="sc">&amp;</span> <span class="fu">theme_minimal</span>()</span>
<span id="cb35-557"><a href="#cb35-557" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-558"><a href="#cb35-558" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-559"><a href="#cb35-559" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-560"><a href="#cb35-560" aria-hidden="true" tabindex="-1"></a>Example: The weight of an individual of *Amphibola crenata* (snail) is normally distributed with a mean of $40g$ and variance of $20g^2$ as shown in @fig-nrmeg.  </span>
<span id="cb35-561"><a href="#cb35-561" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-564"><a href="#cb35-564" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-565"><a href="#cb35-565" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-566"><a href="#cb35-566" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-nrmeg</span></span>
<span id="cb35-567"><a href="#cb35-567" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Distribution of snail weight'</span></span>
<span id="cb35-568"><a href="#cb35-568" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 3</span></span>
<span id="cb35-569"><a href="#cb35-569" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 4</span></span>
<span id="cb35-570"><a href="#cb35-570" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-571"><a href="#cb35-571" aria-hidden="true" tabindex="-1"></a>dfs <span class="ot">&lt;-</span> <span class="fu">tibble</span>(</span>
<span id="cb35-572"><a href="#cb35-572" aria-hidden="true" tabindex="-1"></a>  <span class="at">x=</span><span class="fu">seq</span>(<span class="dv">20</span>, <span class="dv">60</span>, <span class="at">length=</span><span class="dv">1000</span>), </span>
<span id="cb35-573"><a href="#cb35-573" aria-hidden="true" tabindex="-1"></a>  <span class="st">`</span><span class="at">f(x)</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">dnorm</span>(x, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>))</span>
<span id="cb35-574"><a href="#cb35-574" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb35-575"><a href="#cb35-575" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-576"><a href="#cb35-576" aria-hidden="true" tabindex="-1"></a>ps <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfs) <span class="sc">+</span> <span class="fu">aes</span>(<span class="at">x =</span> x, <span class="at">y =</span> <span class="st">`</span><span class="at">f(x)</span><span class="st">`</span>) <span class="sc">+</span> </span>
<span id="cb35-577"><a href="#cb35-577" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">fill=</span><span class="st">"gray"</span>) <span class="sc">+</span></span>
<span id="cb35-578"><a href="#cb35-578" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_vline</span>(<span class="at">xintercept=</span><span class="dv">40</span>) </span>
<span id="cb35-579"><a href="#cb35-579" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-580"><a href="#cb35-580" aria-hidden="true" tabindex="-1"></a>ps</span>
<span id="cb35-581"><a href="#cb35-581" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-582"><a href="#cb35-582" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-583"><a href="#cb35-583" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-584"><a href="#cb35-584" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-585"><a href="#cb35-585" aria-hidden="true" tabindex="-1"></a>What is the probability of getting a snail that weighs between $35g$ and $50g$? In R, the function <span class="in">`pnorm()`</span> gives the CDF.</span>
<span id="cb35-586"><a href="#cb35-586" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-587"><a href="#cb35-587" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-588"><a href="#cb35-588" aria-hidden="true" tabindex="-1"></a><span class="fu">pnorm</span>(<span class="dv">50</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>)) <span class="sc">-</span> </span>
<span id="cb35-589"><a href="#cb35-589" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pnorm</span>(<span class="dv">35</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>)) </span>
<span id="cb35-590"><a href="#cb35-590" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-591"><a href="#cb35-591" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-592"><a href="#cb35-592" aria-hidden="true" tabindex="-1"></a>@fig-snail illustrates the computation.</span>
<span id="cb35-593"><a href="#cb35-593" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-596"><a href="#cb35-596" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-597"><a href="#cb35-597" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-snail</span></span>
<span id="cb35-598"><a href="#cb35-598" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-599"><a href="#cb35-599" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Probability of snail weight being between 35 g and 50 g'</span></span>
<span id="cb35-600"><a href="#cb35-600" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 3</span></span>
<span id="cb35-601"><a href="#cb35-601" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 4</span></span>
<span id="cb35-602"><a href="#cb35-602" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-603"><a href="#cb35-603" aria-hidden="true" tabindex="-1"></a>ps <span class="sc">+</span> </span>
<span id="cb35-604"><a href="#cb35-604" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">data =</span> dfs <span class="sc">|&gt;</span> <span class="fu">filter</span>(x <span class="sc">&lt;</span> <span class="dv">50</span> <span class="sc">&amp;</span> x <span class="sc">&gt;</span> <span class="dv">35</span>),</span>
<span id="cb35-605"><a href="#cb35-605" aria-hidden="true" tabindex="-1"></a>            <span class="at">fill=</span><span class="st">"coral1"</span>, <span class="at">alpha=</span>.<span class="dv">5</span>)</span>
<span id="cb35-606"><a href="#cb35-606" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-607"><a href="#cb35-607" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-608"><a href="#cb35-608" aria-hidden="true" tabindex="-1"></a>What is the probability of getting a snail that weighs below $35g$ or over $50g$?</span>
<span id="cb35-609"><a href="#cb35-609" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-610"><a href="#cb35-610" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-611"><a href="#cb35-611" aria-hidden="true" tabindex="-1"></a><span class="fu">pnorm</span>(<span class="dv">35</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>)) <span class="sc">+</span> </span>
<span id="cb35-612"><a href="#cb35-612" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pnorm</span>(<span class="dv">50</span>, <span class="at">mean=</span><span class="dv">40</span>, <span class="at">sd=</span><span class="fu">sqrt</span>(<span class="dv">20</span>), <span class="at">lower.tail=</span><span class="cn">FALSE</span>) </span>
<span id="cb35-613"><a href="#cb35-613" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-614"><a href="#cb35-614" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-615"><a href="#cb35-615" aria-hidden="true" tabindex="-1"></a>Under standard normal, the areas under the pdf curve are shown in @fig-ncurves for various situations.</span>
<span id="cb35-616"><a href="#cb35-616" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-619"><a href="#cb35-619" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-620"><a href="#cb35-620" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-ncurves</span></span>
<span id="cb35-621"><a href="#cb35-621" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-622"><a href="#cb35-622" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Standard normal coverage'</span></span>
<span id="cb35-623"><a href="#cb35-623" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 5</span></span>
<span id="cb35-624"><a href="#cb35-624" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-625"><a href="#cb35-625" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-626"><a href="#cb35-626" aria-hidden="true" tabindex="-1"></a>plot_area <span class="ot">&lt;-</span> <span class="cf">function</span>(lower, upper) {</span>
<span id="cb35-627"><a href="#cb35-627" aria-hidden="true" tabindex="-1"></a>  x <span class="ot">&lt;-</span> <span class="fu">seq</span>(<span class="sc">-</span><span class="dv">3</span>,<span class="dv">3</span>,<span class="at">length.out =</span> <span class="dv">1000</span>)</span>
<span id="cb35-628"><a href="#cb35-628" aria-hidden="true" tabindex="-1"></a>  y <span class="ot">&lt;-</span> <span class="fu">dnorm</span>(x, <span class="dv">0</span>, <span class="dv">1</span>)</span>
<span id="cb35-629"><a href="#cb35-629" aria-hidden="true" tabindex="-1"></a>  df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">x=</span>x, <span class="at">y=</span>y)</span>
<span id="cb35-630"><a href="#cb35-630" aria-hidden="true" tabindex="-1"></a>  area <span class="ot">&lt;-</span> <span class="fu">round</span>(<span class="dv">100</span> <span class="sc">*</span> (<span class="fu">pnorm</span>(upper) <span class="sc">-</span> <span class="fu">pnorm</span>(lower)), <span class="dv">1</span>)</span>
<span id="cb35-631"><a href="#cb35-631" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>(df, <span class="fu">aes</span>(<span class="at">x=</span>x, <span class="at">y=</span>y)) <span class="sc">+</span> </span>
<span id="cb35-632"><a href="#cb35-632" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_line</span>() <span class="sc">+</span></span>
<span id="cb35-633"><a href="#cb35-633" aria-hidden="true" tabindex="-1"></a>    <span class="fu">stat_function</span>(<span class="at">fun =</span> dnorm,</span>
<span id="cb35-634"><a href="#cb35-634" aria-hidden="true" tabindex="-1"></a>                  <span class="at">xlim =</span> <span class="fu">c</span>(lower, upper),</span>
<span id="cb35-635"><a href="#cb35-635" aria-hidden="true" tabindex="-1"></a>                  <span class="at">geom =</span> <span class="st">"area"</span>,</span>
<span id="cb35-636"><a href="#cb35-636" aria-hidden="true" tabindex="-1"></a>                  <span class="at">fill=</span><span class="st">'coral1'</span>,</span>
<span id="cb35-637"><a href="#cb35-637" aria-hidden="true" tabindex="-1"></a>                  <span class="at">alpha=</span>.<span class="dv">5</span>) <span class="sc">+</span></span>
<span id="cb35-638"><a href="#cb35-638" aria-hidden="true" tabindex="-1"></a>    <span class="fu">scale_x_continuous</span>(<span class="at">breaks=</span><span class="fu">c</span>(lower, upper)) <span class="sc">+</span></span>
<span id="cb35-639"><a href="#cb35-639" aria-hidden="true" tabindex="-1"></a>    <span class="fu">annotate</span>(<span class="st">"text"</span>, <span class="at">label =</span> <span class="fu">paste</span>(area, <span class="st">"%"</span>, <span class="at">sep=</span><span class="st">""</span>), <span class="at">size=</span><span class="dv">3</span>, <span class="at">x =</span> <span class="dv">2</span>, <span class="at">y =</span> <span class="fl">0.3</span>) <span class="sc">+</span></span>
<span id="cb35-640"><a href="#cb35-640" aria-hidden="true" tabindex="-1"></a>    <span class="fu">xlab</span>(<span class="st">""</span>) <span class="sc">+</span> <span class="fu">ylab</span>(<span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb35-641"><a href="#cb35-641" aria-hidden="true" tabindex="-1"></a>    <span class="fu">theme</span>(<span class="at">axis.text.y =</span> <span class="fu">element_blank</span>(),</span>
<span id="cb35-642"><a href="#cb35-642" aria-hidden="true" tabindex="-1"></a>          <span class="at">axis.ticks.y =</span> <span class="fu">element_blank</span>())</span>
<span id="cb35-643"><a href="#cb35-643" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb35-644"><a href="#cb35-644" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-645"><a href="#cb35-645" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">plot_area</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="dv">1</span>)</span>
<span id="cb35-646"><a href="#cb35-646" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">plot_area</span>(<span class="sc">-</span><span class="dv">2</span>, <span class="dv">2</span>)</span>
<span id="cb35-647"><a href="#cb35-647" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">plot_area</span>(<span class="sc">-</span><span class="dv">3</span>, <span class="dv">3</span>)</span>
<span id="cb35-648"><a href="#cb35-648" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">plot_area</span>(<span class="sc">-</span><span class="fl">1.96</span>, <span class="fl">1.96</span>)</span>
<span id="cb35-649"><a href="#cb35-649" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-650"><a href="#cb35-650" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(patchwork)</span>
<span id="cb35-651"><a href="#cb35-651" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">+</span>p2<span class="sc">+</span>p3<span class="sc">+</span>p4</span>
<span id="cb35-652"><a href="#cb35-652" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-653"><a href="#cb35-653" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-654"><a href="#cb35-654" aria-hidden="true" tabindex="-1"></a><span class="fu">### Skewed distributions</span></span>
<span id="cb35-655"><a href="#cb35-655" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-656"><a href="#cb35-656" aria-hidden="true" tabindex="-1"></a>There are many probability distributions which are asymmetrical. A few of them are briefed here.</span>
<span id="cb35-657"><a href="#cb35-657" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-658"><a href="#cb35-658" aria-hidden="true" tabindex="-1"></a> **Log-normal**</span>
<span id="cb35-659"><a href="#cb35-659" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-660"><a href="#cb35-660" aria-hidden="true" tabindex="-1"></a>A random variable $X$ is log-normally distributed, when $log_e(X)$ follows normal. Alternatively, if $X$ follows normal, then $e^X$ follows log-normal.</span>
<span id="cb35-661"><a href="#cb35-661" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-662"><a href="#cb35-662" aria-hidden="true" tabindex="-1"></a>The log-normal density function is shown below:</span>
<span id="cb35-663"><a href="#cb35-663" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-664"><a href="#cb35-664" aria-hidden="true" tabindex="-1"></a>$$f(x)={\frac {1}{x\sigma {\sqrt {2\pi }}}}\ \exp \left(-{\frac {\left(\ln x-\mu \right)^{2}}{2\sigma ^{2}}}\right),~~ x&gt;0$$</span>
<span id="cb35-665"><a href="#cb35-665" aria-hidden="true" tabindex="-1"></a>While the R function <span class="in">`dnorm()`</span> gives the normal density, the function <span class="in">`dlnorm()`</span> gives the log-normal density.</span>
<span id="cb35-666"><a href="#cb35-666" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-667"><a href="#cb35-667" aria-hidden="true" tabindex="-1"></a>The standard log-normal pdf and CDF are plotted in @fig-lnrm</span>
<span id="cb35-668"><a href="#cb35-668" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-669"><a href="#cb35-669" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-672"><a href="#cb35-672" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-673"><a href="#cb35-673" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-lnrm</span></span>
<span id="cb35-674"><a href="#cb35-674" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-675"><a href="#cb35-675" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Log-normal pdf and CDF'</span></span>
<span id="cb35-676"><a href="#cb35-676" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 5</span></span>
<span id="cb35-677"><a href="#cb35-677" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 5</span></span>
<span id="cb35-678"><a href="#cb35-678" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-679"><a href="#cb35-679" aria-hidden="true" tabindex="-1"></a>dfln <span class="ot">&lt;-</span> <span class="fu">tibble</span>(<span class="at">x=</span><span class="fu">seq</span>(<span class="dv">0</span>, <span class="dv">4</span>, <span class="at">length=</span><span class="dv">1000</span>), </span>
<span id="cb35-680"><a href="#cb35-680" aria-hidden="true" tabindex="-1"></a>              <span class="st">`</span><span class="at">f(x)</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">dlnorm</span>(x), </span>
<span id="cb35-681"><a href="#cb35-681" aria-hidden="true" tabindex="-1"></a>              <span class="st">`</span><span class="at">F(x)</span><span class="st">`</span> <span class="ot">=</span> <span class="fu">plnorm</span>(x))</span>
<span id="cb35-682"><a href="#cb35-682" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-683"><a href="#cb35-683" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfln) <span class="sc">+</span> </span>
<span id="cb35-684"><a href="#cb35-684" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(<span class="at">x=</span>x, <span class="at">y=</span><span class="st">`</span><span class="at">f(x)</span><span class="st">`</span>) <span class="sc">+</span> </span>
<span id="cb35-685"><a href="#cb35-685" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_area</span>(<span class="at">alpha=</span>.<span class="dv">8</span>) <span class="sc">+</span> </span>
<span id="cb35-686"><a href="#cb35-686" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Standard log-normal density"</span>, </span>
<span id="cb35-687"><a href="#cb35-687" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"standard log-normal deviate, z"</span>)</span>
<span id="cb35-688"><a href="#cb35-688" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-689"><a href="#cb35-689" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfln) <span class="sc">+</span> </span>
<span id="cb35-690"><a href="#cb35-690" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(<span class="at">x=</span>x,<span class="at">y=</span><span class="st">`</span><span class="at">F(x)</span><span class="st">`</span>) <span class="sc">+</span> </span>
<span id="cb35-691"><a href="#cb35-691" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_line</span>() <span class="sc">+</span> </span>
<span id="cb35-692"><a href="#cb35-692" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Cumulative standard log-normal density"</span>, </span>
<span id="cb35-693"><a href="#cb35-693" aria-hidden="true" tabindex="-1"></a>       <span class="at">x =</span> <span class="st">"standard log-normal deviate, z"</span>)</span>
<span id="cb35-694"><a href="#cb35-694" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-695"><a href="#cb35-695" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">/</span>p2</span>
<span id="cb35-696"><a href="#cb35-696" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-697"><a href="#cb35-697" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-698"><a href="#cb35-698" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-699"><a href="#cb35-699" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-700"><a href="#cb35-700" aria-hidden="true" tabindex="-1"></a>The mean &amp; variance of the lognormal random variables are:</span>
<span id="cb35-701"><a href="#cb35-701" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-702"><a href="#cb35-702" aria-hidden="true" tabindex="-1"></a>$$\mu_X=e^{\left(\mu +{\frac {\sigma ^{2}}{2}}\right)}~~~~\sigma_X^2=(e^{\sigma ^{2}}-1) e^{(2\mu +\sigma ^{2})}$$</span>
<span id="cb35-703"><a href="#cb35-703" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-704"><a href="#cb35-704" aria-hidden="true" tabindex="-1"></a> **Weibull**</span>
<span id="cb35-705"><a href="#cb35-705" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-706"><a href="#cb35-706" aria-hidden="true" tabindex="-1"></a>The probability function of the Weibull distribution is given below:</span>
<span id="cb35-707"><a href="#cb35-707" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-708"><a href="#cb35-708" aria-hidden="true" tabindex="-1"></a>$$f(t;\eta,\beta) =</span>
<span id="cb35-709"><a href="#cb35-709" aria-hidden="true" tabindex="-1"></a>\begin{cases}</span>
<span id="cb35-710"><a href="#cb35-710" aria-hidden="true" tabindex="-1"></a>\frac{\beta}{\eta}\left(\frac{x}{\eta}\right)^{\beta-1}e^{-(x/\eta)^{\beta}}  ~~x\geq0 ,<span class="sc">\\</span></span>
<span id="cb35-711"><a href="#cb35-711" aria-hidden="true" tabindex="-1"></a>0~~~~~~~~~~~~~~~~~~~~~~~~~~~~~  x&lt;0,</span>
<span id="cb35-712"><a href="#cb35-712" aria-hidden="true" tabindex="-1"></a>\end{cases}$$</span>
<span id="cb35-713"><a href="#cb35-713" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-714"><a href="#cb35-714" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-715"><a href="#cb35-715" aria-hidden="true" tabindex="-1"></a>$\beta~(&gt; 0)$  is the called the **shape** parameter and $\eta~(&gt; 0)$ is the called **scale** parameter, and different values of $\beta$ will result in different distributional shapes; see @fig-weib.</span>
<span id="cb35-716"><a href="#cb35-716" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-717"><a href="#cb35-717" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-720"><a href="#cb35-720" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-721"><a href="#cb35-721" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-weib</span></span>
<span id="cb35-722"><a href="#cb35-722" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-723"><a href="#cb35-723" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Weibull distributional shapes'</span></span>
<span id="cb35-724"><a href="#cb35-724" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 4</span></span>
<span id="cb35-725"><a href="#cb35-725" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 7</span></span>
<span id="cb35-726"><a href="#cb35-726" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-727"><a href="#cb35-727" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">5</span>)) <span class="sc">|&gt;</span> </span>
<span id="cb35-728"><a href="#cb35-728" aria-hidden="true" tabindex="-1"></a>  <span class="fu">ggplot</span>() <span class="sc">+</span> <span class="fu">aes</span>(<span class="at">x=</span>x) <span class="sc">+</span></span>
<span id="cb35-729"><a href="#cb35-729" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dweibull, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="fl">0.5</span>, <span class="at">scale =</span> <span class="dv">1</span>),</span>
<span id="cb35-730"><a href="#cb35-730" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-731"><a href="#cb35-731" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 0.5, scale = 1"</span>)</span>
<span id="cb35-732"><a href="#cb35-732" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-733"><a href="#cb35-733" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">5</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-734"><a href="#cb35-734" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dweibull, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="dv">1</span>, <span class="at">scale =</span> <span class="dv">1</span>),</span>
<span id="cb35-735"><a href="#cb35-735" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-736"><a href="#cb35-736" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 1, scale = 1"</span>)</span>
<span id="cb35-737"><a href="#cb35-737" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-738"><a href="#cb35-738" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">5</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-739"><a href="#cb35-739" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dweibull, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="dv">3</span>, <span class="at">scale =</span> <span class="dv">1</span>),</span>
<span id="cb35-740"><a href="#cb35-740" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-741"><a href="#cb35-741" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 3, scale = 1"</span>)</span>
<span id="cb35-742"><a href="#cb35-742" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-743"><a href="#cb35-743" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">7</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-744"><a href="#cb35-744" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dweibull, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="dv">6</span>, <span class="at">scale =</span> <span class="dv">4</span>),</span>
<span id="cb35-745"><a href="#cb35-745" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-746"><a href="#cb35-746" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 6, scale = 4"</span>)</span>
<span id="cb35-747"><a href="#cb35-747" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-748"><a href="#cb35-748" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(patchwork)</span>
<span id="cb35-749"><a href="#cb35-749" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">+</span>p2<span class="sc">+</span>p3<span class="sc">+</span>p4</span>
<span id="cb35-750"><a href="#cb35-750" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-751"><a href="#cb35-751" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-752"><a href="#cb35-752" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-753"><a href="#cb35-753" aria-hidden="true" tabindex="-1"></a>The Weibull distribution becomes the exponential distribution for $\beta=1$.</span>
<span id="cb35-754"><a href="#cb35-754" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-755"><a href="#cb35-755" aria-hidden="true" tabindex="-1"></a>The scale parameter $\eta$ is called the *characteristic life* because $\eta$ becomes the quantile with slightly less than two-thirds of the population (63.21%) below it irrespective of the shape $\beta$ when this distribution is used to model lifetimes. </span>
<span id="cb35-756"><a href="#cb35-756" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-757"><a href="#cb35-757" aria-hidden="true" tabindex="-1"></a> **Gamma**</span>
<span id="cb35-758"><a href="#cb35-758" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-759"><a href="#cb35-759" aria-hidden="true" tabindex="-1"></a>The probability function of the gamma distribution with shape parameter $\alpha$ and scale parameter $\beta$ is given below:</span>
<span id="cb35-760"><a href="#cb35-760" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-761"><a href="#cb35-761" aria-hidden="true" tabindex="-1"></a>$$\displaystyle {\begin{aligned}f(x)={\frac {\beta ^{\alpha }x^{\alpha -1}e^{-\beta x}}{\Gamma (\alpha )}}\quad {\text{ for }}x&gt;0\quad \alpha ,\beta &gt;0,<span class="sc">\\</span><span class="co">[</span><span class="ot">6pt</span><span class="co">]</span>\end{aligned}}$$</span>
<span id="cb35-762"><a href="#cb35-762" aria-hidden="true" tabindex="-1"></a>where $\displaystyle \Gamma (\alpha)=\int _{0}^{\infty }x^{\alpha-1}e^{-x}\,dx.$ </span>
<span id="cb35-763"><a href="#cb35-763" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-764"><a href="#cb35-764" aria-hidden="true" tabindex="-1"></a>@fig-gam shows the plot of gamma density for various shape and scale parameters.</span>
<span id="cb35-765"><a href="#cb35-765" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-768"><a href="#cb35-768" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-769"><a href="#cb35-769" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-gam</span></span>
<span id="cb35-770"><a href="#cb35-770" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-771"><a href="#cb35-771" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Gamma distributional shapes'</span></span>
<span id="cb35-772"><a href="#cb35-772" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 4</span></span>
<span id="cb35-773"><a href="#cb35-773" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 7</span></span>
<span id="cb35-774"><a href="#cb35-774" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-775"><a href="#cb35-775" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">5</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-776"><a href="#cb35-776" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dgamma, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="fl">0.5</span>, <span class="at">scale =</span> <span class="dv">1</span>),</span>
<span id="cb35-777"><a href="#cb35-777" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-778"><a href="#cb35-778" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 0.5, scale = 1"</span>)</span>
<span id="cb35-779"><a href="#cb35-779" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-780"><a href="#cb35-780" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">5</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-781"><a href="#cb35-781" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dgamma, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="dv">1</span>, <span class="at">scale =</span> <span class="dv">1</span>),</span>
<span id="cb35-782"><a href="#cb35-782" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-783"><a href="#cb35-783" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 1, scale = 1"</span>)</span>
<span id="cb35-784"><a href="#cb35-784" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-785"><a href="#cb35-785" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">5</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-786"><a href="#cb35-786" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dgamma, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="dv">3</span>, <span class="at">scale =</span> <span class="dv">1</span>),</span>
<span id="cb35-787"><a href="#cb35-787" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-788"><a href="#cb35-788" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 3, scale = 1"</span>)</span>
<span id="cb35-789"><a href="#cb35-789" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-790"><a href="#cb35-790" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">7</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-791"><a href="#cb35-791" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dgamma, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape =</span> <span class="dv">6</span>, <span class="at">scale =</span> <span class="dv">4</span>),</span>
<span id="cb35-792"><a href="#cb35-792" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-793"><a href="#cb35-793" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape = 6, scale = 4"</span>)</span>
<span id="cb35-794"><a href="#cb35-794" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-795"><a href="#cb35-795" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">+</span>p2<span class="sc">+</span>p3<span class="sc">+</span>p4</span>
<span id="cb35-796"><a href="#cb35-796" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-797"><a href="#cb35-797" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-798"><a href="#cb35-798" aria-hidden="true" tabindex="-1"></a>Lognormal, Weibull  and  gamma distributions are useful to model right skewed data, particularly lifetimes data, but we need large amount of data, say over 300, to discriminate the three fitted distributions.</span>
<span id="cb35-799"><a href="#cb35-799" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-800"><a href="#cb35-800" aria-hidden="true" tabindex="-1"></a> **Beta distribution**</span>
<span id="cb35-801"><a href="#cb35-801" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-802"><a href="#cb35-802" aria-hidden="true" tabindex="-1"></a>The beta distribution is bounded on the interval $<span class="co">[</span><span class="ot">0, 1</span><span class="co">]</span>$ and  parameterised by two positive shape parameters, say $\alpha$ and $\beta$.</span>
<span id="cb35-803"><a href="#cb35-803" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-804"><a href="#cb35-804" aria-hidden="true" tabindex="-1"></a>The probability function of the Weibull distribution is given below:</span>
<span id="cb35-805"><a href="#cb35-805" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-806"><a href="#cb35-806" aria-hidden="true" tabindex="-1"></a>$$\begin{aligned}f(x;\alpha ,\beta ) ={\frac {x^{\alpha -1}(1-x)^{\beta -1}}{\displaystyle \int _{0}^{1}u^{\alpha -1}(1-u)^{\beta -1}\,du}}={\frac {1}{\mathrm {B} (\alpha ,\beta )}}x^{\alpha -1}(1-x)^{\beta -1}\end{aligned} $$</span>
<span id="cb35-807"><a href="#cb35-807" aria-hidden="true" tabindex="-1"></a>where $\mathrm {B} (\alpha ,\beta )=\frac {\Gamma (\alpha )\Gamma (\beta )}{\Gamma (\alpha +\beta )}$.  @fig-betas shows the plot of beta density for various shape parameter combinations.</span>
<span id="cb35-808"><a href="#cb35-808" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-809"><a href="#cb35-809" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-812"><a href="#cb35-812" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-813"><a href="#cb35-813" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-betas</span></span>
<span id="cb35-814"><a href="#cb35-814" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-815"><a href="#cb35-815" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Beta distributional shapes'</span></span>
<span id="cb35-816"><a href="#cb35-816" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 4</span></span>
<span id="cb35-817"><a href="#cb35-817" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 7</span></span>
<span id="cb35-818"><a href="#cb35-818" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-819"><a href="#cb35-819" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-820"><a href="#cb35-820" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dbeta, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape1 =</span> <span class="fl">0.5</span>, <span class="at">shape2 =</span> <span class="fl">0.5</span>),</span>
<span id="cb35-821"><a href="#cb35-821" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-822"><a href="#cb35-822" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape1 = 0.5, shape2 = 0.5"</span>)</span>
<span id="cb35-823"><a href="#cb35-823" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-824"><a href="#cb35-824" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-825"><a href="#cb35-825" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dbeta, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape1 =</span> <span class="dv">2</span>, <span class="at">shape2 =</span> <span class="dv">2</span>),</span>
<span id="cb35-826"><a href="#cb35-826" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-827"><a href="#cb35-827" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape1 = 2, shape2 = 2"</span>)</span>
<span id="cb35-828"><a href="#cb35-828" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-829"><a href="#cb35-829" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-830"><a href="#cb35-830" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dbeta, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape1 =</span> <span class="dv">2</span>, <span class="at">shape2 =</span> <span class="dv">5</span>),</span>
<span id="cb35-831"><a href="#cb35-831" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-832"><a href="#cb35-832" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape1 = 2, shape2 = 5"</span>)</span>
<span id="cb35-833"><a href="#cb35-833" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-834"><a href="#cb35-834" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">c</span>(<span class="dv">0</span>,<span class="dv">1</span>)), <span class="fu">aes</span>(<span class="at">x=</span>x)) <span class="sc">+</span></span>
<span id="cb35-835"><a href="#cb35-835" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dbeta, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">shape1 =</span> <span class="dv">5</span>, <span class="at">shape2 =</span> <span class="dv">2</span>),</span>
<span id="cb35-836"><a href="#cb35-836" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom=</span><span class="st">'area'</span>) <span class="sc">+</span></span>
<span id="cb35-837"><a href="#cb35-837" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"shape1 = 6, shape2 = 4"</span>)</span>
<span id="cb35-838"><a href="#cb35-838" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-839"><a href="#cb35-839" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">+</span>p2<span class="sc">+</span>p3<span class="sc">+</span>p4</span>
<span id="cb35-840"><a href="#cb35-840" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-841"><a href="#cb35-841" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-842"><a href="#cb35-842" aria-hidden="true" tabindex="-1"></a>When $\alpha=\beta=1$, the beta distribution becomes the continuous uniform distribution.  This distribution is useful to model component fractions such as percentage of protein in milk powder, and is also widely used in Bayesian analysis.</span>
<span id="cb35-843"><a href="#cb35-843" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-844"><a href="#cb35-844" aria-hidden="true" tabindex="-1"></a><span class="fu">### Empirical distributions</span></span>
<span id="cb35-845"><a href="#cb35-845" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-846"><a href="#cb35-846" aria-hidden="true" tabindex="-1"></a>Data from a normal (or any other distribution) may not look very much like the expected theoretical shape ('bell-curve' for normal)  unless there is very large amount of it.  @fig-rn200 is for a random sample of 200 from the standard normal distribution.  Compare the departures from the expected theoretical shape.  Recall that we covered the ECDF curves in the last Chapter.  The CDF curve is just theoretical one but fitted with data.  More on this later.</span>
<span id="cb35-847"><a href="#cb35-847" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-850"><a href="#cb35-850" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-851"><a href="#cb35-851" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-rn200</span></span>
<span id="cb35-852"><a href="#cb35-852" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-853"><a href="#cb35-853" aria-hidden="true" tabindex="-1"></a><span class="co">#| warning: false</span></span>
<span id="cb35-854"><a href="#cb35-854" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb35-855"><a href="#cb35-855" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Normal(0,1) data (n=200)'</span></span>
<span id="cb35-856"><a href="#cb35-856" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 5</span></span>
<span id="cb35-857"><a href="#cb35-857" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 4</span></span>
<span id="cb35-858"><a href="#cb35-858" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-859"><a href="#cb35-859" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-860"><a href="#cb35-860" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">1234</span>)</span>
<span id="cb35-861"><a href="#cb35-861" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-862"><a href="#cb35-862" aria-hidden="true" tabindex="-1"></a>dfm <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">x=</span><span class="fu">rnorm</span>(<span class="dv">200</span>, <span class="at">mean=</span><span class="dv">80</span>, <span class="at">sd=</span><span class="dv">12</span>))</span>
<span id="cb35-863"><a href="#cb35-863" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-864"><a href="#cb35-864" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm) <span class="sc">+</span> </span>
<span id="cb35-865"><a href="#cb35-865" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(x) <span class="sc">+</span> </span>
<span id="cb35-866"><a href="#cb35-866" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_histogram</span>(<span class="fu">aes</span>(<span class="at">y=</span><span class="fu">after_stat</span>(density)), <span class="at">colour=</span><span class="dv">1</span>) <span class="sc">+</span></span>
<span id="cb35-867"><a href="#cb35-867" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_function</span>(<span class="at">fun =</span> dnorm, </span>
<span id="cb35-868"><a href="#cb35-868" aria-hidden="true" tabindex="-1"></a>                <span class="at">args =</span> <span class="fu">list</span>(<span class="at">mean=</span><span class="fu">mean</span>(dfm<span class="sc">$</span>x), <span class="at">sd=</span><span class="fu">sd</span>(dfm<span class="sc">$</span>x)), </span>
<span id="cb35-869"><a href="#cb35-869" aria-hidden="true" tabindex="-1"></a>                <span class="at">geom =</span> <span class="st">"line"</span>)</span>
<span id="cb35-870"><a href="#cb35-870" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-871"><a href="#cb35-871" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm) <span class="sc">+</span> <span class="fu">aes</span>(x) <span class="sc">+</span> <span class="fu">geom_boxplot</span>()</span>
<span id="cb35-872"><a href="#cb35-872" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-873"><a href="#cb35-873" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(patchwork)</span>
<span id="cb35-874"><a href="#cb35-874" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-875"><a href="#cb35-875" aria-hidden="true" tabindex="-1"></a>p1<span class="sc">/</span>p2<span class="sc">+</span> <span class="fu">plot_layout</span>(<span class="at">heights =</span> <span class="fu">c</span>(<span class="dv">5</span>,<span class="dv">1</span>))</span>
<span id="cb35-876"><a href="#cb35-876" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-877"><a href="#cb35-877" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-878"><a href="#cb35-878" aria-hidden="true" tabindex="-1"></a> **Small sample effect**   For small samples, we do not expect many outliers but the 'ideal' shape may not be seen; see @fig-rn50.</span>
<span id="cb35-879"><a href="#cb35-879" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-882"><a href="#cb35-882" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-883"><a href="#cb35-883" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-rn50</span></span>
<span id="cb35-884"><a href="#cb35-884" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-885"><a href="#cb35-885" aria-hidden="true" tabindex="-1"></a><span class="co">#| warning: false</span></span>
<span id="cb35-886"><a href="#cb35-886" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb35-887"><a href="#cb35-887" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Normal(0,1) data (n=50)'</span></span>
<span id="cb35-888"><a href="#cb35-888" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 3</span></span>
<span id="cb35-889"><a href="#cb35-889" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 4</span></span>
<span id="cb35-890"><a href="#cb35-890" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-891"><a href="#cb35-891" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb35-892"><a href="#cb35-892" aria-hidden="true" tabindex="-1"></a>dfm <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">x =</span> <span class="fu">rnorm</span>(<span class="dv">50</span>, <span class="at">mean=</span><span class="dv">80</span>, <span class="at">sd =</span><span class="dv">12</span>))</span>
<span id="cb35-893"><a href="#cb35-893" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(x))<span class="sc">+</span><span class="fu">stat_ecdf</span>()<span class="sc">+</span><span class="fu">stat_function</span>(<span class="at">fun =</span> pnorm, <span class="at">args=</span><span class="fu">list</span>(<span class="at">mean=</span><span class="dv">80</span>, <span class="at">sd=</span><span class="dv">12</span>), <span class="at">colour=</span><span class="st">"red"</span>)</span>
<span id="cb35-894"><a href="#cb35-894" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> p1<span class="sc">+</span><span class="fu">labs</span>(<span class="at">title=</span><span class="st">"ECDF &amp; CDF curves"</span> )</span>
<span id="cb35-895"><a href="#cb35-895" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">sample =</span> x))<span class="sc">+</span><span class="fu">stat_qq</span>(<span class="at">distribution =</span> stats<span class="sc">::</span>qnorm)<span class="sc">+</span><span class="fu">stat_qq_line</span>()</span>
<span id="cb35-896"><a href="#cb35-896" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> p2<span class="sc">+</span><span class="fu">labs</span>(<span class="at">title =</span> <span class="st">"Normal quantile plot"</span>)</span>
<span id="cb35-897"><a href="#cb35-897" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(patchwork)</span>
<span id="cb35-898"><a href="#cb35-898" aria-hidden="true" tabindex="-1"></a>p1 </span>
<span id="cb35-899"><a href="#cb35-899" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-900"><a href="#cb35-900" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-901"><a href="#cb35-901" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-902"><a href="#cb35-902" aria-hidden="true" tabindex="-1"></a><span class="fu">### Importance of the normal distribution</span></span>
<span id="cb35-903"><a href="#cb35-903" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-904"><a href="#cb35-904" aria-hidden="true" tabindex="-1"></a>***Question***: Why do we use the Normal distribution so much? Answer: Because:</span>
<span id="cb35-905"><a href="#cb35-905" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-906"><a href="#cb35-906" aria-hidden="true" tabindex="-1"></a><span class="ss">1.  </span>Many physical, biological, engineering and economic quantities do seem to follow a Normal distribution (sometimes after a transformation of the data to an appropriate scale). An example is that the logarithm of the 'waiting time' between earthquakes in Wellington is normally distributed. (The use of the logarithm may worry you, but it is natural for the subject area. Note that the magnitude of earthquakes is also measured on a logarithmic scale, namely the familiar Richter scale.)</span>
<span id="cb35-907"><a href="#cb35-907" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-908"><a href="#cb35-908" aria-hidden="true" tabindex="-1"></a><span class="ss">2.  </span>Data that are *approximately* Normal seem to turn up very frequently in practice. In particular this may occur because</span>
<span id="cb35-909"><a href="#cb35-909" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-910"><a href="#cb35-910" aria-hidden="true" tabindex="-1"></a><span class="ss">3.  </span>The Normal distribution is often what you get *(for mathematical reasons)* when you add together (or average) lots of different things. This result is known as the central limit theorem (CLT). For example, test scores are calculated by adding together marks from different questions, or share market movements are the result of many individuals and firms making private decisions. There is another theory which establishes that the sampling distributions of quantiles of continuous distributions of are approximately normal for *large* sample sizes.</span>
<span id="cb35-911"><a href="#cb35-911" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-912"><a href="#cb35-912" aria-hidden="true" tabindex="-1"></a><span class="ss">4.  </span>We shall see in this course that residuals play a large part. These are, for example, the 'errors' left over after fitting a regression line to data. Quite often, we expect residuals to follow a Normal distribution.</span>
<span id="cb35-913"><a href="#cb35-913" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-914"><a href="#cb35-914" aria-hidden="true" tabindex="-1"></a><span class="fu">### Normal data in practice</span></span>
<span id="cb35-915"><a href="#cb35-915" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-916"><a href="#cb35-916" aria-hidden="true" tabindex="-1"></a>In practice, data from a 'Normal distribution' *may* not look very much like the 'bell-curve' unless there is very large amount of it. This statement has practical implications but first let us check whether the statement is true. We can do this by simulating batches of data from a Normal distribution (using software) and noting the variations across the data sets. Moreover, because the Normal distribution and its bell shaped curve are so important and ubiquitous, it is instructive to apply EDA methods to Normal data, for we would certainly like EDA methods to work well for that type of curve.</span>
<span id="cb35-917"><a href="#cb35-917" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-918"><a href="#cb35-918" aria-hidden="true" tabindex="-1"></a>For the first batch, we simulated 1000 numbers from a Normal distribution with mean 80 and standard deviation of 12. The numbers could represent, for example, the diastolic blood pressure of 1000 adult New Zealanders. @fig-simnor shows the histogram and the boxplot of the random normal data. Even though the typical bell-shaped pattern (theoretical normal curve shown in dotted lines) is evident, the histogram is not perfectly symmetric as the numbers were chosen at random so that by chance more fall into some classifications than would be expected. Also there are a few unusual values at the both extremes (see the boxplot). In practice if we didn't know that the data came from a Normal distribution we might investigate the three highest points to see if they were valid points or outliers. The extreme edges of a data often comprise very atypical values we would not tend to place too much weight on the conflicting evidence they give about symmetry. The mean and median of this simulated batch are somewhat closer but the median is slightly off-centred from the quartiles.*Well, so far so good. Now what if we have smaller batches?*</span>
<span id="cb35-919"><a href="#cb35-919" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-922"><a href="#cb35-922" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-923"><a href="#cb35-923" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-simnor</span></span>
<span id="cb35-924"><a href="#cb35-924" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-925"><a href="#cb35-925" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb35-926"><a href="#cb35-926" aria-hidden="true" tabindex="-1"></a><span class="co">#| warning: false</span></span>
<span id="cb35-927"><a href="#cb35-927" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Simulated normal data, n=100 and n=1000'</span></span>
<span id="cb35-928"><a href="#cb35-928" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 6</span></span>
<span id="cb35-929"><a href="#cb35-929" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 6</span></span>
<span id="cb35-930"><a href="#cb35-930" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-931"><a href="#cb35-931" aria-hidden="true" tabindex="-1"></a>Mean  <span class="ot">&lt;-</span>  <span class="dv">80</span></span>
<span id="cb35-932"><a href="#cb35-932" aria-hidden="true" tabindex="-1"></a>Sd  <span class="ot">&lt;-</span>  <span class="dv">12</span></span>
<span id="cb35-933"><a href="#cb35-933" aria-hidden="true" tabindex="-1"></a>n <span class="ot">&lt;-</span> <span class="dv">100</span></span>
<span id="cb35-934"><a href="#cb35-934" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb35-935"><a href="#cb35-935" aria-hidden="true" tabindex="-1"></a>dfm<span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">RNDnorm =</span> <span class="fu">rnorm</span>(n, Mean, Sd))</span>
<span id="cb35-936"><a href="#cb35-936" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> RNDnorm)) <span class="sc">+</span> </span>
<span id="cb35-937"><a href="#cb35-937" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_histogram</span>(<span class="fu">aes</span>(<span class="at">y =</span><span class="fu">after_stat</span>(density)),</span>
<span id="cb35-938"><a href="#cb35-938" aria-hidden="true" tabindex="-1"></a>                   <span class="at">breaks =</span> <span class="fu">seq</span>(Mean<span class="dv">-4</span><span class="sc">*</span>Sd, Mean<span class="sc">+</span><span class="dv">4</span><span class="sc">*</span>Sd, <span class="at">length.out =</span><span class="dv">20</span>), </span>
<span id="cb35-939"><a href="#cb35-939" aria-hidden="true" tabindex="-1"></a>                   <span class="at">colour =</span> <span class="st">"black"</span>, </span>
<span id="cb35-940"><a href="#cb35-940" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fill =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb35-941"><a href="#cb35-941" aria-hidden="true" tabindex="-1"></a><span class="fu">stat_function</span>(<span class="at">fun =</span> dnorm, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">mean =</span> <span class="fu">mean</span>(dfm<span class="sc">$</span>RNDnorm), <span class="at">sd =</span> <span class="fu">sd</span>(dfm<span class="sc">$</span>RNDnorm)))</span>
<span id="cb35-942"><a href="#cb35-942" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> p1<span class="sc">+</span><span class="fu">xlab</span>(<span class="st">"N(80,12) random data n=100"</span>)</span>
<span id="cb35-943"><a href="#cb35-943" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-944"><a href="#cb35-944" aria-hidden="true" tabindex="-1"></a>p11 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">y =</span> RNDnorm, <span class="at">x=</span><span class="fu">factor</span>(<span class="st">""</span>))) </span>
<span id="cb35-945"><a href="#cb35-945" aria-hidden="true" tabindex="-1"></a>p11 <span class="ot">&lt;-</span> p11 <span class="sc">+</span> <span class="fu">geom_boxplot</span>()<span class="sc">+</span><span class="fu">xlab</span>(<span class="st">""</span>)</span>
<span id="cb35-946"><a href="#cb35-946" aria-hidden="true" tabindex="-1"></a>p11 <span class="ot">&lt;-</span> p11 <span class="sc">+</span> <span class="fu">coord_flip</span>()<span class="sc">+</span><span class="fu">ylab</span>(<span class="st">"N(80,12) random data n=100"</span>)</span>
<span id="cb35-947"><a href="#cb35-947" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-948"><a href="#cb35-948" aria-hidden="true" tabindex="-1"></a>n <span class="ot">&lt;-</span> <span class="dv">1000</span></span>
<span id="cb35-949"><a href="#cb35-949" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb35-950"><a href="#cb35-950" aria-hidden="true" tabindex="-1"></a>dfm<span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">RNDnorm =</span> <span class="fu">rnorm</span>(n, Mean, Sd))</span>
<span id="cb35-951"><a href="#cb35-951" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> RNDnorm)) <span class="sc">+</span> </span>
<span id="cb35-952"><a href="#cb35-952" aria-hidden="true" tabindex="-1"></a>    <span class="fu">geom_histogram</span>(<span class="fu">aes</span>(<span class="at">y =</span>..density..),</span>
<span id="cb35-953"><a href="#cb35-953" aria-hidden="true" tabindex="-1"></a>                   <span class="at">breaks =</span> <span class="fu">seq</span>(Mean<span class="dv">-4</span><span class="sc">*</span>Sd, Mean<span class="sc">+</span><span class="dv">4</span><span class="sc">*</span>Sd, <span class="at">length.out =</span><span class="dv">20</span>), </span>
<span id="cb35-954"><a href="#cb35-954" aria-hidden="true" tabindex="-1"></a>                   <span class="at">colour =</span> <span class="st">"black"</span>, </span>
<span id="cb35-955"><a href="#cb35-955" aria-hidden="true" tabindex="-1"></a>                   <span class="at">fill =</span> <span class="st">"white"</span>) <span class="sc">+</span></span>
<span id="cb35-956"><a href="#cb35-956" aria-hidden="true" tabindex="-1"></a><span class="fu">stat_function</span>(<span class="at">fun =</span> dnorm, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">mean =</span> <span class="fu">mean</span>(dfm<span class="sc">$</span>RNDnorm), <span class="at">sd =</span> <span class="fu">sd</span>(dfm<span class="sc">$</span>RNDnorm)))</span>
<span id="cb35-957"><a href="#cb35-957" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-958"><a href="#cb35-958" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> p2<span class="sc">+</span><span class="fu">xlab</span>(<span class="st">"N(80,12) random data n=1000"</span>)</span>
<span id="cb35-959"><a href="#cb35-959" aria-hidden="true" tabindex="-1"></a>p22 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">y =</span> RNDnorm, <span class="at">x=</span><span class="fu">factor</span>(<span class="st">""</span>))) </span>
<span id="cb35-960"><a href="#cb35-960" aria-hidden="true" tabindex="-1"></a>p22 <span class="ot">&lt;-</span> p22 <span class="sc">+</span> <span class="fu">geom_boxplot</span>()<span class="sc">+</span><span class="fu">xlab</span>(<span class="st">""</span>)</span>
<span id="cb35-961"><a href="#cb35-961" aria-hidden="true" tabindex="-1"></a>p22 <span class="ot">&lt;-</span> p22 <span class="sc">+</span> <span class="fu">coord_flip</span>()<span class="sc">+</span><span class="fu">ylab</span>(<span class="st">"N(80,12) random data n=1000"</span>)</span>
<span id="cb35-962"><a href="#cb35-962" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-963"><a href="#cb35-963" aria-hidden="true" tabindex="-1"></a>gridExtra<span class="sc">::</span><span class="fu">grid.arrange</span>(p2,p1,p22,p11, <span class="at">ncol=</span><span class="dv">2</span>)</span>
<span id="cb35-964"><a href="#cb35-964" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-965"><a href="#cb35-965" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-966"><a href="#cb35-966" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-967"><a href="#cb35-967" aria-hidden="true" tabindex="-1"></a>For a normal distribution,</span>
<span id="cb35-968"><a href="#cb35-968" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-969"><a href="#cb35-969" aria-hidden="true" tabindex="-1"></a>68% of the values fall within 1 standard deviation of the mean\</span>
<span id="cb35-970"><a href="#cb35-970" aria-hidden="true" tabindex="-1"></a>95% of the values fall within 2 standard deviation of the mean\</span>
<span id="cb35-971"><a href="#cb35-971" aria-hidden="true" tabindex="-1"></a>99% of the values fall within 3 standard deviation of the mean.\</span>
<span id="cb35-972"><a href="#cb35-972" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-973"><a href="#cb35-973" aria-hidden="true" tabindex="-1"></a>As a special case, 50% of the values will fall within 0.6745 standard deviations of the mean. For symmetric distributions, mean equals the median. Hence the fourths (quartiles) of random data obtained drawn from theoretical normal population will be</span>
<span id="cb35-974"><a href="#cb35-974" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-975"><a href="#cb35-975" aria-hidden="true" tabindex="-1"></a>**Lower hinge** = median - $0.6745\times S~(\simeq$lower quartile),</span>
<span id="cb35-976"><a href="#cb35-976" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-977"><a href="#cb35-977" aria-hidden="true" tabindex="-1"></a>and</span>
<span id="cb35-978"><a href="#cb35-978" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-979"><a href="#cb35-979" aria-hidden="true" tabindex="-1"></a>**Upper hinge** = median + $0.6745 \times S~(\simeq$ upper quartile),</span>
<span id="cb35-980"><a href="#cb35-980" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-981"><a href="#cb35-981" aria-hidden="true" tabindex="-1"></a>where $S$ is the standard deviation. Thus F-spread (or $\simeq$ the IQR) $\simeq$ 1.35$\times S$. Hence</span>
<span id="cb35-982"><a href="#cb35-982" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-983"><a href="#cb35-983" aria-hidden="true" tabindex="-1"></a>**inner fences**= median $\pm$ 2 F-spread = median $\pm$ 2.7$\times S$</span>
<span id="cb35-984"><a href="#cb35-984" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-985"><a href="#cb35-985" aria-hidden="true" tabindex="-1"></a>The probability that an observation falls between these fences is 0.993, or the probability of falling outside these fences is 0.007. Thus in our simulated batch of size 1000, we expect about 7 possible outliers but none for the simulated batch size 100. **Outer fences** are defined as being 2 steps (3$\times$F-spread) away from the fourths so that</span>
<span id="cb35-986"><a href="#cb35-986" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-987"><a href="#cb35-987" aria-hidden="true" tabindex="-1"></a>**outer fences**= median $\pm$ 3.5$\times$F-spread = median $\pm$ 4.72$\times S$</span>
<span id="cb35-988"><a href="#cb35-988" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-989"><a href="#cb35-989" aria-hidden="true" tabindex="-1"></a>The probability of a point falling outside of this range is the extremely small amount of 0.000002. It is not surprising, then, that none of the values in our simulated batches fell outside of the outer fences. If they did, we would class them as probable outliers. Of course, with distributions other than the Gaussian one, and particularly for asymmetric distributions, the probability of a value falling outside these fences is higher.</span>
<span id="cb35-990"><a href="#cb35-990" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-991"><a href="#cb35-991" aria-hidden="true" tabindex="-1"></a><span class="fu">### Assessment of Normality</span></span>
<span id="cb35-992"><a href="#cb35-992" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-993"><a href="#cb35-993" aria-hidden="true" tabindex="-1"></a>The shapes of the ECDF curves shown in @fig-simnorecdf are typical of the empirical cumulative distributions for data from Normal populations. This curve was drawn using 1000 and 100 simulated random normal data. It could perhaps be described as an S-shape on its side (or lazy or stretched out S). The curve is flatter at either end which reflects the small amounts of relative frequency in the tails of the Normal distribution. The theoretical Cumulative Distribution Function (CDF) curve for the normal distribution is also shown in @fig-simnorecdf as a smooth curve. For smaller samples from a Normal population, the ECDF curve will vary about the theoretical curve, and, of course, the variation is likely to be larger the smaller the sample; see @fig-simnorecdf(B).</span>
<span id="cb35-994"><a href="#cb35-994" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-997"><a href="#cb35-997" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-998"><a href="#cb35-998" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-simnorecdf</span></span>
<span id="cb35-999"><a href="#cb35-999" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-1000"><a href="#cb35-1000" aria-hidden="true" tabindex="-1"></a><span class="co">#| warning: false</span></span>
<span id="cb35-1001"><a href="#cb35-1001" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Simulated normal data, n=100 and n=1000'</span></span>
<span id="cb35-1002"><a href="#cb35-1002" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1003"><a href="#cb35-1003" aria-hidden="true" tabindex="-1"></a>Mean  <span class="ot">&lt;-</span>  <span class="dv">80</span></span>
<span id="cb35-1004"><a href="#cb35-1004" aria-hidden="true" tabindex="-1"></a>Sd  <span class="ot">&lt;-</span>  <span class="dv">12</span></span>
<span id="cb35-1005"><a href="#cb35-1005" aria-hidden="true" tabindex="-1"></a>n <span class="ot">&lt;-</span> <span class="dv">100</span></span>
<span id="cb35-1006"><a href="#cb35-1006" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb35-1007"><a href="#cb35-1007" aria-hidden="true" tabindex="-1"></a>dfm<span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">RNDnorm =</span> <span class="fu">rnorm</span>(n, Mean, Sd))</span>
<span id="cb35-1008"><a href="#cb35-1008" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> RNDnorm)) <span class="sc">+</span> <span class="fu">stat_ecdf</span>(<span class="fu">aes</span>(<span class="at">color=</span><span class="st">"red"</span>)) </span>
<span id="cb35-1009"><a href="#cb35-1009" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> p1<span class="sc">+</span><span class="fu">stat_function</span>(<span class="at">fun =</span> pnorm, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">mean =</span> <span class="fu">mean</span>(dfm<span class="sc">$</span>RNDnorm), <span class="at">sd =</span> <span class="fu">sd</span>(dfm<span class="sc">$</span>RNDnorm)))</span>
<span id="cb35-1010"><a href="#cb35-1010" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> p1<span class="sc">+</span><span class="fu">guides</span>(<span class="at">colour=</span><span class="cn">FALSE</span>)<span class="sc">+</span><span class="fu">ggtitle</span>(<span class="st">"(A)"</span>)</span>
<span id="cb35-1011"><a href="#cb35-1011" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> p1<span class="sc">+</span><span class="fu">xlab</span>(<span class="st">"N(80,12) random data n=100"</span>)<span class="sc">+</span><span class="fu">ylab</span>(<span class="st">"Cumulative proportion/probability"</span>)</span>
<span id="cb35-1012"><a href="#cb35-1012" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1013"><a href="#cb35-1013" aria-hidden="true" tabindex="-1"></a>n <span class="ot">&lt;-</span> <span class="dv">1000</span></span>
<span id="cb35-1014"><a href="#cb35-1014" aria-hidden="true" tabindex="-1"></a>dfm<span class="ot">&lt;-</span> <span class="fu">data.frame</span>(<span class="at">RNDnorm =</span> <span class="fu">rnorm</span>(n, Mean, Sd))</span>
<span id="cb35-1015"><a href="#cb35-1015" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(dfm, <span class="fu">aes</span>(<span class="at">x =</span> RNDnorm)) <span class="sc">+</span> <span class="fu">stat_ecdf</span>(<span class="fu">aes</span>(<span class="at">color=</span><span class="st">"red"</span>)) </span>
<span id="cb35-1016"><a href="#cb35-1016" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> p2<span class="sc">+</span><span class="fu">stat_function</span>(<span class="at">fun =</span> pnorm, <span class="at">args =</span> <span class="fu">list</span>(<span class="at">mean =</span> <span class="fu">mean</span>(dfm<span class="sc">$</span>RNDnorm), <span class="at">sd =</span> <span class="fu">sd</span>(dfm<span class="sc">$</span>RNDnorm)))</span>
<span id="cb35-1017"><a href="#cb35-1017" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> p2<span class="sc">+</span><span class="fu">guides</span>(<span class="at">colour=</span><span class="cn">FALSE</span>)<span class="sc">+</span><span class="fu">ggtitle</span>(<span class="st">"(B)"</span>)</span>
<span id="cb35-1018"><a href="#cb35-1018" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> p2<span class="sc">+</span><span class="fu">xlab</span>(<span class="st">"N(80,12) random data n=1000"</span>)<span class="sc">+</span><span class="fu">ylab</span>(<span class="st">"Cumulative proportion/probability"</span>)</span>
<span id="cb35-1019"><a href="#cb35-1019" aria-hidden="true" tabindex="-1"></a>gridExtra<span class="sc">::</span><span class="fu">grid.arrange</span>(p2,p1, <span class="at">ncol=</span><span class="dv">2</span>)</span>
<span id="cb35-1020"><a href="#cb35-1020" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1021"><a href="#cb35-1021" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1022"><a href="#cb35-1022" aria-hidden="true" tabindex="-1"></a><span class="fu">### Normal quantile-quantile plots</span></span>
<span id="cb35-1023"><a href="#cb35-1023" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1024"><a href="#cb35-1024" aria-hidden="true" tabindex="-1"></a>To visually check whether a given set of data are normally distributed, one can create a normal quantile-quantile (Q-Q) plot. In this plot the quantiles of the sample are plotted against the theoretical quantiles of a standard normal distribution N(0,1). If the data, $Y_{i}$, are normally distributed N($\mu, \sigma$) then $Z_{i} = \left(Y_{i} -\mu \right)/\sigma$ has a standard normal distribution. If we plot the quantiles of the data against corresponding quantiles of the standard normal distribution, the points should roughly lie on the straight line: $$Y_{i} =\mu +\sigma Z_{i}$$ that is, a line with intercept $\mu$, and slope $\sigma$.</span>
<span id="cb35-1025"><a href="#cb35-1025" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1026"><a href="#cb35-1026" aria-hidden="true" tabindex="-1"></a>We should note that the mean and standard deviation do not appear in the formula $Y_{i} =\mu +\sigma Z_{i}$ by coincidence. Rather, the mean and standard deviation are intimately connected to the normal distribution: if the data had not been normal then the line with these parameters would not pass through the data.</span>
<span id="cb35-1027"><a href="#cb35-1027" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1030"><a href="#cb35-1030" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1031"><a href="#cb35-1031" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-normqq</span></span>
<span id="cb35-1032"><a href="#cb35-1032" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-1033"><a href="#cb35-1033" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Normal Quantile-Quantile Plot for Checking Normality'</span></span>
<span id="cb35-1034"><a href="#cb35-1034" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1035"><a href="#cb35-1035" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb35-1036"><a href="#cb35-1036" aria-hidden="true" tabindex="-1"></a>x  <span class="ot">&lt;-</span>  <span class="fu">rnorm</span>(<span class="at">n =</span> <span class="dv">50</span>, <span class="at">mean =</span> <span class="dv">80</span>, <span class="at">sd =</span> <span class="dv">12</span>) <span class="co">#qqnorm(x); qqline(x) # base function</span></span>
<span id="cb35-1037"><a href="#cb35-1037" aria-hidden="true" tabindex="-1"></a><span class="fu">ggplot</span>(<span class="fu">data.frame</span>(x), <span class="fu">aes</span>(<span class="at">sample =</span> x)) <span class="sc">+</span>  <span class="fu">stat_qq</span>() <span class="sc">+</span> <span class="fu">stat_qq_line</span>() <span class="sc">+</span></span>
<span id="cb35-1038"><a href="#cb35-1038" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_minimal</span>()</span>
<span id="cb35-1039"><a href="#cb35-1039" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1040"><a href="#cb35-1040" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1041"><a href="#cb35-1041" aria-hidden="true" tabindex="-1"></a>@fig-normqq shows the normal Q-Q plot for 50 random values from N(80,12). Note that the points in the middle largely plot on a line. It is natural that the tail part of the distribution shows more variation, and hence we will ignore such departures.</span>
<span id="cb35-1042"><a href="#cb35-1042" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1043"><a href="#cb35-1043" aria-hidden="true" tabindex="-1"></a>The normal Q-Q plot is presented as a **normal probability plot** in some software where the theoretical quantiles are replaced by the associated normal probabilities. Such normal probability plots show the sample data in the $X$-axis and the normal probabilities in the $Y$-axis. In both plots, we mainly look whether or not the points plot roughly on a straight line.</span>
<span id="cb35-1044"><a href="#cb35-1044" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1047"><a href="#cb35-1047" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1048"><a href="#cb35-1048" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-normqqP</span></span>
<span id="cb35-1049"><a href="#cb35-1049" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-1050"><a href="#cb35-1050" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Normal Quantile-Quantile Plot for Checking Normality'</span></span>
<span id="cb35-1051"><a href="#cb35-1051" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1052"><a href="#cb35-1052" aria-hidden="true" tabindex="-1"></a><span class="fu">download.file</span>(</span>
<span id="cb35-1053"><a href="#cb35-1053" aria-hidden="true" tabindex="-1"></a>  <span class="at">url =</span> <span class="st">"http://www.massey.ac.nz/~anhsmith/data/rangitikei.RData"</span>,</span>
<span id="cb35-1054"><a href="#cb35-1054" aria-hidden="true" tabindex="-1"></a>  <span class="at">destfile =</span> <span class="st">"rangitikei.RData"</span>)</span>
<span id="cb35-1055"><a href="#cb35-1055" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1056"><a href="#cb35-1056" aria-hidden="true" tabindex="-1"></a><span class="fu">load</span>(<span class="st">"rangitikei.RData"</span>)</span>
<span id="cb35-1057"><a href="#cb35-1057" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1058"><a href="#cb35-1058" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(rangitikei) <span class="sc">+</span></span>
<span id="cb35-1059"><a href="#cb35-1059" aria-hidden="true" tabindex="-1"></a>  <span class="fu">aes</span>(<span class="at">sample =</span> people) <span class="sc">+</span> </span>
<span id="cb35-1060"><a href="#cb35-1060" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_qq</span>() <span class="sc">+</span> <span class="fu">stat_qq_line</span>()</span>
<span id="cb35-1061"><a href="#cb35-1061" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1062"><a href="#cb35-1062" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(rangitikei) <span class="sc">+</span> <span class="fu">aes</span>(<span class="at">y=</span>people, <span class="at">x=</span><span class="st">""</span>) <span class="sc">+</span></span>
<span id="cb35-1063"><a href="#cb35-1063" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_boxplot</span>() <span class="sc">+</span></span>
<span id="cb35-1064"><a href="#cb35-1064" aria-hidden="true" tabindex="-1"></a>  <span class="fu">xlab</span>(<span class="st">""</span>) <span class="sc">+</span> </span>
<span id="cb35-1065"><a href="#cb35-1065" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>()</span>
<span id="cb35-1066"><a href="#cb35-1066" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1067"><a href="#cb35-1067" aria-hidden="true" tabindex="-1"></a>gridExtra<span class="sc">::</span><span class="fu">grid.arrange</span>(p1, p2, <span class="at">ncol=</span><span class="dv">1</span>) </span>
<span id="cb35-1068"><a href="#cb35-1068" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1069"><a href="#cb35-1069" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1070"><a href="#cb35-1070" aria-hidden="true" tabindex="-1"></a>@fig-normqqP shows the normal Q-Q plot for the number of people who made use of a recreational facility (**rangitikei**). This plot clearly reveals that the data are not normal. The plotted points show a curved but asymmetric pattern. This pattern matches with the right skewness seen in the data (see the boxplot given in @fig-normqqP.</span>
<span id="cb35-1071"><a href="#cb35-1071" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1072"><a href="#cb35-1072" aria-hidden="true" tabindex="-1"></a>Note that **normality and symmetry are not the same**. If the true distribution of the data is symmetric such as uniform, rectangular or triangular distributions, then the normal Q-Q plot will show a symmetric pattern for the plotted points but they will not plot on a straight line. As an example, the normal Q-Q plots for batches of random data from Student's $t$ (to be covered later in the next Chapter) and few other symmetric distributions is shown in @fig-normqqsym. Clearly normality is rejected for other symmetric non-normal distributions.</span>
<span id="cb35-1073"><a href="#cb35-1073" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1076"><a href="#cb35-1076" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1077"><a href="#cb35-1077" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-normqqsym</span></span>
<span id="cb35-1078"><a href="#cb35-1078" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: false</span></span>
<span id="cb35-1079"><a href="#cb35-1079" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Normal Quantile-Quantile Plot for Checking Normality'</span></span>
<span id="cb35-1080"><a href="#cb35-1080" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 6</span></span>
<span id="cb35-1081"><a href="#cb35-1081" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 6</span></span>
<span id="cb35-1082"><a href="#cb35-1082" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1083"><a href="#cb35-1083" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb35-1084"><a href="#cb35-1084" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1085"><a href="#cb35-1085" aria-hidden="true" tabindex="-1"></a>x  <span class="ot">&lt;-</span>  <span class="fu">rt</span>(<span class="dv">500</span>, <span class="dv">2</span>) </span>
<span id="cb35-1086"><a href="#cb35-1086" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(x), <span class="fu">aes</span>(<span class="at">sample =</span> x)) <span class="sc">+</span>  <span class="fu">stat_qq</span>() <span class="sc">+</span> <span class="fu">stat_qq_line</span>() <span class="sc">+</span></span>
<span id="cb35-1087"><a href="#cb35-1087" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_minimal</span>()<span class="sc">+</span><span class="fu">ggtitle</span>(<span class="st">"Student t data with 2 df - n=500"</span>)</span>
<span id="cb35-1088"><a href="#cb35-1088" aria-hidden="true" tabindex="-1"></a>x  <span class="ot">&lt;-</span>  <span class="fu">runif</span>(<span class="dv">500</span>) </span>
<span id="cb35-1089"><a href="#cb35-1089" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(x), <span class="fu">aes</span>(<span class="at">sample =</span> x)) <span class="sc">+</span>  <span class="fu">stat_qq</span>() <span class="sc">+</span> <span class="fu">stat_qq_line</span>() <span class="sc">+</span></span>
<span id="cb35-1090"><a href="#cb35-1090" aria-hidden="true" tabindex="-1"></a>      <span class="fu">theme_minimal</span>()<span class="sc">+</span><span class="fu">ggtitle</span>(<span class="st">"Uniform data - n=500"</span>)</span>
<span id="cb35-1091"><a href="#cb35-1091" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1092"><a href="#cb35-1092" aria-hidden="true" tabindex="-1"></a>x  <span class="ot">&lt;-</span>  <span class="fu">rbeta</span>(<span class="dv">500</span>, <span class="dv">2</span>,<span class="dv">2</span>) </span>
<span id="cb35-1093"><a href="#cb35-1093" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(x), <span class="fu">aes</span>(<span class="at">sample =</span> x)) <span class="sc">+</span> </span>
<span id="cb35-1094"><a href="#cb35-1094" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_qq</span>() <span class="sc">+</span> <span class="fu">stat_qq_line</span>() <span class="sc">+</span></span>
<span id="cb35-1095"><a href="#cb35-1095" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span><span class="fu">ggtitle</span>(<span class="st">"Beta(2,2) data - n=500"</span>)</span>
<span id="cb35-1096"><a href="#cb35-1096" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1097"><a href="#cb35-1097" aria-hidden="true" tabindex="-1"></a>x  <span class="ot">&lt;-</span>  extraDistr<span class="sc">::</span><span class="fu">rlaplace</span>(<span class="dv">500</span>) </span>
<span id="cb35-1098"><a href="#cb35-1098" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1099"><a href="#cb35-1099" aria-hidden="true" tabindex="-1"></a>p4 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(<span class="fu">data.frame</span>(x), <span class="fu">aes</span>(<span class="at">sample =</span> x)) <span class="sc">+</span>  <span class="fu">stat_qq</span>() <span class="sc">+</span> </span>
<span id="cb35-1100"><a href="#cb35-1100" aria-hidden="true" tabindex="-1"></a>  <span class="fu">stat_qq_line</span>() <span class="sc">+</span></span>
<span id="cb35-1101"><a href="#cb35-1101" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>()<span class="sc">+</span><span class="fu">ggtitle</span>(<span class="st">"Laplace data - n=500"</span>)</span>
<span id="cb35-1102"><a href="#cb35-1102" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1103"><a href="#cb35-1103" aria-hidden="true" tabindex="-1"></a>gridExtra<span class="sc">::</span><span class="fu">grid.arrange</span>(p1,p2, p3, p4, <span class="at">ncol=</span><span class="dv">2</span>) </span>
<span id="cb35-1104"><a href="#cb35-1104" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1105"><a href="#cb35-1105" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1106"><a href="#cb35-1106" aria-hidden="true" tabindex="-1"></a><span class="fu">### Some non-normal distributions</span></span>
<span id="cb35-1107"><a href="#cb35-1107" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1108"><a href="#cb35-1108" aria-hidden="true" tabindex="-1"></a>For a normal distribution, which is specified by its mean $\mu$ and variance $\sigma^2$, parameter estimation is fairly straight forward.  The sample mean and variance become the estimates.   The Poisson parameter  $\lambda$ is also the mean of the distribution and hence the sample mean becomes the estimate.  On the other hand, distributions such as beta, and gamma involve shape and/or scale parameters and hence their estimation becomes complicated.  The population mean and SD of these distributions are functions of the distributional parameters.  Data from skewed distributions are not well represented by the sample mean and SD.</span>
<span id="cb35-1109"><a href="#cb35-1109" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1110"><a href="#cb35-1110" aria-hidden="true" tabindex="-1"></a>In order to estimate the distributional parameters, we follow a method called the **Maximum Likelihood** (ML) method. The *likelihood* function (or simply the likelihood) is based on the joint probability of the observed data as a function of the distributional parameters.  The ML method maximises the likelihood function to estimate the distributional (model) parameters.  There are also other methods of estimation such as the method of moments and we will not delve into the theory behind these methods in this course. We will simply use R packages such as <span class="in">`VGAM`</span> or <span class="in">`fitdistrplus`</span> to estimate the parameters.</span>
<span id="cb35-1111"><a href="#cb35-1111" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1112"><a href="#cb35-1112" aria-hidden="true" tabindex="-1"></a>Example: Your are familiar with the **rangitikei** dataset which gives the number of people who made use of a recreational facility.  This variable is right skewed, and let us fit a lognormal distribution to this variable using the <span class="in">`fitdistrplus`</span> package.</span>
<span id="cb35-1113"><a href="#cb35-1113" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1114"><a href="#cb35-1114" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1117"><a href="#cb35-1117" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1118"><a href="#cb35-1118" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-1119"><a href="#cb35-1119" aria-hidden="true" tabindex="-1"></a><span class="co">#| message: false</span></span>
<span id="cb35-1120"><a href="#cb35-1120" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(fitdistrplus, <span class="at">exclude =</span> <span class="fu">c</span>(<span class="st">"select"</span>, <span class="st">"area"</span>))</span>
<span id="cb35-1121"><a href="#cb35-1121" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1122"><a href="#cb35-1122" aria-hidden="true" tabindex="-1"></a><span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"lnorm"</span>)</span>
<span id="cb35-1123"><a href="#cb35-1123" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1124"><a href="#cb35-1124" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1125"><a href="#cb35-1125" aria-hidden="true" tabindex="-1"></a>The adequacy of the lognormal distribution fitted, can be examined using graphical displays as in @fig-lnormfit.</span>
<span id="cb35-1126"><a href="#cb35-1126" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1129"><a href="#cb35-1129" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1130"><a href="#cb35-1130" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-lnormfit</span></span>
<span id="cb35-1131"><a href="#cb35-1131" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-1132"><a href="#cb35-1132" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Log-normal fit for people'</span></span>
<span id="cb35-1133"><a href="#cb35-1133" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 7</span></span>
<span id="cb35-1134"><a href="#cb35-1134" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 7</span></span>
<span id="cb35-1135"><a href="#cb35-1135" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1136"><a href="#cb35-1136" aria-hidden="true" tabindex="-1"></a>lnormfit <span class="ot">&lt;-</span> <span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"lnorm"</span>)</span>
<span id="cb35-1137"><a href="#cb35-1137" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(lnormfit)</span>
<span id="cb35-1138"><a href="#cb35-1138" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1139"><a href="#cb35-1139" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1140"><a href="#cb35-1140" aria-hidden="true" tabindex="-1"></a>The lognormal fit does not appear too bad to people data. How about fitting gamma distribution instead of lognormal?  This fit can again be accomplished using the <span class="in">`fitdistrplus`</span> package.</span>
<span id="cb35-1141"><a href="#cb35-1141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1142"><a href="#cb35-1142" aria-hidden="true" tabindex="-1"></a><span class="in">```{r, echo=TRUE}</span></span>
<span id="cb35-1143"><a href="#cb35-1143" aria-hidden="true" tabindex="-1"></a><span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"gamma"</span>)</span>
<span id="cb35-1144"><a href="#cb35-1144" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1145"><a href="#cb35-1145" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1146"><a href="#cb35-1146" aria-hidden="true" tabindex="-1"></a>The EDA plots to assess the gamma distribution fit are shown in @fig-gammafit.</span>
<span id="cb35-1147"><a href="#cb35-1147" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1150"><a href="#cb35-1150" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1151"><a href="#cb35-1151" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-gammafit</span></span>
<span id="cb35-1152"><a href="#cb35-1152" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-1153"><a href="#cb35-1153" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Gamma fit for people'</span></span>
<span id="cb35-1154"><a href="#cb35-1154" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 7</span></span>
<span id="cb35-1155"><a href="#cb35-1155" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 7</span></span>
<span id="cb35-1156"><a href="#cb35-1156" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1157"><a href="#cb35-1157" aria-hidden="true" tabindex="-1"></a>gammafit <span class="ot">&lt;-</span> <span class="fu">fitdist</span>(rangitikei<span class="sc">$</span>people, <span class="st">"gamma"</span>)</span>
<span id="cb35-1158"><a href="#cb35-1158" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(gammafit)</span>
<span id="cb35-1159"><a href="#cb35-1159" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1160"><a href="#cb35-1160" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1161"><a href="#cb35-1161" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1162"><a href="#cb35-1162" aria-hidden="true" tabindex="-1"></a>Note that this package gives the estimated gamma parameters in a reparametrised form. For small and medium size datasets $(n&lt;300)$, we may not be able discriminate between gamma &amp; Weibull type of right skewed distributions.</span>
<span id="cb35-1163"><a href="#cb35-1163" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1164"><a href="#cb35-1164" aria-hidden="true" tabindex="-1"></a>We can of course fit a gallery of probability distributions but we have to select the *best* fitting distribution, if it exists.</span>
<span id="cb35-1165"><a href="#cb35-1165" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1166"><a href="#cb35-1166" aria-hidden="true" tabindex="-1"></a>The function <span class="in">`descdist()`</span> in  the <span class="in">`fitdistrplus`</span> package will obtain a plot of kurtosis, a measure of excessive peakedness, against the square of a measure of skew for a number of probability distributions.  This plot is known as the *Cullen and Frey plot*.  We can visibly judge the best fitting distribution using this plot.  For the number of people data, this plot is shown in @fig-cfplot.</span>
<span id="cb35-1167"><a href="#cb35-1167" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1170"><a href="#cb35-1170" aria-hidden="true" tabindex="-1"></a><span class="in">```{r}</span></span>
<span id="cb35-1171"><a href="#cb35-1171" aria-hidden="true" tabindex="-1"></a><span class="co">#| label: fig-cfplot</span></span>
<span id="cb35-1172"><a href="#cb35-1172" aria-hidden="true" tabindex="-1"></a><span class="co">#| echo: true</span></span>
<span id="cb35-1173"><a href="#cb35-1173" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-cap: 'Cullen and Frey plot for people'</span></span>
<span id="cb35-1174"><a href="#cb35-1174" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-height: 6</span></span>
<span id="cb35-1175"><a href="#cb35-1175" aria-hidden="true" tabindex="-1"></a><span class="co">#| fig-width: 7</span></span>
<span id="cb35-1176"><a href="#cb35-1176" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1177"><a href="#cb35-1177" aria-hidden="true" tabindex="-1"></a><span class="fu">descdist</span>(rangitikei<span class="sc">$</span>people)</span>
<span id="cb35-1178"><a href="#cb35-1178" aria-hidden="true" tabindex="-1"></a><span class="in">```</span></span>
<span id="cb35-1179"><a href="#cb35-1179" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1180"><a href="#cb35-1180" aria-hidden="true" tabindex="-1"></a>The presence of a large outlier in peoples data particularly affect the measures of skewness and kurtosis.  So Cullen and Frey plot does not particularly strongly support the lognormal fit but suggests the lack of fit for symmetrical distributions such as normal (read corresponding to zero skew).</span>
<span id="cb35-1181"><a href="#cb35-1181" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1182"><a href="#cb35-1182" aria-hidden="true" tabindex="-1"></a>When the observed data is a mixture from two or more distributions or contain a large number of unusual observations, no single distributional fit will be satisfactory.  For example, count data may not fit the Poisson distribution due to excessive zeros.</span>
<span id="cb35-1183"><a href="#cb35-1183" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1184"><a href="#cb35-1184" aria-hidden="true" tabindex="-1"></a>We will cover the power transformation of data in the next Chapter, which is a remedy for dampening the effect of outliers and mixture of subgroups (distributions).</span>
<span id="cb35-1185"><a href="#cb35-1185" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1186"><a href="#cb35-1186" aria-hidden="true" tabindex="-1"></a><span class="fu"># Summary</span></span>
<span id="cb35-1187"><a href="#cb35-1187" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1188"><a href="#cb35-1188" aria-hidden="true" tabindex="-1"></a>This Chapter provided a revision of probability concepts usually covered the first year level covered at the seventh form level or in a first year Statistics course.  A few popular probability distributions are also introduced.</span>
<span id="cb35-1189"><a href="#cb35-1189" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1190"><a href="#cb35-1190" aria-hidden="true" tabindex="-1"></a>Identification of a probability distribution is not an easy task for a set of observed data. This issue is not settled purely on statistical grounds.  EDA helps to short-list distributions fitting reasonably well to the data.  For regression type of models fitted to continuous responses, the normal distribution works generally well.</span>
<span id="cb35-1191"><a href="#cb35-1191" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-1192"><a href="#cb35-1192" aria-hidden="true" tabindex="-1"></a>The material covered in this Chapter forms the foundation for many of the inferential methods.</span>
</code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div></div></div></div></div>
</div> <!-- /content -->



</body></html>